<!DOCTYPE html>
<html>
<title>Abyzov Lab</title>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
<link href="https://fonts.googleapis.com/css?family=Cookie|Charm|Gentium+Basic|Gabriela|Kanit|Germania+One|Inconsolata|Raleway|Pacifico|Merriweather|Share+Tech+Mono" rel="stylesheet">

<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
<!--<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">-->
<style>
body {font-family: "Merriweather", sans-serif}
h1,h2,h3,h4,h5,h6 {font-family: "Gentium Basic", sans-serif}
.w3-top {font-family: "Merriweather", sans-serif;
  font-size:14px;
}
.logo {font-family: "Charm", sans-serif;
  font-size:32px;
  font-weight:bold;
  padding:16px;
  margin-top:4px;
}

body, html {
  height: 100%;
  line-height: 1.8;
}

#image{
  position:absolute;
  display:none
}

/* Full height image header */
.bgimg-1 {
  background-position: center;
  background-size: cover;
  background: #aaaaaa;
  background-image: url("imgs/group2.jpg");
  min-height: 100%;
  background-size: cover;
  background-repeat:no-repeat;
  background-position:center;
}

@media (orientation: landscape) {
  .bgimg-1 {
    background-image: url("imgs/group2.jpg");
    background-size: cover;
    background-position:center;
  }
}
.w3-bar {
  padding: 0px;
}
.w3-button {
  padding: 0px 16px;
}
.mySlides {display:none;}

.papertable {
  border-collapse: collapse;
  width: 100%;
}

.papertable td, .papertable th {
  border: 0px solid #ddd;
  padding: 8px;
  vertical-align:top;
}

.papertable tr:nth-child(even){background-color: #fff;}

.papertable1 tr:hover {background-color: #fff;}

.papertable th {
  padding-top: 12px;
  padding-bottom: 12px;
  text-align: left;
  background-color: #4CAF50;
  color: white;
}

.paperimg {
  height:25px;
  margin-left:5px;
}

</style>
<body>

<!-- Navbar (sit on top) -->
<div class="w3-top">
  <div class="w3-bar w3-white w3-card" id="topbar">
    <a href="#home" class="w3-bar-item w3-button logo">Abyzov lab</a>
    <!-- Right-sided navbar links -->
    <div class="w3-right w3-hide-medium w3-hide-small" style="margin-top:50px">
      <a href="#research" class="w3-bar-item w3-button"><i class="fas fa-flask"></i> RESEARCH</a>
      <a href="#news" class="w3-bar-item w3-button"><i class="fas fa-bullhorn"></i>NEWS</a>
      <a href="#team" class="w3-bar-item w3-button"><i class="fa fa-user"></i> TEAM</a>
      <a href="#papers" class="w3-bar-item w3-button"><i class="fas fa-file"></i>PAPERS</a>
      <a href="#tools" class="w3-bar-item w3-button"><i class="fas fa-tools"></i>TOOLS</a>
      <a href="#positions" class="w3-bar-item w3-button"><i class="fas fa-user-plus"></i>POSITIONS</a>
      <a href="#contact" class="w3-bar-item w3-button"><i class="fa fa-envelope"></i> CONTACT</a>
    </div>
    <!-- Hide right-floated links on small screens and replace them with a menu icon -->

    <a href="javascript:void(0)" class="w3-bar-item w3-button w3-right w3-hide-large" onclick="w3_open()">
      <i class="fa fa-bars"></i>
    </a>
  </div>
</div>

<!-- Sidebar on small screens when clicking the menu icon -->
<nav class="w3-sidebar w3-bar-block w3-black w3-card w3-animate-left w3-hide-large" style="display:none" id="mySidebar">
  <a href="javascript:void(0)" onclick="w3_close()" class="w3-bar-item w3-button w3-large w3-padding-16">Close ×</a>
  <a href="#research" onclick="w3_close()" class="w3-bar-item w3-button">RESEARCH</a>
  <a href="#news" onclick="w3_close()" class="w3-bar-item w3-button">NEWS</a>
  <a href="#team" onclick="w3_close()" class="w3-bar-item w3-button">TEAM</a>
  <a href="#papers" onclick="w3_close()" class="w3-bar-item w3-button">PAPERS</a>
  <a href="#tools" onclick="w3_close()" class="w3-bar-item w3-button">TOOLS</a>
  <a href="#positions" onclick="w3_close()" class="w3-bar-item w3-button">POSITIONS</a>
  <a href="#contact" onclick="w3_close()" class="w3-bar-item w3-button">CONTACT</a>
</nav>

<!-- Header with full-height image -->
<header class="bgimg-1 w3-display-container w3-grayscale-min" id="home">
  <div class="w3-display-left w3-text-white" style="padding:48px">
  </div> 
  <div class="w3-display-bottomleft w3-text-grey w3-large" style="padding:24px 48px">
    <i class="fa fa-facebook-official w3-hover-opacity"></i>
    <i class="fa fa-instagram w3-hover-opacity"></i>
    <i class="fa fa-snapchat w3-hover-opacity"></i>
    <i class="fa fa-pinterest-p w3-hover-opacity"></i>
    <i class="fa fa-twitter w3-hover-opacity"></i>
    <i class="fa fa-linkedin w3-hover-opacity"></i>
  </div>
</header>

<!-- Research Section -->
<!-- News Section -->
<div class="w3-container" style="padding:128px 16px" id="research">
  <h3 class="w3-center">RESEARCH</h3>
        <div class='w3-row'><div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>Analysis of mosaic variants</h3>
</header>
<div class="w3-container">
  <img src='imgs/mosaic.png' class='w3-center' style='width:90%;' /> 
  <p><p>Mutations in DNA accumulate during lifespan of an individual resulting in mosaic body where each cell has its private variants in the genome. This is called somatic mosaicism. Despite of its commonness, studying somatic mosaicism has been limited by lack of means to detect such variants at single-cell level. Recent advances in single cell genomics make it possible. Our group develops new computational methods for precise detection of somatic mosaic variants harnessing new experimental approaches such Next-gen sequencing. By applying the methods to human samples, we pursue to answer the questions about the origin and etiology of somatic mosaic mutations, which involves determining mutation rates during human development, the difference in numbers and patterns of mutations between tissues and between development stages, the relationships of somatic mutations to disease, aging, and healthy life.</p></b><br>
</div>
<footer class="w3-container w3-light-grey">
  <h5>Analysis of mosaic variants</h5>
</footer>
</div>
</div>


<div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>Single cell studies</h3>
</header>
<div class="w3-container">
  <img src='imgs/singlecell.jpg' class='w3-center' style='width:90%;' /> 
  <p><p>Somatic mutations start accumulating with every cell divisions as early as the zygotic stage. These mutations accumulate in the organism and could become a precursor for malignancies. Thus, study of these mutations can shed light on prognosis and possible treatment of the disease. Single cell sequencing has been shown to be better at detecting somatic mosaicism compared to bulk sequencing. However, due to the scares amount of DNA in a single cell, an amplification process is required. There are two type of amplifications, clonal expansion: where a single cell is cultured to produce a colony, which provides enough DNA for sequencing and; MDA (Multiple Displacement Amplification) where the DNA is amplified using PCR-like methods. Different amplification methods have their pros and cons and also introduces specific artifacts.</p><p>We are working on strategies for distinguishing signal from noise that might be introduced during cell culture or DNA amplification. Applying these methods, we are able to distinguish properly amplified good cells from bad cells with multiple artifacts introduced due to various reasons. We are also constantly trying to understand the origin of these biases in order to address them better.</p><p>With the advent of next generation sequencing, single cell genomics provides new perspective to the understanding of genetics by studying genomes at cellular level. Our focus being whole genome single cell sequencing, we are working on involve methods/protocols development for minimizing amplification biases introduced during MDA (Multiple Displacement Amplification) and strategies for distinguishing signal from noise that might be introduced during cell culture or DNA amplification. Applying these methods, we are able to distinguish properly amplified good cells from bad cells with multiple artifacts introduced due to various reasons. We are also constantly trying to understand the origin of these biases in order to address them better.</p></b><br>
</div>
<footer class="w3-container w3-light-grey">
  <h5>Single cell studies</h5>
</footer>
</div>
</div>


<div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>Cancer genomics</h3>
</header>
<div class="w3-container">
  <img src='imgs/polip.png' class='w3-center' style='width:90%;' /> 
  <p><p>Over the past decade, high throughput next generation technologies coupled with computational algorithms have enabled us to better understand the biology of cancer as well as the molecular underpinnings of its development and progression. Numerous functionally significant point mutations as well as structural alterations have been identified in several types and sub-types of cancers, illustrating the diverse landscape of the cancer genome. In our lab, we focus on the discovery and analysis of somatic point mutations and structural alterations including deletions, duplications and copy number changes in colon cancer and glioma. Specifically, we are interested in understanding the relationship between patterns of genetic alterations and modes of evolution of the cancer, as well as molecular differences between a cancer-free and cancer adjacent polyp.</p></b><br>
</div>
<footer class="w3-container w3-light-grey">
  <h5>Cancer genomics</h5>
</footer>
</div>
</div>


<div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>CNV and CNA analysis</h3>
</header>
<div class="w3-container">
  <img src='imgs/cnvnator.png' class='w3-center' style='width:90%;' /> 
  <p><p>Copy number variation (CNV) in the genome is a complex phenomenon, and not completely understood. Somatic copy number alterations (CNA) are frequent in cancers and have been related to cancer susceptibility, cancer progression and invasiveness, individual response to the treatment and patients’ quality life after treatment. Detection of CNVs and CNAs is important to address large spectrum of clinical and scientific questions. Research in our laboratory is focused on the discovery and analysis of genomic variants and their relevance to diseases. We have developed and constantly improving a method, CNVnator, for CNV discovery and genotyping from read-depth (RD) analysis of personal genome sequencing. Our method is based on combining the mean-shift approach used in machine learning with additional refinements. C++ implementation of CNVnator is freely available in laboratory github page</p></b><br>
</div>
<footer class="w3-container w3-light-grey">
  <h5>CNV and CNA analysis</h5>
</footer>
</div>
</div>


      </div>
</div>

<!-- News Section -->
<div class="w3-container" style="padding:128px 16px" id="news">
  <h3 class="w3-center">NEWS</h3>
        <div class='w3-row'><div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>Publication from the lab in Nucleic Acids Research</h3>
</header>
<div class="w3-container">
  <b>Chromatin organization modulates the origin of heritable structural variations in human genome</b><br>
  <p>Structural variations (SVs) in the human genome originate from different mechanisms related to DNA repair, replication errors, and retrotransposition. Our analyses of 26 927 SVs from the 1000 Genomes Project revealed differential distributions and consequences of SVs of different origin, e.g. deletions from non-allelic homologous recombination (NAHR) are more prone to disrupt chromatin organization while processed pseudogenes can create accessible chromatin. Spontaneous double stranded breaks (DSBs) are the best predictor of enrichment of NAHR deletions in open chromatin. This evidence, along with strong physical interaction of NAHR breakpoints belonging to the same deletion suggests that majority of NAHR deletions are non-meiotic i.e. originate from errors during homology directed repair (HDR) of spontaneous DSBs. In turn, the origin of the spontaneous DSBs is associated with transcription factor binding in accessible chromatin revealing the vulnerability of functional, open chromatin. The chromatin itself is enriched with repeats, particularly fixed Alu elements that provide the homology required to maintain stability via HDR. Through co-localization of fixed Alus and NAHR deletions in open chromatin we hypothesize that old Alu expansion had a stabilizing role on the human genome.</b><br>
  <b>More: </b><a href='https://doi.org/10.1093/nar/gkz103'>https://doi.org/10.1093/nar/gkz103</a><br>
  <img src='imgs/gkz103fig1.jpg' class='w3-center' style='width:90%;'>
</div>
<footer class="w3-container w3-light-grey">
  <h5>posted Feb 18, 2019, 10:33 AM by Alexej Abyzov</h5>
</footer>
</div>
</div>
<div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>Publication from the lab in Science</h3>
</header>
<div class="w3-container">
  <b>Transcriptome and epigenome landscape of human cortical development modeled in organoids</b><br>
  <p>Genes implicated in neuropsychiatric disorders are active in human fetal brain, yet difficult to study in a longitudinal fashion. We demonstrate that organoids from human pluripotent cells model cerebral cortical development on the molecular level before 16 weeks postconception. A multiomics analysis revealed differentially active genes and enhancers, with the greatest changes occurring at the transition from stem cells to progenitors. Networks of converging gene and enhancer modules were as sembled into six and four global patterns of expression and activity across time. A pattern with progressive down-regulation was enriched with human-gained enhancers, suggesting their importance in early human brain development. A few convergent gene and enhancer modules were enriched in autism-associated genes and genomic variants in autistic children. The organoid model helps identify functional elements that may drive disease onset.</b><br>
  <b>More: </b><a href='http://doi.org/10.1126/science.aat6720'>http://doi.org/10.1126/science.aat6720</a><br>
  
</div>
<footer class="w3-container w3-light-grey">
  <h5>posted Dec 13, 2018, 11:30 AM by Alexej Abyzov</h5>
</footer>
</div>
</div>
<div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>New lab member!!!</h3>
</header>
<div class="w3-container">
  <b></b><br>
  <p>Postdoctoral fellow Shobana Sekar, Ph.D. joined the lab on October 8, 2018. Welcome!</b><br>
  <br>
  
</div>
<footer class="w3-container w3-light-grey">
  <h5>posted Oct 16, 2018, 10:15 AM by Alexej Abyzov</h5>
</footer>
</div>
</div>
<div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>New lab member!!!</h3>
</header>
<div class="w3-container">
  <b></b><br>
  <p>Postdoctoral fellow Milovan Suvakov, Ph.D. joined the lab on September 4, 2018. Welcome!</b><br>
  <br>
  
</div>
<footer class="w3-container w3-light-grey">
  <h5>posted Oct 7, 2018, 9:40 PM by Alexej Abyzov</h5>
</footer>
</div>
</div>
</div>
  <center><b><u>GO TO NEWS ARCHIVE</u><b></center>
</div>

<!-- Team Section -->
<div class="w3-container" style="padding:128px 16px" id="team">
  <h3 class="w3-center">THE TEAM</h3>
  <div class="w3-row-padding w3-grayscale" style="margin-top:64px">
        <div class='w3-row'>    <div class="w3-col l3 m6 w3-padding-16">
      <div class="w3-card-4">
        <img src="imgs/Alexej.jpg" alt="Alexej" style="width:100%">
        <div class="w3-container" style="height:200px">
          <b>Alexej Abyzov, Ph.D.</b><br>
          <span class="w3-opacity">LAB HEAD</span><br>
          <span>Senior Associate Consultant, Assistant Professor of Biomedical Informatics</span>
          <p><button onclick="location.href='mailto:Abyzov.Alexej@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l3 m6 w3-padding-16">
      <div class="w3-card-4">
        <img src="imgs/Leighann.jpg" alt="Leighann" style="width:100%">
        <div class="w3-container" style="height:200px">
          <b>Leighann Wagner</b><br>
          <span class="w3-opacity">Lab Support Personnel</span><br>
          <span>Administrative Assistant</span>
          <p><button onclick="location.href='mailto:Wagner,Wagner.Leighann@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l3 m6 w3-padding-16">
      <div class="w3-card-4">
        <img src="imgs/TJ.jpg" alt="Taejeong" style="width:100%">
        <div class="w3-container" style="height:200px">
          <b>Taejeong Bae, Ph.D.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Research Associate</span>
          <p><button onclick="location.href='mailto:Bae.Taejeong@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l3 m6 w3-padding-16">
      <div class="w3-card-4">
        <img src="imgs/Vivek.jpg" alt="Vivek" style="width:100%">
        <div class="w3-container" style="height:200px">
          <b>Vivekananda Sarangi, M.S.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>PhD Student</span>
          <p><button onclick="location.href='mailto:Sarangi.Vivekananda@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l3 m6 w3-padding-16">
      <div class="w3-card-4">
        <img src="imgs/Suki.jpg" alt="Milovan" style="width:100%">
        <div class="w3-container" style="height:200px">
          <b>Milovan Suvakov, Ph.D.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Postdoctoral fellow</span>
          <p><button onclick="location.href='mailto:Suvakov.Milovan@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l3 m6 w3-padding-16">
      <div class="w3-card-4">
        <img src="imgs/Shobana.jpg" alt="Shobana" style="width:100%">
        <div class="w3-container" style="height:200px">
          <b>Shobana Sekar, Ph.D.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Postdoctoral fellow</span>
          <p><button onclick="location.href='mailto:Sekar.Shobana@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l3 m6 w3-padding-16">
      <div class="w3-card-4">
        <img src="imgs/Arijit.jpg" alt="Arijit" style="width:100%">
        <div class="w3-container" style="height:200px">
          <b>Arijit Panda, Ph.D.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Postdoctoral fellow</span>
          <p><button onclick="location.href='mailto:Panda.Arijit@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l3 m6 w3-padding-16">
      <div class="w3-card-4">
        <img src="imgs/Yeongjun.jpg" alt="Yeongjun" style="width:100%">
        <div class="w3-container" style="height:200px">
          <b>Yeongjun Jang, Ph.D.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Postdoctoral fellow</span>
          <p><button onclick="location.href='mailto:Jang.Yeongjun@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l3 m6 w3-padding-16">
      <div class="w3-card-4">
        <img src="imgs/Tanmoy.jpg" alt="Tanmoy" style="width:100%">
        <div class="w3-container" style="height:200px">
          <b>Tanmoy Roychowdhury, Ph.D.</b><br>
          <span class="w3-opacity">Affiliated Lab Member</span><br>
          <span>Research Collaborator</span>
          <p><button onclick="location.href='mailto:-';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l3 m6 w3-padding-16">
      <div class="w3-card-4">
        <img src="imgs/Chen.jpg" alt="Chen" style="width:100%">
        <div class="w3-container" style="height:200px">
          <b>Chen Wang, Ph.D.</b><br>
          <span class="w3-opacity">Affiliated Lab Member</span><br>
          <span>Senior Associate Consultant</span>
          <p><button onclick="location.href='mailto:Wang.Chen@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
</div>
  </div>
</div>


<div class="w3-container" style="padding:128px 16px" id="papers">
  <h3 class="w3-center">PAPERS</h3>
            </table>
</div>

<div class="w3-container" style="padding:128px 16px" id="tools">
  <h3 class="w3-center">TOOLS</h3>
        <div class='w3-row'><div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>CNVnator</h3>
</header>
<div class="w3-container">
  
  <div style="min-height:60px;">
  <p>A tool for CNV discovery and genotyping from depth of read mapping.</b><br>
  </div>
  <div class="github-card" data-github="abyzovlab/CNVnator" data-width="100%" data-height="153" data-theme="default"></div>
  <br>More: <a href='http://www.ncbi.nlm.nih.gov/pubmed/21293372'>ref</a>
</div>
<footer class="w3-container w3-light-grey">
  <h5>CNVnator</h5>
</footer>
</div>
</div>


<div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>CNVpytor</h3>
</header>
<div class="w3-container">
  
  <div style="min-height:60px;">
  <p>CNVpytor is a Python extension of CNVnator: package and command line tool for CNV analysis.</b><br>
  </div>
  <div class="github-card" data-github="abyzovlab/CNVpytor" data-width="100%" data-height="153" data-theme="default"></div>
  <br>More: <a href='https://github.com/abyzovlab/CNVpytor'>ref</a>
</div>
<footer class="w3-container w3-light-grey">
  <h5>CNVpytor</h5>
</footer>
</div>
</div>


<div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>AGE</h3>
</header>
<div class="w3-container">
  
  <div style="min-height:60px;">
  <p>A tool that implements an algorithm for optimal alignment of sequences with SVs.</b><br>
  </div>
  <div class="github-card" data-github="abyzovlab/AGE" data-width="100%" data-height="153" data-theme="default"></div>
  <br>More: <a href='http://www.ncbi.nlm.nih.gov/pubmed/21233167''>ref</a>
</div>
<footer class="w3-container w3-light-grey">
  <h5>AGE</h5>
</footer>
</div>
</div>


<div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>Leucippus</h3>
</header>
<div class="w3-container">
  
  <div style="min-height:60px;">
  <p>A tool for confirmation of mosaic variants from targeted sequencing.</b><br>
  </div>
  <div class="github-card" data-github="abyzovlab/Leucippus" data-width="100%" data-height="153" data-theme="default"></div>
  <br>More: <a href='https://www.ncbi.nlm.nih.gov/pubmed/28235832'>ref</a>
</div>
<footer class="w3-container w3-light-grey">
  <h5>Leucippus</h5>
</footer>
</div>
</div>


<div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>vcf2diploid</h3>
</header>
<div class="w3-container">
  
  <div style="min-height:60px;">
  <p>Personal genome constructor, it can be used to construct a personal diploid genome sequence by including personal variants into reference genome.</b><br>
  </div>
  <div class="github-card" data-github="abyzovlab/vcf2diploid" data-width="100%" data-height="153" data-theme="default"></div>
  <br>More: <a href='http://www.ncbi.nlm.nih.gov/pubmed/21811232'>ref</a>
</div>
<footer class="w3-container w3-light-grey">
  <h5>vcf2diploid</h5>
</footer>
</div>
</div>


<div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>Belka</h3>
</header>
<div class="w3-container">
  
  <div style="min-height:60px;">
  <p>A software that implements RigidFinder, a method to detect protein rigid blocks from two different conformations.<br><br>Download <a href="http://belka.molmovdb.org/Belka_v0.1.zip"><b><u>Belka v0.1</u></b></a></b><br>
  </div>
  
  <br>More: <a href='http://www.ncbi.nlm.nih.gov/pubmed/19705487'>ref</a>
</div>
<footer class="w3-container w3-light-grey">
  <h5>Belka</h5>
</footer>
</div>
</div>


</div><script src="//cdn.jsdelivr.net/github-cards/latest/widget.js"></script>

</div>

<div class="w3-container" style="padding:128px 16px" id="positions">
  <h3 class="w3-center">POSITIONS</h3>
  <div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>PostDoc in computational biology and bioinformatics</h3>
</header>
<div class="w3-container">
  <p>application accepted year-round</p>
  <hr>
  <img src="imgs/phd.png" alt="Avatar" class="w3-left w3-circle w3-padding" style="width:20%;max-width:200px">
  <p>Applicants are invited to apply for a post-doctoral (i.e., postdoc) position in Abyzov lab at Mayo Clinic. The choice of project will depend on the applicant's interests and skills, however, the research must be purely computational and focus on one of the following main fields of computational biology: population/personal human omics, cancer omics, single cell and somatic omics, and the analysis of next-generation sequencing data. Specific sub-areas of interest are discovery, annotation, and the functional annotation of human genomic variants, cancer genomics, cancer evolution, somatic mosaicism in normal human cells.</p>
  <p>The ideal applicant will have a Ph.D. in computational biology or bioinformatics, experience in one of the aforementioned research areas, demonstrate a record of peer-reviewed publications, and possess motivation for independent research. He or she should have a very strong understanding of biology and be skilled in programming and using computers to solve problems (e.g., experience with C/C++, Java, Python/Perl, R/ROOT, etc.). Oral and written proficiency in English is also a big plus.</p>
  <p>To apply, please email your CV, including a list of publications and details for three references, to abyzov dot alexej at mayo dot edu. Please include the phrase “PostDoc application” and your full name in the subject of the email.</p>
</div>
<footer class="w3-container w3-light-grey">
<br>
</footer>
</div>
</div>

<div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>Internship in computational biology and bioinformatics</h3>
</header>
<div class="w3-container">
  <p>application accepted year-round</p>
  <hr>
  <img src="imgs/img_avatar3.png" alt="Avatar" class="w3-left w3-circle w3-padding" style="width:20%;max-width:200px">
  <p>Applications are invited for an internship at the Mayo Clinic. Anticipated projects will be related to the analysis of whole genome sequencing data, with the aims of studying germline and somatic variants (SNPs, CNVs, etc.). The analysis will involve applications of commonly used, and in-house developed, software tools, and making biological hypothesis from statistical data analysis. Intern applicants with strong programming skills will have opportunities to participate in developing new tools and improving our existing software.</p>
  <p>To apply, please email your CV, including a list of publications to abyzov dot alexej at mayo dot edu. Please include the phrase “Internship application” and your full name in the subject of the email.</p>
</div>
<footer class="w3-container w3-light-grey">
<br>
</footer>
</div>
</div>

<div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>Graduate student in computational biology and bioinformatics</h3>
</header>
<div class="w3-container">
  <img src="imgs/img_avatar3.png" alt="Avatar" class="w3-left w3-circle w3-padding" style="width:20%;max-width:200px">
  <p>Graduate students (M.S or Ph.D.) wishing to conduct research in the Abyzov lab at Mayo Clinic are invited to contact Dr. Abyzov (abyzov dot alexej at mayo dot edu). The choice of the project will depend on the applicant's interests and skills. However, the research must be purely computational and focus on one of the following main fields of computational biology: population/personal human omics, cancer omics, single cell and somatic omics, and the analysis of next-generation sequencing data. Specific sub-areas of interest are discovery, annotation, and the functional annotation of human genomic variants such as SNPs, SNVs, indels, structural variations, retrotransposition, etc.</p>
  <p>We are looking for candidates that possess motivation for independent research, have experience in computational biology or bioinformatics, and are familiar with one of the aforementioned research areas. They should have a very strong understanding of biology and be skilled in programming and using computers to solve problems (e.g., experience with C/C++ Java, Python/Perl, R/ROOT, etc.). Record of peer-reviewed publications and oral and written proficiency in English is also a big plus.</p>
  <p>Dr. Abyzov is affiliated with <a href="https://college.mayo.edu">Mayo Clinic College of Medicine</a> and with the program in <a href="https://r.umn.edu/academics-research/graduate-programs/bicb">Biomedical Informatics and Computational Biology</a> at University of Minnesota Rochester.</p>
  <p>Please express your interest by emailing your CV, including a list of publications to abyzov dot alexej at mayo dot edu. Please include the phrase “PhD/MS interest” and your full name in the subject of the email.</p>
</div>
<footer class="w3-container w3-light-grey">
<br>
</footer>
</div>
</div>

</div>


<!-- Contact Section -->
<div class="w3-container" style="padding:128px 16px" id="contact">
  <div class="w3-row-padding">
    <div class="w3-col m6">
      Click on image for map:
      <a href="https://goo.gl/maps/TFaFnWVykEAG2qBDA"><img width="100%" src="imgs/map.png"></img></a>

    </div>
    <div class="w3-col m6 w3-center">
      <h3 class="w3-center">CONTACT</h3>
        Lets get in touch. Send us a message:<br><br>
        <b>Alexej Abyzov</b>, Ph.D., Principal Investigator<br>
        <p><i class="fa fa-envelope fa-fw w3-xxlarge w3-margin-right"> </i> Email: abyzov.alexej@mayo.edu</p>
        <br><br>
        <b>Leighann Wagner</b>, Administrative Assistant<br>
        <p><i class="fa fa-phone fa-fw w3-xxlarge w3-margin-right"></i> Phone: (507) 284-5569</p>
        <p><i class="fa fa-envelope fa-fw w3-xxlarge w3-margin-right"> </i> Email:
        wagner.leighann@mayo.edu</p><br><br>
      <table style="margin:auto;">
        <tr><td>
        <img src="imgs/pin.png" width="48px" /><br>
        <td> 
          Harwick Building 3rd floor, Mayo Clinic<br>
          200 1st street SW<br>
          Rochester, MN 55905, USA<br>
      </table>
    
    </div>
  </div>
</div>

<!-- Footer -->
<footer class="w3-center w3-light-grey w3-padding-32">
  <a href="#home" class="w3-button w3-dark-grey"><i class="fa fa-arrow-up w3-margin-right"></i>To the top</a><br>
  <p>Powered by <a href="https://www.w3schools.com/w3css/default.asp" title="W3.CSS" target="_blank" class="w3-hover-text-green">w3.css</a></p>
</footer>
<img id="image" width=150 src="imgs/he.jpg"></img> 

<script>
// Modal Image Gallery
function onClick(element) {
  document.getElementById("img01").src = element.src;
  document.getElementById("modal01").style.display = "block";
  var captionText = document.getElementById("caption");
  captionText.innerHTML = element.alt;
}


// Toggle between showing and hiding the sidebar when clicking the menu icon
var mySidebar = document.getElementById("mySidebar");

function w3_open() {
  if (mySidebar.style.display === 'block') {
    mySidebar.style.display = 'none';
  } else {
    mySidebar.style.display = 'block';
  }
}

// Close the sidebar with the close button
function w3_close() {
    mySidebar.style.display = "none";
}

$(document).mousemove(function(e){
    $("#image").css({left:e.pageX, top:e.pageY-$("#image").height()});
});

function showimage() {
  $("#image").show();
  $("#image").attr("src",author_image)
}

function hideimage() {
  $("#image").hide();
}

$(".author").hover(showimage,hideimage);

$('a[href^="#"]').on('click', function(event) {

    var target = $(this.getAttribute('href'));

    if( target.length ) {
        event.preventDefault();
        $('html, body').stop().animate({
            scrollTop: target.offset().top
        }, 600);
    }

});

window.onscroll = scroll;

function scroll() {
  off=70;
  maxh=160;
  st=$(document).scrollTop();
  if(st>(maxh+off)) sp=maxh;
  else if(st<off) sp=0;
  else sp=st-off;
    $(".sidebarh").css({ height: sp });
}

var slideIndex = 1;
showDivs(slideIndex);

function plusDivs(n) {
  showDivs(slideIndex += n);
}

function showDivs(n) {
  var i;
  var x = document.getElementsByClassName("mySlides");
  if (n > x.length) {slideIndex = 1}
  if (n < 1) {slideIndex = x.length}
  for (i = 0; i < x.length; i++) {
    x[i].style.display = "none";  
  }
  x[slideIndex-1].style.display = "block";  
}
</script>

</body>
</html>


