<!DOCTYPE html>
<html>
  <title>Abyzov Lab</title>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<link rel="shortcut icon" href="imgs/favicon.ico" type="image/x-icon" />
<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
<link href="https://fonts.googleapis.com/css?family=Cookie|Charm|Gentium+Basic|Gabriela|Kanit|Germania+One|Inconsolata|Raleway|Pacifico|Merriweather|Share+Tech+Mono" rel="stylesheet">

<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
<!--<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">-->

<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-MK7PXR7WZB"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-MK7PXR7WZB');
</script>

<style>
body {font-family: "Merriweather", sans-serif}
h1,h2,h3,h4,h5,h6 {font-family: "Gentium Basic", sans-serif}

.w3-top {font-family: "Merriweather", sans-serif;
  font-size:14px;
}
.logo {font-family: "Charm", sans-serif;
  font-size:32px;
  font-weight:bold;
  padding:16px;
  margin-top:4px;
}

body, html {
  height: 100%;
  line-height: 1.8;
}

#image{
  position:absolute;
  display:none
}

/* Full height image header */
.bgimg-1 {
  margin-top:50px;
  border:20px solid white;
  background-position: center;
  background-size: cover;
  background: #ffffff;
  background-image: url("imgs/group.2022.1.jpg");
  min-height: 100%;
  background-size: contain;
  background-repeat:no-repeat;
  background-position:center;
}

@media (orientation: landscape) {
  .bgimg-1 {
    background-image: url("imgs/group.2021.jpg");
    background-size: contain;
    background-position:center;
  }
}
.w3-bar {
  padding: 0px;
}
.w3-button {
  padding: 0px 16px;
}
.mySlides {display:none;}

#news-panel {
      width:75%;
      max-height: 60%;
      position: fixed;
      overflow-y:auto;
      top: 50%;
      left: 50%;
      transform: translate(-50%, -50%);
      z-index: 10;
      background-color: rgba(255,255,255,0.8);
      padding: 5px;
      border: 1px solid #999;
      text-align: left;
      padding-left: 10px;
}

.hidden { display: none !important; }

.papertable {
  border-collapse: collapse;
  width: 100%;
}

.papertable td, .papertable th {
  border: 0px solid #ddd;
  padding: 8px;
  vertical-align:top;
}

.papertable tr:nth-child(even){background-color: #fff;}

.papertable1 tr:hover {background-color: #fff;}

.papertable th {
  padding-top: 12px;
  padding-bottom: 12px;
  text-align: left;
  background-color: #4CAF50;
  color: white;
}

.paperimg {
  height:25px;
  margin-left:5px;
}

.research-div {
    -moz-column-count:2;
    -moz-column-gap: 0%;
    -webkit-column-count:2;
    -webkit-column-gap: 0%;
    column-count: 2;
    column-gap: 0%;
}

@media only screen and (max-width: 800px) {
.research-div {
    -moz-column-count:1;
    -moz-column-gap: 0%;
    -webkit-column-count:1;
    -webkit-column-gap: 0%;
    column-count: 1;
    column-gap: 0%;
}
}

.research {
   display: inline-block;
}

</style>

<body onload="init();">
<!-- Navbar (sit on top) -->
<div class="w3-top">
  <div class="w3-bar w3-white w3-card" id="topbar">
    <a href="#home" class="w3-bar-item w3-button logo">Abyzov lab</a>
    <!-- Right-sided navbar links -->
    <div class="w3-right w3-hide-medium w3-hide-small" style="margin-top:50px">
      <a href="#research" class="w3-bar-item w3-button"><i class="fas fa-flask"></i> RESEARCH</a>
      <a href="#news" class="w3-bar-item w3-button"><i class="fas fa-bullhorn"></i>NEWS</a>
      <a href="#team" class="w3-bar-item w3-button"><i class="fa fa-user"></i> TEAM</a>
      <a href="#papers" class="w3-bar-item w3-button"><i class="fas fa-file"></i>PAPERS</a>
      <a href="#tools" class="w3-bar-item w3-button"><i class="fas fa-tools"></i>TOOLS</a>
      <a href="#positions" class="w3-bar-item w3-button"><i class="fas fa-user-plus"></i>POSITIONS</a>
      <a href="#contact" class="w3-bar-item w3-button"><i class="fa fa-envelope"></i> CONTACT</a>
    </div>
    <!-- Hide right-floated links on small screens and replace them with a menu icon -->

    <a href="javascript:void(0)" class="w3-bar-item w3-button w3-right w3-hide-large" onclick="w3_open()">
      <i class="fa fa-bars"></i>
    </a>
  </div>
</div>

<!-- Sidebar on small screens when clicking the menu icon -->
<nav class="w3-sidebar w3-bar-block w3-black w3-card w3-animate-left w3-hide-large" style="display:none" id="mySidebar">
  <a href="javascript:void(0)" onclick="w3_close()" class="w3-bar-item w3-button w3-large w3-padding-16">Close ×</a>
  <a href="#research" onclick="w3_close()" class="w3-bar-item w3-button">RESEARCH</a>
  <a href="#news" onclick="w3_close()" class="w3-bar-item w3-button">NEWS</a>
  <a href="#team" onclick="w3_close()" class="w3-bar-item w3-button">TEAM</a>
  <a href="#papers" onclick="w3_close()" class="w3-bar-item w3-button">PAPERS</a>
  <a href="#tools" onclick="w3_close()" class="w3-bar-item w3-button">TOOLS</a>
  <a href="#positions" onclick="w3_close()" class="w3-bar-item w3-button">POSITIONS</a>
  <a href="#contact" onclick="w3_close()" class="w3-bar-item w3-button">CONTACT</a>
</nav>

<!-- Header with full-height image -->
<header class="bgimg-1 w3-display-container w3" id="home">
  <div class="w3-display-left w3-text-white" style="padding:48px">
  </div>
</header>

<!-- News archive -->
<div id="news-panel" class="hidden" onclick="news_archive()">
    <button style="float:right">X</button>
      <h2>News archive</h2>
      <ul><li><b>Publication from the lab in Science</b><br><i>Analysis of somatic mutations in 131 human brains reveals aging-associated hypermutability</i><br> We analyzed 131 human brains (44 neurotypical, 19 with Tourette syndrome, 9 with schizophrenia, and 59 with autism) for somatic mutations after whole genome sequencing to a depth of more than 200×. Typically, brains had 20 to 60 detectable single-nucleotide mutations, but ~6% of brains harbored hundreds of somatic mutations. Hypermutability was associated with age and damaging mutations in genes implicated in cancers and, in some brains, reflected in vivo clonal expansions. Somatic duplications, likely arising during development, were found in ~5% of normal and diseased brains, reflecting background mutagenesis. Brains with autism were associated with mutations creating putative transcription factor binding motifs in enhancer-like regions in the developing brain. The top-ranked affected motifs corresponded to MEIS (myeloid ectopic viral integration site) transcription factors, suggesting a potential link between their involvement in gene regulation and autism.<br> <b>More: </b><a href='https://www.science.org/doi/10.1126/science.abm6222'>https://www.science.org/doi/10.1126/science.abm6222</a><br> posted August 1, 2022 by TJ and Suki <br><br><li><b>Publication from the lab in PLoS Compututional Biology</b><br><i>All2: A tool for selecting mosaic mutations from comprehensive multi-cell comparisons</i><br> Accurate discovery of somatic mutations in a cell is a challenge that partially lays in immaturity of dedicated analytical approaches. Approaches comparing a cell's genome to a control bulk sample miss common mutations, while approaches to find such mutations from bulk suffer from low sensitivity. We developed a tool, All2, which enables accurate filtering of mutations in a cell without the need for data from bulk(s). It is based on pair-wise comparisons of all cells to each other where every call for base pair substitution and indel is classified as either a germline variant, mosaic mutation, or false positive. As All2 allows for considering dropped-out regions, it is applicable to whole genome and exome analysis of cloned and amplified cells. By applying the approach to a variety of available data, we showed that its application reduces false positives, enables sensitive discovery of high frequency mutations, and is indispensable for conducting high resolution cell lineage tracing.<br> <b>More: </b><a href='https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009487'>https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009487</a><br> posterd April 23, 2022 by Alexej Abyzov <br><br><li><b>Publication from the lab in GigaScience</b><br><i>CNVpytor: a tool for copy number variation detection and analysis from read depth and allele imbalance in whole-genome sequencing</i><br> Detecting copy number variations (CNVs) and copy number alterations (CNAs) based on whole-genome sequencing data is important for personalized genomics and treatment. CNVnator is one of the most popular tools for CNV/CNA discovery and analysis based on read depth. Herein, we present an extension of CNVnator developed in Python -- CNVpytor. CNVpytor inherits the reimplemented core engine of its predecessor and extends visualization, modularization, performance, and functionality. Additionally, CNVpytor uses B-allele frequency likelihood information from single-nucleotide polymorphisms and small indels data as additional evidence for CNVs/CNAs and as primary information for copy number neutral losses of heterozygosity. CNVpytor is significantly faster than CNVnator -- particularly for parsing alignment files (2-20 times faster) -- and has (20-50 times) smaller intermediate files. CNV calls can be filtered using several criteria, annotated, and merged over multiple samples. Modular architecture allows it to be used in shared and cloud environments such as Google Colab and Jupyter notebook. Data can be exported into JBrowse, while a lightweight plugin version of CNVpytor for JBrowse enables nearly instant and GUI-assisted analysis of CNVs by any user. CNVpytor release and the source code are available on GitHub at https://github.com/abyzovlab/CNVpytor under the MIT license.<br> <b>More: </b><a href='https://academic.oup.com/gigascience/article/10/11/giab074/6431715'>https://academic.oup.com/gigascience/article/10/11/giab074/6431715</a><br> posted November 29, 2021 by Alexej Abyzov <br><br><li><b>We are highlighted in Nature Reviews Genetics</b><br><i>Human cell-lineage imbalances</i><br> Various barcoding and labelling strategies have been developed for cell-lineage tracing ...<br> <b>More: </b><a href='https://www.nature.com/articles/s41576-021-00358-4'>https://www.nature.com/articles/s41576-021-00358-4</a><br> posted April 1, 2021 by Alexej Abyzov <br><br><li><b>Publication from the lab in Genome Biology</b><br><i>Comprehensive identification of somatic nucleotide variants in human brain tissue</i><br> Post-zygotic mutations incurred during DNA replication, DNA repair, and other cellular processes lead to somatic mosaicism. Somatic mosaicism is an established cause of various diseases, including cancers. However, detecting mosaic variants in DNA from non-cancerous somatic tissues poses significant challenges, particularly if the variants only are present in a small fraction of cells. Here, the Brain Somatic Mosaicism Network conducts a coordinated, multi-institutional study to examine the ability of existing methods to detect simulated somatic single-nucleotide variants (SNVs) in DNA mixing experiments, generate multiple replicates of whole-genome sequencing data from the dorsolateral prefrontal cortex, other brain regions, dura mater, and dural fibroblasts of a single neurotypical individual, devise strategies to discover somatic SNVs, and apply various approaches to validate somatic SNVs. These efforts lead to the identification of 43 bona fide somatic SNVs that range in variant allele fractions from 0.005 to 0.28. Guided by these results, we devise best practices for calling mosaic SNVs from 250× whole-genome sequencing data in the accessible portion of the human genome that achieve 90% specificity and sensitivity. Finally, we demonstrate that analysis of multiple bulk DNA samples from a single individual allows the reconstruction of early developmental cell lineage trees. This study provides a unified set of best practices to detect somatic SNVs in non-cancerous tissues. The data and methods are freely available to the scientific community and should serve as a guide to assess the contributions of somatic SNVs to neuropsychiatric diseases.<br> <b>More: </b><a href='https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02285-3'>https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02285-3</a><br> posted March 29, 2021 by Alexej Abyzov <br><br><li><b>Publication from the lab in Science</b><br><i>Early developmental asymmetries in cell lineage trees in living individuals</i><br> Mosaic mutations can be used to track cell lineages in humans. We used cell cloning to analyze embryonic cell lineages in two living individuals and a postmortem human specimen. Of 10 reconstructed postzygotic divisions, none resulted in balanced contributions of daughter lineages to tissues. In both living individuals, one of two lineages from the first cleavage was dominant across tissues, with 90% frequency in blood. We propose that the efficiency of DNA repair contributes to lineage imbalance. Allocation of lineages in postmortem brain correlated with anterior-posterior axis, associating lineage history with cell fate choices in embryos. We establish a minimally invasive framework for defining cell lineages in any living individual, which paves the way for studying their relevance in health and disease.<br> <b>More: </b><a href='https://science.sciencemag.org/content/371/6535/1245'>https://science.sciencemag.org/content/371/6535/1245</a><br> posted March 18, 2021 by Alexej Abyzov <br><br><li><b>Publication from the lab is featured by Genome Research</b><br><i>Complex mosaic structural variations in human fetal brains</i><br> Somatic mosaicism, manifesting as single nucleotide variants (SNVs), mobile element insertions, and structural changes in the DNA, is a common phenomenon in human brain cells, with potential functional consequences. Using a clonal approach, we previously detected 200-400 mosaic SNVs per cell in three human fetal brains (15-21 wk postconception). However, structural variation in the human fetal brain has not yet been investigated. Here, we discover and validate four mosaic structural variants (SVs) in the same brains and resolve their precise breakpoints. The SVs were of kilobase scale and complex, consisting of deletion(s) and rearranged genomic fragments, which sometimes originated from different chromosomes. Sequences at the breakpoints of these rearrangements had microhomologies, suggesting their origin from replication errors. One SV was found in two clones, and we timed its origin to ~14 wk postconception. No large scale mosaic copy number variants (CNVs) were detectable in normal fetal human brains, suggesting that previously reported megabase-scale CNVs in neurons arise at later stages of development. By reanalysis of public single nuclei data from adult brain neurons, we detected an extrachromosomal circular DNA event. Our study reveals the existence of mosaic SVs in the developing human brain, likely arising from cell proliferation during mid-neurogenesis. Although relatively rare compared to SNVs and present in ~10% of neurons, SVs in developing human brain affect a comparable number of bases in the genome (~6200 vs. ~4000 bp), implying that they may have similar functional consequences.<br> <b>More: </b><a href='https://genome.cshlp.org/content/30/12/1695'>https://genome.cshlp.org/content/30/12/1695</a><br> posted Dec 3, 2020 by Alexej Abyzov <br><br><li><b>Publication from the lab in BMC Bioinformatics</b><br><i>SCELLECTOR: ranking amplification bias in single cells using shallow sequencing</i><br> The study of mosaic mutation is important since it has been linked to cancer and various disorders. Single cell sequencing has become a powerful tool to study the genome of individual cells for the detection of mosaic mutations. The amount of DNA in a single cell needs to be amplified before sequencing and multiple displacement amplification (MDA) is widely used owing to its low error rate and long fragment length of amplified DNA. However, the phi29 polymerase used in MDA is sensitive to template fragmentation and presence of sites with DNA damage that can lead to biases such as allelic imbalance, uneven coverage and over representation of C to T mutations. It is therefore important to select cells with uniform amplification to decrease false positives and increase sensitivity for mosaic mutation detection. We propose a method, Scellector (single cell selector), which uses haplotype information to detect amplification quality in shallow coverage sequencing data. We tested Scellector on single human neuronal cells, obtained in vitro and amplified by MDA. Qualities were estimated from shallow sequencing with coverage as low as 0.3× per cell and then confirmed using 30× deep coverage sequencing. The high concordance between shallow and high coverage data validated the method. Scellector can potentially be used to rank amplifications obtained from single cell platforms relying on a MDA-like amplification step, such as Chromium Single Cell profiling solution.<br> <b>More: </b><a href='https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03858-y'>https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03858-y</a><br> posted Nov 13, 2020 by Alexej Abyzov <br><br><li><b>We are featured in neuroDEVELOPMENTS</b><br><i>Mosaics in mind</i><br> For this issue of neuroDEVELOPMENTS we focus on the startling reality of the mosaic nature of genomes in the human brain. Since the meeting of Craig Venter, Francis Collins, and Bill Clinton at the White House on June 6, 2000 to announce the first draft of the human genome, the idea that we all carry our own version of the human genetic code is commonplace. It is now clear that this is a simplified view of reality because every cell in our body does not have precisely the same genome ...<br> <b>More: </b><a href='https://mailchi.mp/libd/neurodevelopments-1361622'>https://mailchi.mp/libd/neurodevelopments-1361622</a><br> posted Oct 27, 2020 by Alexej Abyzov <br><br><li><b>Publication from the lab in Annual Review of Genomics and Human Genetics</b><br><i>Cell Lineage Tracing and Cellular Diversity in Humans</i><br> Tracing cell lineages is fundamental for understanding the rules governing development in multicellular organisms and delineating complex biological processes involving the differentiation of multiple cell types with distinct lineage hierarchies. In humans, experimental lineage tracing is unethical, and one has to rely on natural-mutation markers that are created within cells as they proliferate and age. Recent studies have demonstrated that it is now possible to trace lineages in normal, noncancerous cells with a variety of data types using natural variations in the nuclear and mitochondrial DNA as well as variations in DNA methylation status. It is also apparent that the scientific community is on the verge of being able to make a comprehensive and detailed cell lineage map of human embryonic and fetal development. In this review, we discuss the advantages and disadvantages of different approaches and markers for lineage tracing. We also describe the general conceptual design for how to derive a lineage map for humans.<br> <b>More: </b><a href='https://doi.org/10.1146/annurev-genom-083118-015241'>https://doi.org/10.1146/annurev-genom-083118-015241</a><br> posted Aug 12, 2020 by Alexej Abyzov <br><br><li><b>Publication from the lab in Bioinformatics</b><br><i>LongAGE: defining breakpoints of genomic structural variants through optimal and memory efficient alignments of long reads</i><br> Defining the precise location of structural variations (SVs) at single-nucleotide breakpoint resolution is a challenging problem due to large gaps in alignment. Previously, Alignment with Gap Excision (AGE) enabled us to define breakpoints of SVs at single-nucleotide resolution, however, AGE requires a vast amount of memory when aligning a pair of long sequences. To address this, we developed a memory-efficient implementation - LongAGE - based on the classical Hirschberg algorithm. We demonstrate an application of LongAGE for resolving breakpoints of SVs embedded into segmental duplications on Pacific Biosciences (PacBio) reads that can be longer than 10Kbp. Furthermore, we observed different breakpoints for a deletion and a duplication in the same locus, providing direct evidence that such multi-allelic copy number variants (mCNVs) arise from two or more independent ancestral mutations.<br> <b>More: </b><a href='https://doi.org/10.1093/bioinformatics/btaa703'>https://doi.org/10.1093/bioinformatics/btaa703</a><br> posted Aug 12, 2020 by Alexej Abyzov <br><br><li><b>New lab member!!!</b><br><i></i><br> Postdoctoral fellow Yifan Wang, Ph.D. joined the lab on February 17, 2020. Welcome!<br> <br> posted Feb 17, 2020 by Alexej Abyzov <br><br><li><b>New lab member!!!</b><br><i></i><br> Postdoctoral fellow Yeongjun Jang, Ph.D. joined the lab on September 2, 2019. Welcome!<br> <br> posted Sep 2, 2019 by Alexej Abyzov <br><br><li><b>New lab member!!!</b><br><i></i><br> Postdoctoral fellow Arijit Panda, Ph.D. joined the lab on June 10, 2019. Welcome!<br> <br> posted June 10, 2019 by Alexej Abyzov <br><br><li><b>New lab member!!!</b><br><i></i><br> Postdoctoral fellow Shobana Sekar, Ph.D. joined the lab on October 8, 2018. Welcome!<br> <br> posted Oct 16, 2018 by Alexej Abyzov <br><br><li><b>New lab member!!!</b><br><i></i><br> Postdoctoral fellow Milovan Suvakov, Ph.D. joined the lab on September 4, 2018. Welcome!<br> <br> posted Oct 7, 2018 by Alexej Abyzov <br><br><li><b>Publication from the lab in Nucleic Acids Research</b><br><i>Chromatin organization modulates the origin of heritable structural variations in human genome</i><br> Structural variations (SVs) in the human genome originate from different mechanisms related to DNA repair, replication errors, and retrotransposition. Our analyses of 26 927 SVs from the 1000 Genomes Project revealed differential distributions and consequences of SVs of different origin, e.g. deletions from non-allelic homologous recombination (NAHR) are more prone to disrupt chromatin organization while processed pseudogenes can create accessible chromatin. Spontaneous double stranded breaks (DSBs) are the best predictor of enrichment of NAHR deletions in open chromatin. This evidence, along with strong physical interaction of NAHR breakpoints belonging to the same deletion suggests that majority of NAHR deletions are non-meiotic i.e. originate from errors during homology directed repair (HDR) of spontaneous DSBs. In turn, the origin of the spontaneous DSBs is associated with transcription factor binding in accessible chromatin revealing the vulnerability of functional, open chromatin. The chromatin itself is enriched with repeats, particularly fixed Alu elements that provide the homology required to maintain stability via HDR. Through co-localization of fixed Alus and NAHR deletions in open chromatin we hypothesize that old Alu expansion had a stabilizing role on the human genome.<br> <b>More: </b><a href='https://doi.org/10.1093/nar/gkz103'>https://doi.org/10.1093/nar/gkz103</a><br> posted Feb 18, 2019 by Alexej Abyzov <br><br><li><b>Publication from the lab in Science</b><br><i>Transcriptome and epigenome landscape of human cortical development modeled in organoids</i><br> Genes implicated in neuropsychiatric disorders are active in human fetal brain, yet difficult to study in a longitudinal fashion. We demonstrate that organoids from human pluripotent cells model cerebral cortical development on the molecular level before 16 weeks postconception. A multiomics analysis revealed differentially active genes and enhancers, with the greatest changes occurring at the transition from stem cells to progenitors. Networks of converging gene and enhancer modules were as sembled into six and four global patterns of expression and activity across time. A pattern with progressive down-regulation was enriched with human-gained enhancers, suggesting their importance in early human brain development. A few convergent gene and enhancer modules were enriched in autism-associated genes and genomic variants in autistic children. The organoid model helps identify functional elements that may drive disease onset.<br> <b>More: </b><a href='http://doi.org/10.1126/science.aat6720'>http://doi.org/10.1126/science.aat6720</a><br> posted Dec 13, 2018 by Alexej Abyzov <br><br><li><b>Publication from the lab in Science</b><br><i>Different mutational rates and mechanisms in human cells at pregastrulation and neurogenesis</i><br> Somatic mosaicism in the human brain may alter function of individual neurons. We analyzed genomes of single cells from the forebrains of three human fetuses (15 to 21 weeks post-conception) using clonal cell populations. We detected 200-400 single nucleotide variations (SNVs) per cell. SNV patterns resembled those found in cancer cell genomes, indicating a role of background mutagenesis in cancer. SNVs with a frequency of >2% in brain were shared with the spleen, revealing a pregastrulation origin. We reconstructed cell lineages for the first five post-zygotic cleavages and calculated a mutation rate of ~1.3 per division per cell. Later in development, during neurogenesis, the mutation spectrum shifted toward oxidative damage and the mutation rate increased. Both neurogenesis and early embryogenesis exhibit drastically more mutagenesis than adulthood.<br> <b>More: </b><a href='http://doi.org/10.1126/science.aan8690'>http://doi.org/10.1126/science.aan8690</a><br> posted Dec 7, 2017 by Alexej Abyzov <br><br><li><b>Publication from the lab in Science</b><br><i>Intersection of diverse neuronal genomes and neuropsychiatric disease: The Brain Somatic Mosaicism Network</i><br> Neuropsychiatric disorders have a complex genetic architecture. Human genetic population-based studies have identified numerous heritable sequence and structural genomic variants associated with susceptibility to neuropsychiatric disease. However, these germline variants do not fully account for disease risk. During brain development, progenitor cells undergo billions of cell divisions to generate the ~80 billion neurons in the brain. The failure to accurately repair DNA damage arising during replication, transcription, and cellular metabolism amid this dramatic cellular expansion can lead to somatic mutations. Somatic mutations that alter subsets of neuronal transcriptomes and proteomes can, in turn, affect cell proliferation and survival and lead to neurodevelopmental disorders. The long life span of individual neurons and the direct relationship between neural circuits and behavior suggest that somatic mutations in small populations of neurons can significantly affect individual neurodevelopment. The Brain Somatic Mosaicism Network has been founded to study somatic mosaicism both in neurotypical human brains and in the context of complex neuropsychiatric disorders.<br> <b>More: </b><a href='http://science.sciencemag.org/content/356/6336/eaal1641.full'>http://science.sciencemag.org/content/356/6336/eaal1641.full</a><br> posted Apr 27, 2017 by Alexej Abyzov <br><br><li><b>Publication from the lab in Genome Research</b><br><i>One thousand somatic SNVs per skin fibroblast cell set baseline of mosaic mutational load with patterns that suggest proliferative origin</i><br> Few studies have been conducted to understand post-zygotic accumulation of mutations in cells of the healthy human body. We reprogrammed 32 skin fibroblast cells from families of donors into human induced pluripotent stem cell (hiPSC) lines. The clonal nature of hiPSC lines allows a high-resolution analysis of the genomes of the founder fibroblast cells without being confounded by the artifacts of single cell whole genome amplification. We estimate that on average a fibroblast cell in children has 1,035 mostly benign mosaic SNVs. On average, 235 SNVs could be directly confirmed in the original fibroblast population by ultra-deep sequencing, down to an allele frequency (AF) of 0.1%. More sensitive droplet digital PCR experiments confirmed more SNVs as mosaic with AF as low as 0.01%, suggesting that 1,035 mosaic SNVs per fibroblast cell is the true average. Similar analyses in adults revealed no significant increase in the number of SNVs per cell, suggesting that a major fraction of mosaic SNVs in fibroblasts arises during development. Mosaic SNVs were distributed uniformly across the genome and were enriched in a mutational signature previously observed in cancers and in de novo variants and which, we hypothesize, is a hallmark of normal cell proliferation. Finally, AF distribution of mosaic SNVs had distinct narrow peaks, which could be a characteristic of clonal cell selection, clonal expansion, or both. These findings reveal a large degree of somatic mosaicism in healthy human tissues, link de novo and cancer mutations to somatic mosaicism and couple somatic mosaicism with cell proliferation.<br> <b>More: </b><a href='http://genome.cshlp.org/content/early/2017/02/24/gr.215517.116.abstract'>http://genome.cshlp.org/content/early/2017/02/24/gr.215517.116.abstract</a><br> posted Mar 7, 2017 by Alexej Abyzov <br><br><li><b>Publication from the lab in Genome Research</b><br><i>Elevated variant density around SVs breakpoints in germline lineage lends support to error prone replication hypothesis</i><br> Copy number variants (CNVs) are a class of structural variants that may involve complex genomic rearrangements (CGRs) and that are hypothesized to have additional mutations around their breakpoints. Understanding the mechanisms underlying CNV formation is fundamental for understanding the repair and mutation mechanisms in cells, thereby shedding light on evolution, genomic disorders, cancer, and complex human traits. In this study, we employ data from the 1000 Genomes Project, to analyze hundreds of loci harboring heterozygous germline deletions in the subjects NA12878 and NA19240. By utilizing synthetic long-read data (longer than 2 kbp) in combination with high coverage short-read data and, in parallel, by comparing with parental genomes, we interrogated the phasing of these deletions with the flanking tens of thousands of heterozygous SNPs and indels. We found, that the density of SNPs/indels flanking the breakpoints of deletions (in-phase variants) is approximately twice as high as the corresponding density for the variants on the haplotype without deletion (out-of-phase variants). This fold-change was even larger, for the subset of deletions with signatures of replication-based mechanism of formation. The allele frequency (AF) spectrum for deletions is enriched for rare events<br> <b>More: </b><a href='http://genome.cshlp.org/content/early/2016/05/23/gr.205484.116.abstract'>http://genome.cshlp.org/content/early/2016/05/23/gr.205484.116.abstract</a><br> posted May 29, 2016 by Taejeong Bae <br><br><li><b>New lab member!!!</b><br><i></i><br> Postdoctoral fellow Tanmoy Roychowdhury, Ph.D. joined the lab on February 8, 2016. Welcome!<br> <br> posted Feb 17, 2016 by Alexej Abyzov <br><br><li><b>Publication from the lab in Nature</b><br><i>An integrated map of structural variation in 2,504 human genomes</i><br> Our collaborative work within the framework of the 1000 Genomes Project on discovery and analysis of genome structural variants has been published in Nature.<br><br>Structural variants are implicated in numerous diseases and make up the majority of varying nucleotides among human genomes. Here we describe an integrated set of eight structural variant classes comprising both balanced and unbalanced variants, which we constructed using short-read DNA sequencing data and statistically phased onto haplotype blocks in 26 human populations. Analysing this set, we identify numerous gene-intersecting structural variants exhibiting population stratification and describe naturally occurring homozygous gene knockouts that suggest the dispensability of a variety of human genes. We demonstrate that structural variants are enriched on haplotypes identified by genome-wide association studies and exhibit enrichment for expression quantitative trait loci. Additionally, we uncover appreciable levels of structural variant complexity at different scales, including genic loci subject to clusters of repeated rearrangement and complex structural variants with multiple breakpoints likely to have formed through individual mutational events. Our catalogue will enhance future studies into structural variant demography, functional impact and disease association.<br> <b>More: </b><a href='http://www.nature.com/nature/journal/v526/n7571/full/nature15394.html'>http://www.nature.com/nature/journal/v526/n7571/full/nature15394.html</a><br> posted Oct 1, 2015 by Alexej Abyzov <br><br>
</div>


<!-- Research Section -->
<!-- News Section -->
<div class="w3-container" style="padding:128px 16px" id="research">
  <h3 class="w3-center">RESEARCH</h3>
        <div class='research-div'>
<div class="w3-card-4 w3-margin research">
<header class="w3-container w3-light-grey">
  <h3>Analysis of mosaic variants and cell lineage reconstruction</h3>
</header>
<div class="w3-container w3-justify">
  <img src='imgs/lineage.png' class='w3-center' style='width:100%;' /> 
  <p><p>Mutations in DNA that accumulate during the lifespan of each individual result in mosaic bodies, in which each cell has unique variants in the genome. That phenomenon is called somatic mosaicism. Despite the prevalence of somatic mosaicism, studying it has been limited by the lack of means to detect such variants at the level of single cells. Recent advances in single-cell genomics, however, make such research possible. Our group develops computational methods for precisely detecting somatic mosaic variants by harnessing new experimental approaches, including clonal expansion and whole genome amplification. By applying those methods to human samples, we aim to answer questions about the origin, spread, and consequence of mosaic mutations, which involves determining mutation rates, differences in the number and pattern of mutations between tissues and ages, relevance of the mutation to diseases and aging. Additionally, we are developing scalable approaches for tracing cell lineages using mutations as lineage markers.</p></b><br>
</div>
</div>


<div class="w3-card-4 w3-margin research">
<header class="w3-container w3-light-grey">
  <h3>Single cell studies</h3>
</header>
<div class="w3-container w3-justify">
  <img src='imgs/singlecell2.jpg' class='w3-center' style='width:100%;' /> 
  <p><p>Single-cell sequencing is the ultimate way to study somatic mosaicism in healthy tissues and in cancer. However, due to the scarcity of DNA in a single cell, an amplification process is required. Such amplifications can be achieved via clonal expansion, in which a single cell is cultured to produce a colony, and via in vitro whole genome amplification (WGA), in which DNA is amplified by using polymerases. We are currently formulating strategies for the quality control of WGA and to distinguish signal from noise that may be introduced during cell culture or DNA amplification, as well as developing approaches to estimate the contributions of signal and noise when they cannot be distinguished unambiguously.</p></b><br>
</div>
</div>


<div class="w3-card-4 w3-margin research">
<header class="w3-container w3-light-grey">
  <h3>Cancer genomics</h3>
</header>
<div class="w3-container w3-justify">
  <img src='imgs/polyp2.png' class='w3-center' style='width:100%;' /> 
  <p><p>During the past decade, high-throughput next-generation technologies coupled with computational algorithms have enabled us to better understand the biology of cancer as well as the molecular underpinnings of its development and progression. Numerous functionally significant point mutations as well as structural alterations have been identified in several types and subtypes of cancers that illustrate the diverse landscape of the cancer genome. In our laboratory, we focus on the discovery and analysis of somatic point mutations and structural alterations, including deletions, duplications, and copy number changes, in colon cancer and glioma. We are especially interested in understanding the relationship between patterns of genetic alterations and modes of evolution of cancer, as well as molecular differences between cancer-free and cancer-adjacent polyps.</p></b><br>
</div>
</div>


<div class="w3-card-4 w3-margin research">
<header class="w3-container w3-light-grey">
  <h3>CNV and CNA analysis</h3>
</header>
<div class="w3-container w3-justify">
  <img src='imgs/cnvpytor.jpg' class='w3-center' style='width:100%;' /> 
  <p><p>Copy number variation (CNV) in the genome is a complex phenomenon that remains incompletely understood. Frequent in cancers, somatic copy number alterations (CNA) have been related to cancer susceptibility, cancer progression and invasiveness, individual response to the treatment, and patients’ quality of life after treatment. The detection of CNVs and CNAs is important to address a wide spectrum of clinical and scientific questions. Research in our laboratory is focused on the discovery and analysis of CNVs and CNAs along with their relevance to diseases. We have developed and continually improved a method, CNVnator/CNVpytor, for CNV discovery and genotyping from a read-depth analysis of personal genome or cancer sequencing that currently ranks among the best, most widely used methods for CNV analysis.</p></b><br>
</div>
</div>


<div class="w3-card-4 w3-margin research">
<header class="w3-container w3-light-grey">
  <h3>Variant function</h3>
</header>
<div class="w3-container w3-justify">
  <img src='imgs/function.png' class='w3-center' style='width:100%;' /> 
  <p><p>Simultaneous advances in genomics (i.e., in variant discovery), epigenomics, and functional genomics (i.e., emergence of ChiP-seq, ATAC-seq, Hi-C, and RNA-seq techniques) provide opportunities to study both the origins and consequences of genomic variants. We are interested in understanding various epigenomic properties that predispose mutational processes generating single nucleotide variation (SNV) and structural variation (SV). Inversely, germline and somatic variants affect genome function. However, because many of those variants occur in non-coding regions of the genome, their effects remain poorly understood. In response, our laboratory is actively working to elucidate such effects with a particular focus on variants contributing to neuro-developmental disorders such as autism spectrum disorders and Tourette syndrome.</p></b><br>
</div>
</div>


      </div>
</div>

<!-- News Section -->
<div class="w3-container" style="padding:128px 16px" id="news">
  <h3 class="w3-center">NEWS</h3>
        <div class='w3-row'><div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>Publication from the lab in Science</h3>
</header>
<div class="w3-container">
  <b>Analysis of somatic mutations in 131 human brains reveals aging-associated hypermutability</b><br>
  <p>We analyzed 131 human brains (44 neurotypical, 19 with Tourette syndrome, 9 with schizophrenia, and 59 with autism) for somatic mutations after whole genome sequencing to a depth of more than 200×. Typically, brains had 20 to 60 detectable single-nucleotide mutations, but ~6% of brains harbored hundreds of somatic mutations. Hypermutability was associated with age and damaging mutations in genes implicated in cancers and, in some brains, reflected in vivo clonal expansions. Somatic duplications, likely arising during development, were found in ~5% of normal and diseased brains, reflecting background mutagenesis. Brains with autism were associated with mutations creating putative transcription factor binding motifs in enhancer-like regions in the developing brain. The top-ranked affected motifs corresponded to MEIS (myeloid ectopic viral integration site) transcription factors, suggesting a potential link between their involvement in gene regulation and autism.</b><br>
  <b>More: </b><a href='https://www.science.org/doi/10.1126/science.abm6222'>https://www.science.org/doi/10.1126/science.abm6222</a><br>
  
</div>
<footer class="w3-container w3-light-grey">
  <h5>posted August 1, 2022 by TJ and Suki</h5>
</footer>
</div>
</div>
<div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>Publication from the lab in PLoS Compututional Biology</h3>
</header>
<div class="w3-container">
  <b>All2: A tool for selecting mosaic mutations from comprehensive multi-cell comparisons</b><br>
  <p>Accurate discovery of somatic mutations in a cell is a challenge that partially lays in immaturity of dedicated analytical approaches. Approaches comparing a cell's genome to a control bulk sample miss common mutations, while approaches to find such mutations from bulk suffer from low sensitivity. We developed a tool, All2, which enables accurate filtering of mutations in a cell without the need for data from bulk(s). It is based on pair-wise comparisons of all cells to each other where every call for base pair substitution and indel is classified as either a germline variant, mosaic mutation, or false positive. As All2 allows for considering dropped-out regions, it is applicable to whole genome and exome analysis of cloned and amplified cells. By applying the approach to a variety of available data, we showed that its application reduces false positives, enables sensitive discovery of high frequency mutations, and is indispensable for conducting high resolution cell lineage tracing.</b><br>
  <b>More: </b><a href='https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009487'>https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009487</a><br>
  
</div>
<footer class="w3-container w3-light-grey">
  <h5>posterd April 23, 2022 by Alexej Abyzov</h5>
</footer>
</div>
</div>
<div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>Publication from the lab in GigaScience</h3>
</header>
<div class="w3-container">
  <b>CNVpytor: a tool for copy number variation detection and analysis from read depth and allele imbalance in whole-genome sequencing</b><br>
  <p>Detecting copy number variations (CNVs) and copy number alterations (CNAs) based on whole-genome sequencing data is important for personalized genomics and treatment. CNVnator is one of the most popular tools for CNV/CNA discovery and analysis based on read depth. Herein, we present an extension of CNVnator developed in Python -- CNVpytor. CNVpytor inherits the reimplemented core engine of its predecessor and extends visualization, modularization, performance, and functionality. Additionally, CNVpytor uses B-allele frequency likelihood information from single-nucleotide polymorphisms and small indels data as additional evidence for CNVs/CNAs and as primary information for copy number neutral losses of heterozygosity. CNVpytor is significantly faster than CNVnator -- particularly for parsing alignment files (2-20 times faster) -- and has (20-50 times) smaller intermediate files. CNV calls can be filtered using several criteria, annotated, and merged over multiple samples. Modular architecture allows it to be used in shared and cloud environments such as Google Colab and Jupyter notebook. Data can be exported into JBrowse, while a lightweight plugin version of CNVpytor for JBrowse enables nearly instant and GUI-assisted analysis of CNVs by any user. CNVpytor release and the source code are available on GitHub at https://github.com/abyzovlab/CNVpytor under the MIT license.</b><br>
  <b>More: </b><a href='https://academic.oup.com/gigascience/article/10/11/giab074/6431715'>https://academic.oup.com/gigascience/article/10/11/giab074/6431715</a><br>
  
</div>
<footer class="w3-container w3-light-grey">
  <h5>posted November 29, 2021 by Alexej Abyzov</h5>
</footer>
</div>
</div>
<div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>We are highlighted in Nature Reviews Genetics</h3>
</header>
<div class="w3-container">
  <b>Human cell-lineage imbalances</b><br>
  <p>Various barcoding and labelling strategies have been developed for cell-lineage tracing ...</b><br>
  <b>More: </b><a href='https://www.nature.com/articles/s41576-021-00358-4'>https://www.nature.com/articles/s41576-021-00358-4</a><br>
  
</div>
<footer class="w3-container w3-light-grey">
  <h5>posted April 1, 2021 by Alexej Abyzov</h5>
</footer>
</div>
</div>
</div>
  <center><b><u><span onclick="news_archive()">GO TO NEWS ARCHIVE</span></u><b></center>
</div>

<!-- Team Section -->
<div class="w3-container" style="padding:128px 16px" id="team">
  <h3 class="w3-center">THE TEAM</h3>
  <div class="w3-row-padding w3" style="margin-top:64px"><div class='w3-row'>
      <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/Alexej.jpg" alt="Alexej" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Alexej Abyzov (Алексей Абызов), Ph.D.</b><br>
          <span class="w3-opacity">LAB HEAD</span><br>
          <span>Associate Professor of Biomedical Informatics</span>
          <p><button onclick="location.href='mailto:Abyzov.Alexej@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/Leighann.jpg" alt="Leighann" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Leighann Wagner</b><br>
          <span class="w3-opacity">Lab Support Personnel</span><br>
          <span>Administrative Assistant</span>
          <p><button onclick="location.href='mailto:Wagner,Wagner.Leighann@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/TJ.jpg" alt="Taejeong" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Taejeong Bae, Ph.D.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Research Associate</span>
          <p><button onclick="location.href='mailto:Bae.Taejeong@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/Suki.jpg" alt="Milovan" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Milovan Suvakov (Милован Шуваков), Ph.D.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Research Associate</span>
          <p><button onclick="location.href='mailto:Suvakov.Milovan@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/Arijit.jpg" alt="Arijit" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Arijit Panda, Ph.D.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Postdoctoral fellow</span>
          <p><button onclick="location.href='mailto:Panda.Arijit@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/Yeongjun.jpg" alt="Yeongjun" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Yeongjun Jang, Ph.D.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Postdoctoral fellow</span>
          <p><button onclick="location.href='mailto:Jang.Yeongjun@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/Yifan.jpg" alt="Yifan" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Yifan Wang, Ph.D.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Postdoctoral fellow</span>
          <p><button onclick="location.href='mailto:Yifan.Wang@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/Vivek.jpg" alt="Vivek" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Vivekananda Sarangi, M.S.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>PhD Student</span>
          <p><button onclick="location.href='mailto:Sarangi.Vivekananda@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l4 m8 w3-padding-16 w3-left">
      <div class="w3-card-4 w3-left" style="width:100%">
        <div class="w3-container" style="height:500px">
          <p class="w3-light-grey w3-block">ALUMNI:</p>
          <ul><li>Shobana Sekar, Ph.D.<li>Tanmoy Roychowdhury, Ph.D.<li>Quang Tran, B.S.<li>Logan J. Manlove, B.S.<li>Marcela Aguilera, B.S.<li>Dhananjay Dhokarh, Ph.D.<li>Nikolaos Vasmatzis<li>Minsoo Kim, B.S.<li>Justin Wood</ul>
        </div>
      </div>
    </div>

  </div></div>
</div>


<div class="w3-container" style="padding:128px 16px" id="papers">
  <h3 class="w3-center">PAPERS</h3>
        <b>2022</b><br>
      <table class='papertable'>
<tr>
<td width="40px">78.</td>
<td><b><font color=blue>Analysis of somatic mutations in 131 human brains reveals aging-associated hypermutability.</font></b><br>
<span class='author' onmouseover='author_image="imgs/TJ.jpg";'><b><u>Bae T</u></b></span>, Fasching L, <span class='author' onmouseover='author_image="imgs/Yifan.jpg";'><b><u>Wang Y</u></b></span>, Shin JH, <span class='author' onmouseover='author_image="imgs/Suki.jpg";'><b><u>Suvakov M</u></b></span>, <span class='author' onmouseover='author_image="imgs/Yeongjun.jpg";'><b><u>Jang Y</u></b></span>, Norton S, Dias C, Mariani J, Jourdon A, Wu F, <span class='author' onmouseover='author_image="imgs/Arijit.jpg";'><b><u>Panda A</u></b></span>, Pattni R, Chahine Y, Yeh R, Roberts RC, Huttner A, Kleinman JE, Hyde TM, Straub RE, Walsh CA, Urban AE, Leckman JF, Weinberger DR, Vaccarino FM, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Walsh CA, Park PJ, Sestan N, Weinberger D, Moran JV, Gage FH, Vaccarino FM, Gleeson J, Mathern G, Courchesne E, Roy S, Chess AJ, Akbarian S, Bizzotto S, Coulter M, Dias C, D'Gama A, Ganz J, Hill R, Huang AY, Khoshkhoo S, Kim S, Lee A, Lodato M, Maury EA, Miller M, Borges-Monroy R, Rodin R, Zhou Z, Bohrson C, Chu C, Cortes-Ciriano I, Dou Y, Galor A, Gulhan D, Kwon M, Luquette J, Sherman M, Viswanadham V, Jones A, Rosenbluh C, Cho S, Langmead B, Thorpe J, Erwin J, Jaffe A, McConnell M, Narurkar R, Paquola A, Shin J, Straub R, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, <span class='author' onmouseover='author_image="imgs/TJ.jpg";'><b><u>Bae T</u></b></span>, <span class='author' onmouseover='author_image="imgs/Yeongjun.jpg";'><b><u>Jang Y</u></b></span>, <span class='author' onmouseover='author_image="imgs/Yifan.jpg";'><b><u>Wang Y</u></b></span>, Molitor C, Peters M, Linker S, Reed P, Wang M, Urban A, Zhou B, Zhu X, Pattni R, Serres Amero A, Juan D, Lobon I, Marques-Bonet T, Solis Moruno M, Garcia Perez R, Povolotskaya I, Soriano E, Antaki D, Averbuj D, Ball L, Breuss M, Yang X, Chung C, Emery SB, Flasch DA, Kidd JM, Kopera HC, Kwan KY, Mills RE, Moldovan JB, Sun C, Zhao X, Zhou W, Frisbie TJ, Cherskov A, Fasching L, Jourdon A, Pochareddy S, Scuderi S<br>
<b>Science</b> 2022; 377(6605):511-517<br>
<a href='https://www.science.org/doi/10.1126/science.abm6222?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.science.org-pb-assets-images-logos-pubmed-science.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/35901164'><img class='paperimg' src='imgs/pubmed.png'/></a><a href='https://www.science.org/stoken/author-tokens/ST-648/full'><img class='paperimg' src='imgs/link.png'/></a>
</td>
</tr>
<tr>
<td width="40px">77.</td>
<td><b><font color=blue>Postmortem Human Dura Mater Cells Exhibit Phenotypic, Transcriptomic and Genetic Abnormalities that Impact their Use for Disease Modeling.</font></b><br>
Argouarch AR, Schultz N, Yang AC, <span class='author' onmouseover='author_image="imgs/Yeongjun.jpg";'><b><u>Jang Y</u></b></span>, Garcia K, Cosme CG, Corrales CI, Nana AL, Karydas AM, Spina S, Grinberg LT, Miller B, Wyss-Coray T, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Goodarzi H, Seeley WW, Kao AW<br>
<b>Stem Cell Rev Rep</b> 2022; [Epub ahead of print]<br>
<a href='https://doi.org/10.1007/s12015-022-10416-x'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--production.springer.de-OnlineResources-Logos-springerlink.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/35809166'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">76.</td>
<td><b><font color=blue>All2: A tool for selecting mosaic mutations from comprehensive multi-cell comparisons.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Vivek.jpg";'><b><u>Sarangi V</u></b></span>, <span class='author' onmouseover='author_image="imgs/Yeongjun.jpg";'><b><u>Jang Y</u></b></span>, <span class='author' onmouseover='author_image="imgs/Suki.jpg";'><b><u>Suvakov M</u></b></span>, <span class='author' onmouseover='author_image="imgs/TJ.jpg";'><b><u>Bae T</u></b></span>, Fasching L, <span class='author' onmouseover='author_image="imgs/Shobana.alumni.jpg";'><b><u>Sekar S</u></b></span>, Tomasini L, Mariani J, Vaccarino FM, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span><br>
<b>PLoS Comput Biol</b> 2022; 18(4):e1009487<br>
<a href='https://dx.plos.org/10.1371/journal.pcbi.1009487'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--journals.plos.org-resource-img-external-pcbi_120x30.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/35442945'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2021</b><br>
      <table class='papertable'>
<tr>
<td width="40px">75.</td>
<td><b><font color=blue>CNVpytor: a tool for copy number variation detection and analysis from read depth and allele imbalance in whole-genome sequencing.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Suki.jpg";'><b><u>Suvakov M</u></b></span>, <span class='author' onmouseover='author_image="imgs/Arijit.jpg";'><b><u>Panda A</u></b></span>, Diesh C, Holmes I, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span><br>
<b>Gigascience</b> 2021; 10(11):giab074<br>
<a href='https://academic.oup.com/gigascience/article-lookup/doi/10.1093/gigascience/giab074'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--academic.oup.com-images-oup_pubmed.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/34817058'><img class='paperimg' src='imgs/pubmed.png'/></a><a href='https://academic.oup.com/gigascience/article/doi/10.1093/gigascience/giab074/6431715?guestAccessKey=730624d2-17bf-405f-9093-54d317a5a7d8'><img class='paperimg' src='imgs/link.png'/></a>
</td>
</tr>
<tr>
<td width="40px">74.</td>
<td><b><font color=blue>LongAGE: defining breakpoints of genomic structural variants through optimal and memory efficient alignments of long reads.</font></b><br>
<span class='author' onmouseover='author_image="imgs/he.jpg";'><b><u>Tran Q</u></b></span>, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span><br>
<b>Bioinformatics</b> 2021; 37(7):1015-1017<br>
<a href='https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btaa703'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--academic.oup.com-images-oup_pubmed.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/32777815'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">73.</td>
<td><b><font color=blue>Comprehensive identification of somatic nucleotide variants in human brain tissue.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Yifan.jpg";'><b><u>Wang Y</u></b></span>, <span class='author' onmouseover='author_image="imgs/TJ.jpg";'><b><u>Bae T</u></b></span>, Thorpe J, Sherman MA, Jones AG, Cho S, Daily K, Dou Y, Ganz J, Galor A, Lobon I, Pattni R, Rosenbluh C, Tomasi S, Tomasini L, Yang X, Zhou B, Akbarian S, Ball LL, Bizzotto S, Emery SB, Doan R, Fasching L, <span class='author' onmouseover='author_image="imgs/Yeongjun.jpg";'><b><u>Jang Y</u></b></span>, Juan D, Lizano E, Luquette LJ, Moldovan JB, Narurkar R, Oetjens MT, Rodin RE, <span class='author' onmouseover='author_image="imgs/Shobana.alumni.jpg";'><b><u>Sekar S</u></b></span>, Shin JH, Soriano E, Straub RE, Zhou W, Chess A, Gleeson JG, Marquès-Bonet T, Park PJ, Peters MA, Pevsner J, Walsh CA, Weinberger DR, Vaccarino FM, Moran JV, Urban AE, Kidd JM, Mills RE, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span><br>
<b>Genome Biol</b> 2021; 22(1):92<br>
<a href='https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02285-3'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--s3-service-broker-live-ddda94b7-dbb0-4917-917d-776dae91ebba.s3.amazonaws.com-bmc-linkout-fulltext.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/33781308'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">72.</td>
<td><b><font color=blue>Early developmental asymmetries in cell lineage trees in living individuals.</font></b><br>
Fasching L, <span class='author' onmouseover='author_image="imgs/Yeongjun.jpg";'><b><u>Jang Y</u></b></span>, Tomasi S, Schreiner J, Tomasini L, Brady MV, <span class='author' onmouseover='author_image="imgs/TJ.jpg";'><b><u>Bae T</u></b></span>, <span class='author' onmouseover='author_image="imgs/Vivek.jpg";'><b><u>Sarangi V</u></b></span>, <span class='author' onmouseover='author_image="imgs/Niko.alumni.jpg";'><b><u>Vasmatzis N</u></b></span>, <span class='author' onmouseover='author_image="imgs/Yifan.jpg";'><b><u>Wang Y</u></b></span>, Szekely A, Fernandez TV, Leckman JF, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Vaccarino FM<br>
<b>Science</b> 2021; 371(6535):1245-1248<br>
<a href='https://www.science.org/doi/10.1126/science.abe0981?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.science.org-pb-assets-images-logos-pubmed-science.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/33737484'><img class='paperimg' src='imgs/pubmed.png'/></a><a href='https://www.science.org/stoken/author-tokens/ST-24/full'><img class='paperimg' src='imgs/link.png'/></a>
</td>
</tr>
<tr>
<td width="40px">71.</td>
<td><b><font color=blue>Landmarks of human embryonic development inscribed in somatic mutations.</font></b><br>
Bizzotto S, Dou Y, Ganz J, Doan RN, Kwon M, Bohrson CL, Kim SN, <span class='author' onmouseover='author_image="imgs/TJ.jpg";'><b><u>Bae T</u></b></span>, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Park PJ, Walsh CA<br>
<b>Science</b> 2021; 371(6535):1249-1253<br>
<a href='https://www.science.org/doi/10.1126/science.abe1544?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.science.org-pb-assets-images-logos-pubmed-science.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/33737485'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">70.</td>
<td><b><font color=blue>Machine learning reveals bilateral distribution of somatic L1 insertions in human neurons and glia.</font></b><br>
Zhu X, Zhou B, Pattni R, Gleason K, Tan C, Kalinowski A, Sloan S, Fiston-Lavier AS, Mariani J, Petrov D, Barres BA, Duncan L, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Vogel H, Moran JV, Vaccarino FM, Tamminga CA, Levinson DF, Urban AE<br>
<b>Nat Neurosci</b> 2021; 24(2):186-196<br>
<a href='https://doi.org/10.1038/s41593-020-00767-4'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.nature.com-images-npg_logo.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/33432196'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">69.</td>
<td><b><font color=blue>PsychENCODE and beyond: transcriptomics and epigenomics of brain development and organoids.</font></b><br>
Jourdon A, Scuderi S, Capauto D, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Vaccarino FM<br>
<b>Neuropsychopharmacology</b> 2021; 46(1):70-85<br>
<a href='https://doi.org/10.1038/s41386-020-0763-3'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.nature.com-images-npg_logo.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/32659782'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2020</b><br>
      <table class='papertable'>
<tr>
<td width="40px">68.</td>
<td><b><font color=blue>Complex mosaic structural variations in human fetal brains.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Shobana.alumni.jpg";'><b><u>Sekar S</u></b></span>, Tomasini L, Proukakis C, <span class='author' onmouseover='author_image="imgs/TJ.jpg";'><b><u>Bae T</u></b></span>, Manlove L, <span class='author' onmouseover='author_image="imgs/Yeongjun.jpg";'><b><u>Jang Y</u></b></span>, Scuderi S, Zhou B, Kalyva M, Amiri A, Mariani J, Sedlazeck FJ, Urban AE, Vaccarino FM, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span><br>
<b>Genome Res</b> 2020; 30(12):1695-1704<br>
<a href='http://genome.cshlp.org/cgi/pmidlookup?view=long&pmid=33122304'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--highwire.stanford.edu-icons-externalservices-pubmed-standard-genome_final_free.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/33122304'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">67.</td>
<td><b><font color=blue>The role of somatic mosaicism in brain disease.</font></b><br>
Jourdon A, Fasching L, Scuderi S, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Vaccarino FM<br>
<b>Curr Opin Genet Dev</b> 2020; 65:84-90<br>
<a href='https://linkinghub.elsevier.com/retrieve/pii/S0959-437X(20)30053-8'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--linkinghub.elsevier.com-ihub-images-PubMedLink.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/32622340'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">66.</td>
<td><b><font color=blue>Adult diffuse glioma GWAS by molecular subtype identifies variants in D2HGDH and FAM20C.</font></b><br>
Eckel-Passow JE, Drucker KL, Kollmeyer TM, Kosel ML, Decker PA, Molinaro AM, Rice T, Praska CE, Clark L, Caron A, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Batzler A, Song JS, Pekmezci M, Hansen HM, McCoy LS, Bracci PM, Wiemels J, Wiencke JK, Francis S, Burns TC, Giannini C, Lachance DH, Wrensch M, Jenkins RB<br>
<b>Neuro Oncol</b> 2020; 22(11):1602-1613<br>
<a href='https://academic.oup.com/neuro-oncology/article-lookup/doi/10.1093/neuonc/noaa117'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--academic.oup.com-images-oup_pubmed.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/32386320'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">65.</td>
<td><b><font color=blue>SCELLECTOR: ranking amplification bias in single cells using shallow sequencing.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Vivek.jpg";'><b><u>Sarangi V</u></b></span>, Jourdon A, <span class='author' onmouseover='author_image="imgs/TJ.jpg";'><b><u>Bae T</u></b></span>, <span class='author' onmouseover='author_image="imgs/Arijit.jpg";'><b><u>Panda A</u></b></span>, Vaccarino F, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span><br>
<b>BMC Bioinformatics</b> 2020; 21(1):521<br>
<a href='https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03858-y'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--s3-service-broker-live-ddda94b7-dbb0-4917-917d-776dae91ebba.s3.amazonaws.com-bmc-linkout-fulltext.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/33183232'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">64.</td>
<td><b><font color=blue>Cell Lineage Tracing and Cellular Diversity in Humans.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Vaccarino FM<br>
<b>Annu Rev Genomics Hum Genet</b> 2020; 21:101-116<br>
<a href='https://arjournals.annualreviews.org/doi/10.1146/annurev-genom-083118-015241?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--arjournals.annualreviews.org-images-AnnualReviews100x25.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/32413272'><img class='paperimg' src='imgs/pubmed.png'/></a><a href='http://www.annualreviews.org/eprint/CFKKEFS5PTNNW4XGQXKC/full/10.1146/annurev-genom-083118-015241'><img class='paperimg' src='imgs/link.png'/></a>
</td>
</tr>
<tr>
<td width="40px">63.</td>
<td><b><font color=blue>Neurological safety of oxaliplatin in patients with uncommon variants in Charcot-Marie-tooth disease genes.</font></b><br>
Le-Rademacher JG, Lopez CL, Kanwar R, Major-Elechi B, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Banck MS, Therneau TM, Sloan JA, Loprinzi CL, Beutler AS<br>
<b>J Neurol Sci</b> 2020; 411:116687<br>
<a href='https://linkinghub.elsevier.com/retrieve/pii/S0022-510X(20)30023-X'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--linkinghub.elsevier.com-ihub-images-PubMedLink.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/32018185'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">62.</td>
<td><b><font color=blue>Combining copy number, methylation markers, and mutations as a panel for endometrial cancer detection via intravaginal tampon collection.</font></b><br>
Sangtani A, Wang C, Weaver A, Hoppman NL, Kerr SE, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Shridhar V, Staub J, Kocher JA, Voss JS, Podratz KC, Wentzensen N, Kisiel JB, Sherman ME, Bakkum-Gamez JN<br>
<b>Gynecol Oncol</b> 2020; 156(2):387-392<br>
<a href='https://linkinghub.elsevier.com/retrieve/pii/S0090-8258(19)31679-8'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--linkinghub.elsevier.com-ihub-images-PubMedLink.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/31787246'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2019</b><br>
      <table class='papertable'>
<tr>
<td width="40px">61.</td>
<td><b><font color=blue>Haplotype-resolved and integrated genome analysis of the cancer cell line HepG2.</font></b><br>
Zhou B, Ho SS, Greer SU, Spies N, Bell JM, Zhang X, Zhu X, Arthur JG, Byeon S, Pattni R, Saha I, Huang Y, Song G, Perrin D, Wong WH, Ji HP, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Urban AE<br>
<b>Nucleic Acids Res</b> 2019; 47(8):3846-3861<br>
<a href='https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkz169'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--academic.oup.com-images-oup_pubmed.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/30864654'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">60.</td>
<td><b><font color=blue>Chromatin organization modulates the origin of heritable structural variations in human genome.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Tanmoy.alumni.jpg";'><b><u>Roychowdhury T</u></b></span>, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span><br>
<b>Nucleic Acids Res</b> 2019; 47(6):2766-2777<br>
<a href='https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkz103'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--academic.oup.com-images-oup_pubmed.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/30773596'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">59.</td>
<td><b><font color=blue>Comprehensive, integrated, and phased whole-genome analysis of the primary ENCODE cell line K562.</font></b><br>
Zhou B, Ho SS, Greer SU, Zhu X, Bell JM, Arthur JG, Spies N, Zhang X, Byeon S, Pattni R, Ben-Efraim N, Haney MS, Haraksingh RR, Song G, Ji HP, Perrin D, Wong WH, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Urban AE<br>
<b>Genome Res</b> 2019; 29(3):472-484<br>
<a href='http://genome.cshlp.org/cgi/pmidlookup?view=long&pmid=30737237'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--highwire.stanford.edu-icons-externalservices-pubmed-standard-genome_final_free.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/30737237'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">58.</td>
<td><b><font color=blue>Molecular signatures of multiple myeloma progression through single cell RNA-Seq.</font></b><br>
Jang JS, Li Y, Mitra AK, Bi L, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, van Wijnen AJ, Baughn LB, Van Ness B, Rajkumar V, Kumar S, Jen J<br>
<b>Blood Cancer J</b> 2019; 9(1):2<br>
<a href='https://doi.org/10.1038/s41408-018-0160-x'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.nature.com-images-npg_logo.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/30607001'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2018</b><br>
      <table class='papertable'>
<tr>
<td width="40px">57.</td>
<td><b><font color=blue>Revealing the brain's molecular architecture.</font></b><br>
<br>
<b>Science</b> 2018; 362(6420):1262-1263<br>
<a href='https://www.science.org/doi/10.1126/science.362.6420.1262?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.science.org-pb-assets-images-logos-pubmed-science.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/30545881'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">56.</td>
<td><b><font color=blue>Transcriptome and epigenome landscape of human cortical development modeled in organoids.</font></b><br>
Amiri A, Coppola G, Scuderi S, Wu F, <span class='author' onmouseover='author_image="imgs/Tanmoy.alumni.jpg";'><b><u>Roychowdhury T</u></b></span>, Liu F, Pochareddy S, Shin Y, Safi A, Song L, Zhu Y, Sousa AMM, Gerstein M, Crawford GE, Sestan N, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Vaccarino FM<br>
<b>Science</b> 2018; 362(6420):eaat6720<br>
<a href='https://www.science.org/doi/10.1126/science.aat6720?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.science.org-pb-assets-images-logos-pubmed-science.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/30545853'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">55.</td>
<td><b><font color=blue>Molecular characterization of colorectal adenomas with and without malignancy reveals distinguishing genome, transcriptome and methylome alterations.</font></b><br>
Druliner BR, Wang P, <span class='author' onmouseover='author_image="imgs/TJ.jpg";'><b><u>Bae T</u></b></span>, Baheti S, Slettedahl S, Mahoney D, <span class='author' onmouseover='author_image="imgs/Niko.alumni.jpg";'><b><u>Vasmatzis N</u></b></span>, Xu H, <span class='author' onmouseover='author_image="imgs/Minsoo.alumni.jpg";'><b><u>Kim M</u></b></span>, Bockol M, O'Brien D, Grill D, Warner N, Munoz-Gomez M, Kossick K, Johnson R, Mouchli M, Felmlee-Devine D, Washechek-Aletto J, Smyrk T, Oberg A, Wang J, Chia N, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Ahlquist D, Boardman LA<br>
<b>Sci Rep</b> 2018; 8(1):3161<br>
<a href='https://doi.org/10.1038/s41598-018-21525-4'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.nature.com-images-npg_logo.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/29453410'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">54.</td>
<td><b><font color=blue>Different mutational rates and mechanisms in human cells at pregastrulation and neurogenesis.</font></b><br>
<span class='author' onmouseover='author_image="imgs/TJ.jpg";'><b><u>Bae T</u></b></span>, Tomasini L, Mariani J, Zhou B, <span class='author' onmouseover='author_image="imgs/Tanmoy.alumni.jpg";'><b><u>Roychowdhury T</u></b></span>, Franjic D, Pletikos M, Pattni R, Chen BJ, Venturini E, Riley-Gillis B, Sestan N, Urban AE, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Vaccarino FM<br>
<b>Science</b> 2018; 359(6375):550-555<br>
<a href='https://www.science.org/doi/10.1126/science.aan8690?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.science.org-pb-assets-images-logos-pubmed-science.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/29217587'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">53.</td>
<td><b><font color=blue>Detection and Quantification of Mosaic Genomic DNA Variation in Primary Somatic Tissues Using ddPCR: Analysis of Mosaic Transposable-Element Insertions, Copy-Number Variants, and Single-Nucleotide Variants.</font></b><br>
Zhou B, Haney MS, Zhu X, Pattni R, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Urban AE<br>
<b>Methods Mol Biol</b> 2018; 1768:173-190<br>
<a href='https://dx.doi.org/10.1007/978-1-4939-7778-9_11'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--production.springer.de-OnlineResources-Logos-springerlink.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/29717444'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2017</b><br>
      <table class='papertable'>
<tr>
<td width="40px">52.</td>
<td><b><font color=blue>Inferring modes of evolution from colorectal cancer with residual polyp of origin.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Minsoo.alumni.jpg";'><b><u>Kim M</u></b></span>, Druliner BR, <span class='author' onmouseover='author_image="imgs/Niko.alumni.jpg";'><b><u>Vasmatzis N</u></b></span>, <span class='author' onmouseover='author_image="imgs/TJ.jpg";'><b><u>Bae T</u></b></span>, Chia N, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Boardman LA<br>
<b>Oncotarget</b> 2017; 9(6):6780-6792<br>
<a href='https://www.oncotarget.com/lookup/doi/10.18632/oncotarget.23687'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--www.oncotarget.com-images-misc-linkout.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/29467928'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">51.</td>
<td><b><font color=blue>Patient-reported (EORTC QLQ-CIPN20) versus physician-reported (CTCAE) quantification of oxaliplatin- and paclitaxel/carboplatin-induced peripheral neuropathy in NCCTG/Alliance clinical trials.</font></b><br>
Le-Rademacher J, Kanwar R, Seisler D, Pachman DR, Qin R, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Ruddy KJ, Banck MS, Lavoie Smith EM, Dorsey SG, Aaronson NK, Sloan J, Loprinzi CL, Beutler AS<br>
<b>Support Care Cancer</b> 2017; 25(11):3537-3544<br>
<a href='https://dx.doi.org/10.1007/s00520-017-3780-y'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--production.springer.de-OnlineResources-Logos-springerlink.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/28634656'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">50.</td>
<td><b><font color=blue>Landscape and variation of novel retroduplications in 26 human populations.</font></b><br>
Zhang Y, Li S, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Gerstein MB<br>
<b>PLoS Comput Biol</b> 2017; 13(6):e1005567<br>
<a href='https://dx.plos.org/10.1371/journal.pcbi.1005567'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--journals.plos.org-resource-img-external-pcbi_120x30.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/28662076'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">49.</td>
<td><b><font color=blue>Human induced pluripotent stem cells for modelling neurodevelopmental disorders.</font></b><br>
Ardhanareeswaran K, Mariani J, Coppola G, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Vaccarino FM<br>
<b>Nat Rev Neurol</b> 2017; 13(5):265-278<br>
<a href='https://doi.org/10.1038/nrneurol.2017.45'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.nature.com-images-npg_logo.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/28418023'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">48.</td>
<td><b><font color=blue>Intersection of diverse neuronal genomes and neuropsychiatric disease: The Brain Somatic Mosaicism Network.</font></b><br>
McConnell MJ, Moran JV, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Akbarian S, <span class='author' onmouseover='author_image="imgs/TJ.jpg";'><b><u>Bae T</u></b></span>, Cortes-Ciriano I, Erwin JA, Fasching L, Flasch DA, Freed D, Ganz J, Jaffe AE, Kwan KY, Kwon M, Lodato MA, Mills RE, Paquola ACM, Rodin RE, Rosenbluh C, Sestan N, Sherman MA, Shin JH, Song S, Straub RE, Thorpe J, Weinberger DR, Urban AE, Zhou B, Gage FH, Lehner T, Senthil G, Walsh CA, Chess A, Courchesne E, Gleeson JG, Kidd JM, Park PJ, Pevsner J, Vaccarino FM<br>
<b>Science</b> 2017; 356(6336):eaal1641<br>
<a href='https://www.science.org/doi/10.1126/science.aal1641?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.science.org-pb-assets-images-logos-pubmed-science.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/28450582'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">47.</td>
<td><b><font color=blue>Comprehensive performance comparison of high-resolution array platforms for genome-wide Copy Number Variation (CNV) analysis in humans.</font></b><br>
Haraksingh RR, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Urban AE<br>
<b>BMC Genomics</b> 2017; 18(1):321<br>
<a href='https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-3658-x'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--s3-service-broker-live-ddda94b7-dbb0-4917-917d-776dae91ebba.s3.amazonaws.com-bmc-linkout-fulltext.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/28438122'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">46.</td>
<td><b><font color=blue>One thousand somatic SNVs per skin fibroblast cell set baseline of mosaic mutational load with patterns that suggest proliferative origin.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Tomasini L, Zhou B, <span class='author' onmouseover='author_image="imgs/Niko.alumni.jpg";'><b><u>Vasmatzis N</u></b></span>, Coppola G, Amenduni M, Pattni R, Wilson M, Gerstein M, Weissman S, Urban AE, Vaccarino FM<br>
<b>Genome Res</b> 2017; 27(4):512-523<br>
<a href='http://genome.cshlp.org/cgi/pmidlookup?view=long&pmid=28235832'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--highwire.stanford.edu-icons-externalservices-pubmed-standard-genome_final_free.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/28235832'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">45.</td>
<td><b><font color=blue>Genomic Mosaicism in Neurons and Other Cell Types</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Urban AE, Vaccarino FM<br>
<b>Principles and Approaches for Discovery and Validation of Somatic Mosaicism in the Human Brain., Springer New York: Springer Nature;</b> 2017; Chapter 1.; 3-24p<br>

</td>
</tr>
      </table><br>      <b>2016</b><br>
      <table class='papertable'>
<tr>
<td width="40px">44.</td>
<td><b><font color=blue>Colorectal Cancer with Residual Polyp of Origin: A Model of Malignant Transformation.</font></b><br>
Druliner BR, Rashtak S, Ruan X, <span class='author' onmouseover='author_image="imgs/TJ.jpg";'><b><u>Bae T</u></b></span>, <span class='author' onmouseover='author_image="imgs/Niko.alumni.jpg";'><b><u>Vasmatzis N</u></b></span>, O'Brien D, Johnson R, Felmlee-Devine D, Washechek-Aletto J, Basu N, Liu H, Smyrk T, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Boardman LA<br>
<b>Transl Oncol</b> 2016; 9(4):280-6<br>
<a href='https://linkinghub.elsevier.com/retrieve/pii/S1936-5233(16)30083-3'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--linkinghub.elsevier.com-ihub-images-elsevieroa.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/27567950'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">43.</td>
<td><b><font color=blue>Elevated variant density around SV breakpoints in germline lineage lends support to error-prone replication hypothesis.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Danny.alumni.jpg";'><b><u>Dhokarh D</u></b></span>, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span><br>
<b>Genome Res</b> 2016; 26(7):874-81<br>
<a href='http://genome.cshlp.org/cgi/pmidlookup?view=long&pmid=27216746'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--highwire.stanford.edu-icons-externalservices-pubmed-standard-genome_final_free.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/27216746'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">42.</td>
<td><b><font color=blue>Single-cell analysis of targeted transcriptome predicts drug sensitivity of single cells within human myeloma tumors.</font></b><br>
Mitra AK, Mukherjee UK, Harding T, Jang JS, Stessman H, Li Y, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Jen J, Kumar S, Rajkumar V, Van Ness B<br>
<b>Leukemia</b> 2016; 30(5):1094-102<br>
<a href='https://doi.org/10.1038/leu.2015.361'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.nature.com-images-npg_logo.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/26710886'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">41.</td>
<td><b><font color=blue>A uniform survey of allele-specific binding and expression over 1000-Genomes-Project individuals.</font></b><br>
Chen J, Rozowsky J, Galeev TR, Harmanci A, Kitchen R, Bedford J, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Kong Y, Regan L, Gerstein M<br>
<b>Nat Commun</b> 2016; 7:11101<br>
<a href='https://doi.org/10.1038/ncomms11101'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.nature.com-images-npg_logo.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/27089393'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">40.</td>
<td><b><font color=blue>Testing of candidate single nucleotide variants associated with paclitaxel neuropathy in the trial NCCTG N08C1 (Alliance).</font></b><br>
Boora GK, Kanwar R, Kulkarni AA, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Sloan J, Ruddy KJ, Banck MS, Loprinzi CL, Beutler AS<br>
<b>Cancer Med</b> 2016; 5(4):631-9<br>
<a href='https://doi.org/10.1002/cam4.625'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--media.wiley.com-assets-7388-68-wiley-free-full-text.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/26763541'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">39.</td>
<td><b><font color=blue>Understanding genome structural variations.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Li S, Gerstein MB<br>
<b>Oncotarget</b> 2016; 7(7):7370-1<br>
<a href='https://www.oncotarget.com/lookup/doi/10.18632/oncotarget.6485'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--www.oncotarget.com-images-misc-linkout.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/26657727'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2015</b><br>
      <table class='papertable'>
<tr>
<td width="40px">38.</td>
<td><b><font color=blue>The PsychENCODE project.</font></b><br>
Akbarian S, Liu C, Knowles JA, Vaccarino FM, Farnham PJ, Crawford GE, Jaffe AE, Pinto D, Dracheva S, Geschwind DH, Mill J, Nairn AC, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Pochareddy S, Prabhakar S, Weissman S, Sullivan PF, State MW, Weng Z, Peters MA, White KP, Gerstein MB, Amiri A, Armoskus C, Ashley-Koch AE, <span class='author' onmouseover='author_image="imgs/TJ.jpg";'><b><u>Bae T</u></b></span>, Beckel-Mitchener A, Berman BP, Coetzee GA, Coppola G, Francoeur N, Fromer M, Gao R, Grennan K, Herstein J, Kavanagh DH, Ivanov NA, Jiang Y, Kitchen RR, Kozlenkov A, Kundakovic M, Li M, Li Z, Liu S, Mangravite LM, Mattei E, Markenscoff-Papadimitriou E, Navarro FC, North N, Omberg L, Panchision D, Parikshak N, Poschmann J, Price AJ, Purcaro M, Reddy TE, Roussos P, Schreiner S, Scuderi S, Sebra R, Shibata M, Shieh AW, Skarica M, Sun W, Swarup V, Thomas A, Tsuji J, van Bakel H, Wang D, <span class='author' onmouseover='author_image="imgs/Yifan.jpg";'><b><u>Wang Y</u></b></span>, Wang K, Werling DM, Willsey AJ, Witt H, Won H, Wong CC, Wray GA, Wu EY, Xu X, Yao L, Senthil G, Lehner T, Sklar P, Sestan N<br>
<b>Nat Neurosci</b> 2015; 18(12):1707-12<br>
<a href='https://doi.org/10.1038/nn.4156'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.nature.com-images-npg_logo.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/26605881'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">37.</td>
<td><b><font color=blue>An integrated map of structural variation in 2,504 human genomes.</font></b><br>
Sudmant PH, Rausch T, Gardner EJ, Handsaker RE, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Huddleston J, Zhang Y, Ye K, Jun G, Fritz MH, Konkel MK, Malhotra A, Stütz AM, Shi X, Casale FP, Chen J, Hormozdiari F, Dayama G, Chen K, Malig M, Chaisson MJP, Walter K, Meiers S, Kashin S, Garrison E, Auton A, Lam HYK, Mu XJ, Alkan C, Antaki D, <span class='author' onmouseover='author_image="imgs/TJ.jpg";'><b><u>Bae T</u></b></span>, Cerveira E, Chines P, Chong Z, Clarke L, Dal E, Ding L, Emery S, Fan X, Gujral M, Kahveci F, Kidd JM, Kong Y, Lameijer EW, McCarthy S, Flicek P, Gibbs RA, Marth G, Mason CE, Menelaou A, Muzny DM, Nelson BJ, Noor A, Parrish NF, Pendleton M, Quitadamo A, Raeder B, Schadt EE, Romanovitch M, Schlattl A, Sebra R, Shabalin AA, Untergasser A, Walker JA, Wang M, Yu F, Zhang C, Zhang J, Zheng-Bradley X, Zhou W, Zichner T, Sebat J, Batzer MA, McCarroll SA, Mills RE, Gerstein MB, Bashir A, Stegle O, Devine SE, Lee C, Eichler EE, Korbel JO<br>
<b>Nature</b> 2015; 526(7571):75-81<br>
<a href='https://doi.org/10.1038/nature15394'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.nature.com-images-npg_logo.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/26432246'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">36.</td>
<td><b><font color=blue>A global reference for human genetic variation.</font></b><br>
Auton A, Brooks LD, Durbin RM, Garrison EP, Kang HM, Korbel JO, Marchini JL, McCarthy S, McVean GA, Abecasis GR<br>
<b>Nature</b> 2015; 526(7571):68-74<br>
<a href='https://doi.org/10.1038/nature15393'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.nature.com-images-npg_logo.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/26432245'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">35.</td>
<td><b><font color=blue>MetaSV: an accurate and integrative structural-variant caller for next generation sequencing.</font></b><br>
Mohiyuddin M, Mu JC, Li J, Bani Asadi N, Gerstein MB, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Wong WH, Lam HY<br>
<b>Bioinformatics</b> 2015; 31(16):2741-4<br>
<a href='https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btv204'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--academic.oup.com-images-oup_pubmed.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/25861968'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">34.</td>
<td><b><font color=blue>FOXG1-Dependent Dysregulation of GABA/Glutamate Neuron Differentiation in Autism Spectrum Disorders.</font></b><br>
Mariani J, Coppola G, Zhang P, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Provini L, Tomasini L, Amenduni M, Szekely A, Palejev D, Wilson M, Gerstein M, Grigorenko EL, Chawarska K, Pelphrey KA, Howe JR, Vaccarino FM<br>
<b>Cell</b> 2015; 162(2):375-390<br>
<a href='https://linkinghub.elsevier.com/retrieve/pii/S0092-8674(15)00759-X'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--linkinghub.elsevier.com-ihub-images-celloa.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/26186191'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">33.</td>
<td><b><font color=blue>Analysis of deletion breakpoints from 1,092 humans reveals details of mutation mechanisms.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Li S, Kim DR, Mohiyuddin M, Stütz AM, Parrish NF, Mu XJ, Clark W, Chen K, Hurles M, Korbel JO, Lam HY, Lee C, Gerstein MB<br>
<b>Nat Commun</b> 2015; 6:7256<br>
<a href='https://doi.org/10.1038/ncomms8256'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.nature.com-images-npg_logo.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/26028266'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">32.</td>
<td><b><font color=blue>VarSim: a high-fidelity simulation and validation framework for high-throughput genome sequencing with cancer applications.</font></b><br>
Mu JC, Mohiyuddin M, Li J, Bani Asadi N, Gerstein MB, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Wong WH, Lam HY<br>
<b>Bioinformatics</b> 2015; 31(9):1469-71<br>
<a href='https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btu828'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--academic.oup.com-images-oup_pubmed.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/25524895'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2013</b><br>
      <table class='papertable'>
<tr>
<td width="40px">31.</td>
<td><b><font color=blue>Analysis of variable retroduplications in human populations suggests coupling of retrotransposition to cell division.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Iskow R, Gokcumen O, Radke DW, Balasubramanian S, Pei B, Habegger L, Lee C, Gerstein M<br>
<b>Genome Res</b> 2013; 23(12):2042-52<br>
<a href='http://genome.cshlp.org/cgi/pmidlookup?view=long&pmid=24026178'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--highwire.stanford.edu-icons-externalservices-pubmed-standard-genome_final_free.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/24026178'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">30.</td>
<td><b><font color=blue>Integrative annotation of variants from 1092 humans: application to cancer genomics.</font></b><br>
Khurana E, Fu Y, Colonna V, Mu XJ, Kang HM, Lappalainen T, Sboner A, Lochovsky L, Chen J, Harmanci A, Das J, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Balasubramanian S, Beal K, Chakravarty D, Challis D, Chen Y, Clarke D, Clarke L, Cunningham F, Evani US, Flicek P, Fragoza R, Garrison E, Gibbs R, Gümüş ZH, Herrero J, Kitabayashi N, Kong Y, Lage K, Liluashvili V, Lipkin SM, MacArthur DG, Marth G, Muzny D, Pers TH, Ritchie GRS, Rosenfeld JA, Sisu C, Wei X, Wilson M, Xue Y, Yu F, Dermitzakis ET, Yu H, Rubin MA, Tyler-Smith C, Gerstein M<br>
<b>Science</b> 2013; 342(6154):1235587<br>
<a href='https://www.science.org/doi/10.1126/science.1235587?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.science.org-pb-assets-images-logos-pubmed-science.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/24092746'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">29.</td>
<td><b><font color=blue>Child development and structural variation in the human genome.</font></b><br>
Zhang Y, Haraksingh R, Grubert F, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Gerstein M, Weissman S, Urban AE<br>
<b>Child Dev</b> 2013; 84(1):34-48<br>
<a href='https://doi.org/10.1111/cdev.12051'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--media.wiley.com-assets-7388-69-wiley-full-text.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/23311762'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2012</b><br>
      <table class='papertable'>
<tr>
<td width="40px">28.</td>
<td><b><font color=blue>Somatic copy number mosaicism in human skin revealed by induced pluripotent stem cells.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Mariani J, Palejev D, Zhang Y, Haney MS, Tomasini L, Ferrandino AF, Rosenberg Belmaker LA, Szekely A, Wilson M, Kocabas A, Calixto NE, Grigorenko EL, Huttner A, Chawarska K, Weissman S, Urban AE, Gerstein M, Vaccarino FM<br>
<b>Nature</b> 2012; 492(7429):438-42<br>
<a href='https://doi.org/10.1038/nature11629'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.nature.com-images-npg_logo.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/23160490'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">27.</td>
<td><b><font color=blue>An integrated map of genetic variation from 1,092 human genomes.</font></b><br>
Abecasis GR, Auton A, Brooks LD, DePristo MA, Durbin RM, Handsaker RE, Kang HM, Marth GT, McVean GA<br>
<b>Nature</b> 2012; 491(7422):56-65<br>
<a href='https://doi.org/10.1038/nature11632'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.nature.com-images-npg_logo.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/23128226'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">26.</td>
<td><b><font color=blue>Architecture of the human regulatory network derived from ENCODE data.</font></b><br>
Gerstein MB, Kundaje A, Hariharan M, Landt SG, Yan KK, Cheng C, Mu XJ, Khurana E, Rozowsky J, Alexander R, Min R, Alves P, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Addleman N, Bhardwaj N, Boyle AP, Cayting P, Charos A, Chen DZ, Cheng Y, Clarke D, Eastman C, Euskirchen G, Frietze S, Fu Y, Gertz J, Grubert F, Harmanci A, Jain P, Kasowski M, Lacroute P, Leng JJ, Lian J, Monahan H, O'Geen H, Ouyang Z, Partridge EC, Patacsil D, Pauli F, Raha D, Ramirez L, Reddy TE, Reed B, Shi M, Slifer T, Wang J, Wu L, Yang X, Yip KY, Zilberman-Schapira G, Batzoglou S, Sidow A, Farnham PJ, Myers RM, Weissman SM, Snyder M<br>
<b>Nature</b> 2012; 489(7414):91-100<br>
<a href='https://doi.org/10.1038/nature11245'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.nature.com-images-npg_logo.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/22955619'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">25.</td>
<td><b><font color=blue>An integrated encyclopedia of DNA elements in the human genome.</font></b><br>
<br>
<b>Nature</b> 2012; 489(7414):57-74<br>
<a href='https://doi.org/10.1038/nature11247'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.nature.com-images-npg_logo.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/22955616'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">24.</td>
<td><b><font color=blue>Regulatory element copy number differences shape primate expression profiles.</font></b><br>
Iskow RC, Gokcumen O, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Malukiewicz J, Zhu Q, Sukumar AT, Pai AA, Mills RE, Habegger L, Cusanovich DA, Rubel MA, Perry GH, Gerstein M, Stone AC, Gilad Y, Lee C<br>
<b>Proc Natl Acad Sci U S A</b> 2012; 109(31):12656-61<br>
<a href='https://www.pnas.org/doi/10.1073/pnas.1205199109?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--pnas.org-pb-assets-images-pubmed-pnas-pubmed-fulltext.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/22797897'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2011</b><br>
      <table class='papertable'>
<tr>
<td width="40px">23.</td>
<td><b><font color=blue>Genome-wide mapping of copy number variation in humans: comparative analysis of high resolution array platforms.</font></b><br>
Haraksingh RR, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Gerstein M, Urban AE, Snyder M<br>
<b>PLoS One</b> 2011; 6(11):e27859<br>
<a href='https://dx.plos.org/10.1371/journal.pone.0027859'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--journals.plos.org-resource-img-external-pone_120x30.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/22140474'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">22.</td>
<td><b><font color=blue>Integration of protein motions with molecular networks reveals different mechanisms for permanent and transient interactions.</font></b><br>
Bhardwaj N, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Clarke D, Shou C, Gerstein MB<br>
<b>Protein Sci</b> 2011; 20(10):1745-54<br>
<a href='https://doi.org/10.1002/pro.710'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--media.wiley.com-assets-7388-68-wiley-free-full-text.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/21826754'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">21.</td>
<td><b><font color=blue>AlleleSeq: analysis of allele-specific expression and binding in a network framework.</font></b><br>
Rozowsky J, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Wang J, Alves P, Raha D, Harmanci A, Leng J, Bjornson R, Kong Y, Kitabayashi N, Bhardwaj N, Rubin M, Snyder M, Gerstein M<br>
<b>Mol Syst Biol</b> 2011; 7:522<br>
<a href='https://doi.org/10.1038/msb.2011.54'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.embopress.org-pb-assets-embo-site-images-pubmed-msb.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/21811232'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">20.</td>
<td><b><font color=blue>Identification of genomic indels and structural variations using split reads.</font></b><br>
Zhang ZD, Du J, Lam H, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Urban AE, Snyder M, Gerstein M<br>
<b>BMC Genomics</b> 2011; 12:375<br>
<a href='https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-12-375'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--s3-service-broker-live-ddda94b7-dbb0-4917-917d-776dae91ebba.s3.amazonaws.com-bmc-linkout-fulltext.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/21787423'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">19.</td>
<td><b><font color=blue>CNVnator: an approach to discover, genotype, and characterize typical and atypical CNVs from family and population genome sequencing.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Urban AE, Snyder M, Gerstein M<br>
<b>Genome Res</b> 2011; 21(6):974-84<br>
<a href='http://genome.cshlp.org/cgi/pmidlookup?view=long&pmid=21324876'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--highwire.stanford.edu-icons-externalservices-pubmed-standard-genome_final_free.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/21324876'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">18.</td>
<td><b><font color=blue>Annual Research Review: The promise of stem cell research for neuropsychiatric disorders.</font></b><br>
Vaccarino FM, Urban AE, Stevens HE, Szekely A, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Grigorenko EL, Gerstein M, Weissman S<br>
<b>J Child Psychol Psychiatry</b> 2011; 52(4):504-16<br>
<a href='https://doi.org/10.1111/j.1469-7610.2010.02348.x'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--media.wiley.com-assets-7388-69-wiley-full-text.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/21204834'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">17.</td>
<td><b><font color=blue>AGE: defining breakpoints of genomic structural variants at single-nucleotide resolution, through optimal alignments with gap excision.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Gerstein M<br>
<b>Bioinformatics</b> 2011; 27(5):595-603<br>
<a href='https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btq713'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--academic.oup.com-images-oup_pubmed.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/21233167'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">16.</td>
<td><b><font color=blue>Mapping copy number variation by population-scale genome sequencing.</font></b><br>
Mills RE, Walter K, Stewart C, Handsaker RE, Chen K, Alkan C, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Yoon SC, Ye K, Cheetham RK, Chinwalla A, Conrad DF, Fu Y, Grubert F, Hajirasouliha I, Hormozdiari F, Iakoucheva LM, Iqbal Z, Kang S, Kidd JM, Konkel MK, Korn J, Khurana E, Kural D, Lam HY, Leng J, Li R, Li Y, Lin CY, Luo R, Mu XJ, Nemesh J, Peckham HE, Rausch T, Scally A, Shi X, Stromberg MP, Stütz AM, Urban AE, Walker JA, Wu J, Zhang Y, Zhang ZD, Batzer MA, Ding L, Marth GT, McVean G, Sebat J, Snyder M, Wang J, Ye K, Eichler EE, Gerstein MB, Hurles ME, Lee C, McCarroll SA, Korbel JO<br>
<b>Nature</b> 2011; 470(7332):59-65<br>
<a href='https://doi.org/10.1038/nature09708'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.nature.com-images-npg_logo.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/21293372'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2010</b><br>
      <table class='papertable'>
<tr>
<td width="40px">15.</td>
<td><b><font color=blue>A map of human genome variation from population-scale sequencing.</font></b><br>
Abecasis GR, Altshuler D, Auton A, Brooks LD, Durbin RM, Gibbs RA, Hurles ME, McVean GA<br>
<b>Nature</b> 2010; 467(7319):1061-73<br>
<a href='https://doi.org/10.1038/nature09534'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.nature.com-images-npg_logo.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/20981092'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">14.</td>
<td><b><font color=blue>Analysis of combinatorial regulation: scaling of partnerships between regulators with the number of governed targets.</font></b><br>
Bhardwaj N, Carson MB, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Yan KK, Lu H, Gerstein MB<br>
<b>PLoS Comput Biol</b> 2010; 6(5):e1000755<br>
<a href='https://dx.plos.org/10.1371/journal.pcbi.1000755'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--journals.plos.org-resource-img-external-pcbi_120x30.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/20523742'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">13.</td>
<td><b><font color=blue>RigidFinder: a fast and sensitive method to detect rigid blocks in large macromolecular complexes.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Bjornson R, Felipe M, Gerstein M<br>
<b>Proteins</b> 2010; 78(2):309-24<br>
<a href='https://doi.org/10.1002/prot.22544'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--media.wiley.com-assets-7388-69-wiley-full-text.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/19705487'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2009</b><br>
      <table class='papertable'>
<tr>
<td width="40px">12.</td>
<td><b><font color=blue>PEMer: a computational framework with simulation-based error models for inferring genomic structural variants from massive paired-end sequencing data.</font></b><br>
Korbel JO, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Mu XJ, Carriero N, Cayting P, Zhang Z, Snyder M, Gerstein MB<br>
<b>Genome Biol</b> 2009; 10(2):R23<br>
<a href='https://genomebiology.biomedcentral.com/articles/10.1186/gb-2009-10-2-r23'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--s3-service-broker-live-ddda94b7-dbb0-4917-917d-776dae91ebba.s3.amazonaws.com-bmc-linkout-fulltext.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/19236709'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">11.</td>
<td><b><font color=blue>MSB: a mean-shift-based approach for the analysis of structural variation in the genome.</font></b><br>
Wang LY, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Korbel JO, Snyder M, Gerstein M<br>
<b>Genome Res</b> 2009; 19(1):106-17<br>
<a href='http://genome.cshlp.org/cgi/pmidlookup?view=long&pmid=19037015'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--highwire.stanford.edu-icons-externalservices-pubmed-standard-genome_final_free.gif'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/19037015'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2008</b><br>
      <table class='papertable'>
<tr>
<td width="40px">10.</td>
<td><b><font color=blue>An AP endonuclease 1-DNA polymerase beta complex: theoretical prediction of interacting surfaces.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Uzun A, Strauss PR, Ilyin VA<br>
<b>PLoS Comput Biol</b> 2008; 4(4):e1000066<br>
<a href='https://dx.plos.org/10.1371/journal.pcbi.1000066'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--journals.plos.org-resource-img-external-pcbi_120x30.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/18437203'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2007</b><br>
      <table class='papertable'>
<tr>
<td width="40px">9.</td>
<td><b><font color=blue>UmuD and RecA directly modulate the mutagenic potential of the Y family DNA polymerase DinB.</font></b><br>
Godoy VG, Jarosz DF, Simon SM, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Ilyin V, Walker GC<br>
<b>Mol Cell</b> 2007; 28(6):1058-70<br>
<a href='https://linkinghub.elsevier.com/retrieve/pii/S1097-2765(07)00732-0'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--linkinghub.elsevier.com-ihub-images-celloa.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/18158902'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">8.</td>
<td><b><font color=blue>A comprehensive analysis of non-sequential alignments between all protein structures.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Ilyin VA<br>
<b>BMC Struct Biol</b> 2007; 7:78<br>
<a href='https://bmcstructbiol.biomedcentral.com/articles/10.1186/1472-6807-7-78'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--s3-service-broker-live-ddda94b7-dbb0-4917-917d-776dae91ebba.s3.amazonaws.com-bmc-linkout-fulltext.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/18005453'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">7.</td>
<td><b><font color=blue>Structure SNP (StSNP): a web server for mapping and modeling nsSNPs on protein structures with linkage to metabolic pathways.</font></b><br>
Uzun A, Leslin CM, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Ilyin V<br>
<b>Nucleic Acids Res</b> 2007; 35(Web Server issue):W384-92<br>
<a href='https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkm232'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--academic.oup.com-images-oup_pubmed.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/17537826'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">6.</td>
<td><b><font color=blue>TOPOFIT-DB, a database of protein structural alignments based on the TOPOFIT method.</font></b><br>
Leslin CM, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Ilyin VA<br>
<b>Nucleic Acids Res</b> 2007; 35(Database issue):D317-21<br>
<a href='https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkl809'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--academic.oup.com-images-oup_pubmed.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/17065464'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2005</b><br>
      <table class='papertable'>
<tr>
<td width="40px">5.</td>
<td><b><font color=blue>Friend, an integrated analytical front-end application for bioinformatics.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Errami M, Leslin CM, Ilyin VA<br>
<b>Bioinformatics</b> 2005; 21(18):3677-8<br>
<a href='https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/bti602'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--academic.oup.com-images-oup_pubmed.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/16076889'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">4.</td>
<td><b><font color=blue>Active site prediction for comparative model structures with thematics.</font></b><br>
Shehadi IA, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Uzun A, Wei Y, Murga LF, Ilyin V, Ondrechen MJ<br>
<b>J Bioinform Comput Biol</b> 2005; 3(1):127-43<br>
<a href='https://www.worldscientific.com/doi/10.1142/s0219720005000916?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.worldscientific.com-pb-assets-wspc-site-wsnetlogo.jpg'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/15751116'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2004</b><br>
      <table class='papertable'>
<tr>
<td width="40px">3.</td>
<td><b><font color=blue>Structural exon database, SEDB, mapping exon boundaries on multiple protein structures.</font></b><br>
Leslin CM, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Ilyin VA<br>
<b>Bioinformatics</b> 2004; 20(11):1801-3<br>
<a href='https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/bth150'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--academic.oup.com-images-oup_pubmed.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/14988102'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">2.</td>
<td><b><font color=blue>Structural alignment of proteins by a novel TOPOFIT method, as a superimposition of common volumes at a topomax point.</font></b><br>
Ilyin VA, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Leslin CM<br>
<b>Protein Sci</b> 2004; 13(7):1865-74<br>
<a href='https://doi.org/10.1110/ps.04672604'><img class='paperimg' src='https://cdn.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--media.wiley.com-assets-7388-68-wiley-free-full-text.png'/></a><a href='https://pubmed.ncbi.nlm.nih.gov/15215530'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2002</b><br>
      <table class='papertable'>
<tr>
<td width="40px">1.</td>
<td><b><font color=blue>Efficiency Profile Method To Study The Hit Efficiency Of Drift Chambers.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Bel'kov A, Lanyov A, Spiridonov A, Walter M, Hulsbergen W<br>
<b>Particles and Nuclei, Letters</b> 2002; 5 (114); :40-52<br>

</td>
</tr>
      </table>
</div>

<div class="w3-container" style="padding:128px 16px" id="tools">
  <h3 class="w3-center">TOOLS</h3>
        <div class='w3-row'><div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/CNVpytor" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
<div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/CNVnator" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
<div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/All2" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
<div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/SCellector" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
<div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/AGE" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
<div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/Leucippus" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
<div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/vcf2diploid" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
<div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/Belka" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
</div><script src="//cdn.jsdelivr.net/github-cards/latest/widget.js"></script>

</div>

<div class="w3-container" style="padding:128px 16px" id="positions">
  <h3 class="w3-center">POSITIONS</h3>
  <div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>PostDoc in computational biology and bioinformatics</h3>
</header>
<div class="w3-container">
  <p>application accepted year-round</p>
  <hr>
  <img src="imgs/phd.png" alt="Avatar" class="w3-left w3-circle w3-padding" style="width:20%;max-width:200px">
  <p>Applicants are invited to apply for a post-doctoral (i.e., postdoc) position in Abyzov lab at Mayo Clinic. The choice of project will depend on the applicant's interests and skills, however, the research must be purely computational and focus on one of the following main fields of computational biology: population/personal human omics, cancer omics, single cell and somatic omics, and the analysis of next-generation sequencing data. Specific sub-areas of interest are discovery, annotation, and the functional annotation of human genomic variants, cancer genomics, cancer evolution, somatic mosaicism in normal human cells.</p>
  <p>The ideal applicant will have a Ph.D. in computational biology or bioinformatics, experience in one of the aforementioned research areas, demonstrate a record of peer-reviewed publications, and possess motivation for independent research. He or she should have a very strong understanding of biology and be skilled in programming and using computers to solve problems (e.g., experience with C/C++, Java, Python/Perl, R/ROOT, etc.). Oral and written proficiency in English is also a big plus.</p>
  <p>To apply, please email your CV, including a list of publications and details for three references, to abyzov dot alexej at mayo dot edu. Please include the phrase “PostDoc application” and your full name in the subject of the email.</p>
</div>
<footer class="w3-container w3-light-grey">
<br>
</footer>
</div>
</div>

<div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>Internship in computational biology and bioinformatics</h3>
</header>
<div class="w3-container">
  <p>application accepted year-round</p>
  <hr>
  <img src="imgs/img_avatar3.png" alt="Avatar" class="w3-left w3-circle w3-padding" style="width:20%;max-width:200px">
  <p>Applications are invited for an internship at the Mayo Clinic. Anticipated projects will be related to the analysis of whole genome sequencing data, with the aims of studying germline and somatic variants (SNPs, CNVs, etc.). The analysis will involve applications of commonly used, and in-house developed, software tools, and making biological hypothesis from statistical data analysis. Intern applicants with strong programming skills will have opportunities to participate in developing new tools and improving our existing software.</p>
  <p>To apply, please email your CV, including a list of publications to abyzov dot alexej at mayo dot edu. Please include the phrase “Internship application” and your full name in the subject of the email.</p>
</div>
<footer class="w3-container w3-light-grey">
<br>
</footer>
</div>
</div>

<div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>Graduate student in computational biology and bioinformatics</h3>
</header>
<div class="w3-container">
  <img src="imgs/img_avatar3.png" alt="Avatar" class="w3-left w3-circle w3-padding" style="width:20%;max-width:200px">
  <p>Graduate students (M.S or Ph.D.) wishing to conduct research in the Abyzov lab at Mayo Clinic are invited to contact Dr. Abyzov (abyzov dot alexej at mayo dot edu). The choice of the project will depend on the applicant's interests and skills. However, the research must be purely computational and focus on one of the following main fields of computational biology: population/personal human omics, cancer omics, single cell and somatic omics, and the analysis of next-generation sequencing data. Specific sub-areas of interest are discovery, annotation, and the functional annotation of human genomic variants such as SNPs, SNVs, indels, structural variations, retrotransposition, etc.</p>
  <p>We are looking for candidates that possess motivation for independent research, have experience in computational biology or bioinformatics, and are familiar with one of the aforementioned research areas. They should have a very strong understanding of biology and be skilled in programming and using computers to solve problems (e.g., experience with C/C++ Java, Python/Perl, R/ROOT, etc.). Record of peer-reviewed publications and oral and written proficiency in English is also a big plus.</p>
  <p>Dr. Abyzov is affiliated with <a href="https://college.mayo.edu">Mayo Clinic College of Medicine</a> and with the program in <a href="https://r.umn.edu/academics-research/graduate-programs/bicb">Biomedical Informatics and Computational Biology</a> at University of Minnesota Rochester.</p>
  <p>Please express your interest by emailing your CV, including a list of publications to abyzov dot alexej at mayo dot edu. Please include the phrase “PhD/MS interest” and your full name in the subject of the email.</p>
</div>
<footer class="w3-container w3-light-grey">
<br>
</footer>
</div>
</div>

</div>


<!-- Contact Section -->
<div class="w3-container" style="padding:128px 16px" id="contact">
  <div class="w3-row-padding">
    <div class="w3-col m6">
      Click on image for map:
      <a href="https://goo.gl/maps/TFaFnWVykEAG2qBDA"><img width="100%" src="imgs/map.png"></img></a>

    </div>
    <div class="w3-col m6 w3-center">
      <h3 class="w3-center">CONTACT</h3>
        Lets get in touch. Send us a message:<br><br>
        <b>Alexej Abyzov</b>, Ph.D., Principal Investigator<br>
        <p><i class="fa fa-envelope fa-fw w3-xxlarge w3-margin-right"> </i> Email: abyzov.alexej@mayo.edu</p>
        <br><br>
        <b>Leighann Wagner</b>, Administrative Assistant<br>
        <p><i class="fa fa-phone fa-fw w3-xxlarge w3-margin-right"></i> Phone: (507) 284-5569</p>
        <p><i class="fa fa-envelope fa-fw w3-xxlarge w3-margin-right"> </i> Email:
        wagner.leighann@mayo.edu</p><br><br>
      <table style="margin:auto;">
        <tr><td>
        <img src="imgs/pin.png" width="48px" /><br>
        <td> 
          Harwick Building 3rd floor, Mayo Clinic<br>
          200 1st street SW<br>
          Rochester, MN 55905, USA<br>
      </table>
    
    </div>
  </div>
</div>

<!-- Footer -->
<footer class="w3-center w3-light-grey w3-padding-32">
  <a href="#home" class="w3-button w3-dark-grey"><i class="fa fa-arrow-up w3-margin-right"></i>To the top</a><br>
  <p>Powered by <a href="https://www.w3schools.com/w3css/default.asp" title="W3.CSS" target="_blank" class="w3-hover-text-green">w3.css</a></p>
</footer>
<img id="image" width=150 src="imgs/he.jpg"></img> 

<script>

function news_archive() {
  document.getElementById('news-panel').classList.toggle('hidden');
}

// Modal Image Gallery
function onClick(element) {
  document.getElementById("img01").src = element.src;
  document.getElementById("modal01").style.display = "block";
  var captionText = document.getElementById("caption");
  captionText.innerHTML = element.alt;
}


// Toggle between showing and hiding the sidebar when clicking the menu icon
var mySidebar = document.getElementById("mySidebar");

function w3_open() {
  if (mySidebar.style.display === 'block') {
    mySidebar.style.display = 'none';
  } else {
    mySidebar.style.display = 'block';
  }
}

// Close the sidebar with the close button
function w3_close() {
    mySidebar.style.display = "none";
}

$(document).mousemove(function(e){
    $("#image").css({left:e.pageX, top:e.pageY-$("#image").height()});
});

function showimage() {
  $("#image").show();
  $("#image").attr("src",author_image)
}

function hideimage() {
  $("#image").hide();
}

$(".author").hover(showimage,hideimage);

$('a[href^="#"]').on('click', function(event) {

    var target = $(this.getAttribute('href'));

    if( target.length ) {
        event.preventDefault();
        $('html, body').stop().animate({
            scrollTop: target.offset().top
        }, 600);
    }

});

window.onscroll = scroll;

function scroll() {
  off=70;
  maxh=160;
  st=$(document).scrollTop();
  if(st>(maxh+off)) sp=maxh;
  else if(st<off) sp=0;
  else sp=st-off;
    $(".sidebarh").css({ height: sp });
}

var slideIndex = 1;
showDivs(slideIndex);

function plusDivs(n) {
  showDivs(slideIndex += n);
}

function showDivs(n) {
  var i;
  var x = document.getElementsByClassName("mySlides");
  if (n > x.length) {slideIndex = 1}
  if (n < 1) {slideIndex = x.length}
  for (i = 0; i < x.length; i++) {
    x[i].style.display = "none";  
  }
  x[slideIndex-1].style.display = "block";  
}


var imageIx
var images

function init() {
  imageIx = 0
  images = ['imgs/group.2022.1.jpg', 'imgs/group.2022.2.jpg', 'imgs/group.2021.jpg', 'imgs/group.2020.jpg', 'imgs/group.2019.jpg', 'imgs/group.2018.jpg', 'imgs/group.2018.19.jpg', 'imgs/group.2018.20.jpg', 'imgs/group.2016.jpg', 'imgs/group.2016.03.jpg', 'imgs/group.2015.27.jpg', 'imgs/group.2015.28.jpg']
  setInterval(changeImage,5000)
}

function changeImage() {
  console.log(images)
  var x = document.getElementsByClassName("bgimg-1")[0];
  imageIx += 1
  imageIx %= images.length  
  x.style.backgroundImage = "url('"+images[imageIx]+"')";
}

</script>

</body>
</html>


