<!DOCTYPE html>
<html>
  <title>Abyzov Lab</title>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<link rel="shortcut icon" href="imgs/favicon.ico" type="image/x-icon" />
<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
<link href="https://fonts.googleapis.com/css?family=Cookie|Charm|Gentium+Basic|Gabriela|Kanit|Germania+One|Inconsolata|Raleway|Pacifico|Merriweather|Share+Tech+Mono" rel="stylesheet">

<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
<!--<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">-->

<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-MK7PXR7WZB"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-MK7PXR7WZB');
</script>

<style>
body {font-family: "Merriweather", sans-serif}
h1,h2,h3,h4,h5,h6 {font-family: "Gentium Basic", sans-serif}

.w3-top {font-family: "Merriweather", sans-serif;
  font-size:14px;
}
.logo {font-family: "Charm", sans-serif;
  font-size:32px;
  font-weight:bold;
  padding:16px;
  margin-top:4px;
}

body, html {
  height: 100%;
  line-height: 1.8;
}

#image{
  position:absolute;
  display:none
}

/* Full height image header */
.bgimg-1 {
  margin-top:50px;
  border:20px solid white;
  background-position: center;
  background-size: cover;
  background: #ffffff;
  background-image: url("imgs/group.2023.1.jpg");
  min-height: 100%;
  background-size: contain;
  background-repeat:no-repeat;
  background-position:center;
}

@media (orientation: landscape) {
  .bgimg-1 {
    background-image: url("imgs/group.2024.jpg");
    background-size: contain;
    background-position:center;
  }
}
.w3-bar {
  padding: 0px;
}
.w3-button {
  padding: 0px 16px;
}
.mySlides {display:none;}

#news-panel {
      width:75%;
      max-height: 60%;
      position: fixed;
      overflow-y:auto;
      top: 50%;
      left: 50%;
      transform: translate(-50%, -50%);
      z-index: 10;
      background-color: rgba(255,255,255,0.8);
      padding: 5px;
      border: 1px solid #999;
      text-align: left;
      padding-left: 10px;
}

.hidden { display: none !important; }

.papertable {
  border-collapse: collapse;
  width: 100%;
}

.papertable td, .papertable th {
  border: 0px solid #ddd;
  padding: 8px;
  vertical-align:top;
}

.papertable tr:nth-child(even){background-color: #fff;}

.papertable1 tr:hover {background-color: #fff;}

.papertable th {
  padding-top: 12px;
  padding-bottom: 12px;
  text-align: left;
  background-color: #4CAF50;
  color: white;
}

.paperimg {
  height:25px;
  margin-left:5px;
}

.research-div {
    -moz-column-count:2;
    -moz-column-gap: 0%;
    -webkit-column-count:2;
    -webkit-column-gap: 0%;
    column-count: 2;
    column-gap: 0%;
}

@media only screen and (max-width: 800px) {
.research-div {
    -moz-column-count:1;
    -moz-column-gap: 0%;
    -webkit-column-count:1;
    -webkit-column-gap: 0%;
    column-count: 1;
    column-gap: 0%;
}
}

.research {
   display: inline-block;
}

</style>

<body onload="init();">
<!-- Navbar (sit on top) -->
<div class="w3-top">
  <div class="w3-bar w3-white w3-card" id="topbar">
    <a href="#home" class="w3-bar-item w3-button logo">Abyzov lab</a>
    <!-- Right-sided navbar links -->
    <div class="w3-right w3-hide-medium w3-hide-small" style="margin-top:50px">
      <a href="#research" class="w3-bar-item w3-button"><i class="fas fa-flask"></i> RESEARCH</a>
      <a href="#news" class="w3-bar-item w3-button"><i class="fas fa-bullhorn"></i>NEWS</a>
      <a href="#team" class="w3-bar-item w3-button"><i class="fa fa-user"></i> TEAM</a>
      <a href="#papers" class="w3-bar-item w3-button"><i class="fas fa-file"></i>PAPERS</a>
      <a href="#tools" class="w3-bar-item w3-button"><i class="fas fa-tools"></i>TOOLS</a>
      <a href="#positions" class="w3-bar-item w3-button"><i class="fas fa-user-plus"></i>POSITIONS</a>
      <a href="#contact" class="w3-bar-item w3-button"><i class="fa fa-envelope"></i> CONTACT</a>
    </div>
    <!-- Hide right-floated links on small screens and replace them with a menu icon -->

    <a href="javascript:void(0)" class="w3-bar-item w3-button w3-right w3-hide-large" onclick="w3_open()">
      <i class="fa fa-bars"></i>
    </a>
  </div>
</div>

<!-- Sidebar on small screens when clicking the menu icon -->
<nav class="w3-sidebar w3-bar-block w3-black w3-card w3-animate-left w3-hide-large" style="display:none" id="mySidebar">
  <a href="javascript:void(0)" onclick="w3_close()" class="w3-bar-item w3-button w3-large w3-padding-16">Close ×</a>
  <a href="#research" onclick="w3_close()" class="w3-bar-item w3-button">RESEARCH</a>
  <a href="#news" onclick="w3_close()" class="w3-bar-item w3-button">NEWS</a>
  <a href="#team" onclick="w3_close()" class="w3-bar-item w3-button">TEAM</a>
  <a href="#papers" onclick="w3_close()" class="w3-bar-item w3-button">PAPERS</a>
  <a href="#tools" onclick="w3_close()" class="w3-bar-item w3-button">TOOLS</a>
  <a href="#positions" onclick="w3_close()" class="w3-bar-item w3-button">POSITIONS</a>
  <a href="#contact" onclick="w3_close()" class="w3-bar-item w3-button">CONTACT</a>
</nav>

<!-- Header with full-height image -->
<header class="bgimg-1 w3-display-container w3" id="home">
  <div class="w3-display-left w3-text-white" style="padding:48px">
  </div>
</header>

<!-- News archive -->
<div id="news-panel" class="hidden" onclick="news_archive()">
    <button style="float:right">X</button>
      <h2>News archive</h2>
      <ul><li><b>Publication from the lab in Biological Psychiatry</b><br><i>Interneuron loss and microglia activation by transcriptome analyses in the basal ganglia of Tourette disorder</i><br> Tourette disorder is characterized by motor hyperactivity and tics that are believed to originate in basal ganglia. Postmortem immunocytochemical analyses previously revealed decreases in cholinergic, parvalbumin, and somatostatin interneurons (IN) within the caudate/putamen of individuals with TS. We obtained transcriptome and open chromatin datasets by snRNAseq and snATAC-seq, respectively, from caudate/putamen postmortem specimens of 6 adult TS and 6 matched normal control (NC). Differential gene expression and differential chromatin accessibility analyses were performed in identified cell types.The data reproduced the known cellular composition of the human striatum, including a majority of medium spiny neurons (MSN) and small populations of GABAergic and cholinergic IN. IN were decreased by ∼50% in TS brains, with no difference in other cell types. Differential gene expression analysis suggested that mitochondrial oxidative metabolism in MSN and synaptic adhesion and function in IN were both decreased in TS subjects, while there was activation of immune response in microglia. Gene expression changes correlated with changes in activity of cis-regulatory elements, suggesting a relationship of transcriptomic and regulatory abnormalities in MSN, OL and AST of TS brains. This initial analysis of the TS basal ganglia transcriptome at the single cell level confirms the loss and synaptic dysfunction of basal ganglia IN, consistent with in vivo basal ganglia hyperactivity. In parallel, oxidative metabolism was decreased in MSN and correlated with activation of microglia cells, attributable at least in part to dysregulated activity of putative enhancers, implicating altered epigenomic regulation in TS.<br> <b>More: </b><a href='https://www.biologicalpsychiatryjournal.com/article/S0006-3223(25)00064-2/abstract'>https://www.biologicalpsychiatryjournal.com/article/S0006-3223(25)00064-2/abstract</a><br> posted March 31, 2025 by Alexej Abyzov <br><br><li><b>Publication from the lab in Nature Communications</b><br><i>Transgenerational transmission of post-zygotic mutations suggests symmetric contribution of first two blastomeres to human germline</i><br> Little is known about the origin of germ cells in humans. We previously leveraged post-zygotic mutations to reconstruct zygote-rooted cell lineage ancestry trees in a phenotypically normal woman, termed NC0. Here, by sequencing the genome of her children and their father, we analyze the transmission of early pre-gastrulation lineages and corresponding mutations across human generations. We find that the germline in NC0 is polyclonal and is founded by at least two cells likely descending from the two blastomeres arising from the first zygotic cleavage. Analyzes of public data from several multi-children families and from 1934 familial quads confirm this finding in larger cohorts, revealing that known imbalances of up to 90:10 in early lineages allocation in somatic tissues are not reflected in mutation transmission to offspring, establishing a fundamental difference in lineage allocation between the soma and the germline. Analyzes of all the data consistently suggest that the germline has a balanced 50:50 lineage allocation from the first two blastomeres.<br> <b>More: </b><a href='https://www.nature.com/articles/s41467-024-53485-x'>https://www.nature.com/articles/s41467-024-53485-x</a><br> posted October 23, 2024 by Alexej Abyzov <br><br><li><b>Publication from the lab in Bioinformatics</b><br><i>Genome-wide analysis and visualization of copy number with CNVpytor in igv.js</i><br> Copy number variation (CNV) and alteration (CNA) analysis is a crucial component in many genomic studies and its applications span from basic research to clinic diagnostics and personalized medicine. CNVpytor is a tool featuring a read depth-based caller and combined read depth and B-allele frequency (BAF) based 2D caller to find CNVs and CNAs. The tool stores processed intermediate data and CNV/CNA calls in a compact HDF5 file—pytor file. Here, we describe a new track in igv.js that utilizes pytor and whole genome variant files as input for on-the-fly read depth and BAF visualization, CNV/CNA calling and analysis. Embedding into HTML pages and Jupiter Notebooks enables convenient remote data access and visualization simplifying interpretation and analysis of omics data.<br> <b>More: </b><a href='https://academic.oup.com/bioinformatics/article/40/8/btae453/7715874'>https://academic.oup.com/bioinformatics/article/40/8/btae453/7715874</a><br> posted August 17, 2024 by Alexej Abyzov <br><br><li><b>Publication from the lab in Scientific Reports</b><br><i>Characterization of enhancer activity in early human neurodevelopment using Massively Parallel Reporter Assay (MPRA) and forebrain organoids</i><br> Regulation of gene expression through enhancers is one of the major processes shaping the structure and function of the human brain during development. High-throughput assays have predicted thousands of enhancers involved in neurodevelopment, and confirming their activity through orthogonal functional assays is crucial. Here, we utilized Massively Parallel Reporter Assays (MPRAs) in stem cells and forebrain organoids to evaluate the activity of ~ 7000 gene-linked enhancers previously identified in human fetal tissues and brain organoids. We used a Gaussian mixture model to evaluate the contribution of background noise in the measured activity signal to confirm the activity of ~ 35% of the tested enhancers, with most showing temporal-specific activity, suggesting their evolving role in neurodevelopment. The temporal specificity was further supported by the correlation of activity with gene expression. Our findings provide a valuable gene regulatory resource to the scientific community.<br> <b>More: </b><a href='https://www.nature.com/articles/s41598-024-54302-7'>https://www.nature.com/articles/s41598-024-54302-7</a><br> posted February 23, 2024 by Alexej Abyzov <br><br><li><b>Publication from the lab in Scientific Data</b><br><i>Genomic data resources of the Brain Somatic Mosaicism Network for neuropsychiatric diseases</i><br> Somatic mosaicism is defined as an occurrence of two or more populations of cells having genomic sequences differing at given loci in an individual who is derived from a single zygote. It is a characteristic of multicellular organisms that plays a crucial role in normal development and disease. To study the nature and extent of somatic mosaicism in autism spectrum disorder, bipolar disorder, focal cortical dysplasia, schizophrenia, and Tourette syndrome, a multi-institutional consortium called the Brain Somatic Mosaicism Network (BSMN) was formed through the National Institute of Mental Health (NIMH). In addition to genomic data of affected and neurotypical brains, the BSMN also developed and validated a best practices somatic single nucleotide variant calling workflow through the analysis of reference brain tissue. These resources, which include >400 terabytes of data from 1087 subjects, are now available to the research community via the NIMH Data Archive (NDA) and are described here.<br> <b>More: </b><a href='https://www.nature.com/articles/s41597-023-02645-7'>https://www.nature.com/articles/s41597-023-02645-7</a><br> posted November 20, 2023 by Alexej Abyzov <br><br><li><b>Publication from the lab in Nature Neuroscience</b><br><i>Modeling idiopathic autism in forebrain organoids reveals an imbalance of excitatory cortical neuron subtypes during early neurogenesis</i><br> Idiopathic autism spectrum disorder (ASD) is highly heterogeneous, and it remains unclear how convergent biological processes in affected individuals may give rise to symptoms. Here, using cortical organoids and single-cell transcriptomics, we modeled alterations in the forebrain development between boys with idiopathic ASD and their unaffected fathers in 13 families. Transcriptomic changes suggest that ASD pathogenesis in macrocephalic and normocephalic probands involves an opposite disruption of the balance between excitatory neurons of the dorsal cortical plate and other lineages such as early-generated neurons from the putative preplate. The imbalance stemmed from divergent expression of transcription factors driving cell fate during early cortical development. While we did not find genomic variants in probands that explained the observed transcriptomic alterations, a significant overlap between altered transcripts and reported ASD risk genes affected by rare variants suggests a degree of gene convergence between rare forms of ASD and the developmental transcriptome in idiopathic ASD.<br> <b>More: </b><a href='https://www.nature.com/articles/s41593-023-01399-0'>https://www.nature.com/articles/s41593-023-01399-0</a><br> posted August 19, 2023 by Alexej Abyzov <br><br><li><b>Publication from the lab in The CRISPR Journal</b><br><i>Clonally Selected Lines After CRISPR-Cas Editing Are Not Isogenic</i><br> The CRISPR-Cas9 system has enabled researchers to precisely modify/edit the sequence of a genome. A typical editing experiment consists of two steps: (1) editing cultured cells, (2) cell cloning and selection of clones with and without intended edit, presumed to be isogenic. The application of CRISPR-Cas9 system may result in off-target edits, whereas cloning will reveal culture-acquired mutations. We analyzed the extent of the former and the latter by whole genome sequencing in three experiments involving separate genomic loci and conducted by three independent laboratories. In all experiments we hardly found any off-target edits, whereas detecting hundreds to thousands of single nucleotide mutations unique to each clone after relatively short culture of 10-20 passages. Notably, clones also differed in copy number alterations (CNAs) that were several kb to several mb in size and represented the largest source of genomic divergence among clones. We suggest that screening of clones for mutations and CNAs acquired in culture is a necessary step to allow correct interpretation of DNA editing experiments. Furthermore, since culture associated mutations are inevitable, we propose that experiments involving derivation of clonal lines should compare a mix of multiple unedited lines and a mix of multiple edited lines.<br> <b>More: </b><a href='https://www.liebertpub.com/doi/10.1089/crispr.2022.0050'>https://www.liebertpub.com/doi/10.1089/crispr.2022.0050</a><br> posted June 14, 2023 by Alexej Abyzov <br><br><li><b>We are part of SMaHT network established by NIH Common Fund Program</b><br><i>NIH launches $140 million effort to investigate genetic variation in normal human cells and tissues. Common Fund Program will accelerate research on human development, aging, and disease.</i><br> <br> <b>More: </b><a href='https://www.nih.gov/news-events/news-releases/nih-launches-140-million-effort-investigate-genetic-variation-normal-human-cells-tissues'>https://www.nih.gov/news-events/news-releases/nih-launches-140-million-effort-investigate-genetic-variation-normal-human-cells-tissues</a><br> posted May 11, 2023 by Alexej Abyzov <br><br><li><b>Publication from the lab in Nucleic Acid Research</b><br><i>Efficient reconstruction of cell lineage trees for cell ancestry and cancer</i><br> Mosaic mutations can be used to track cell ancestries and reconstruct high-resolution lineage trees during cancer progression and during development, starting from the first cell divisions of the zygote. However, this approach requires sampling and analyzing the genomes of multiple cells, which can be redundant in lineage representation, limiting the scalability of the approach. We describe a strategy for cost- and time-efficient lineage reconstruction using clonal induced pluripotent stem cell lines from human skin fibroblasts. The approach leverages shallow sequencing coverage to assess the clonality of the lines, clusters redundant lines and sums their coverage to accurately discover mutations in the corresponding lineages. Only a fraction of lines needs to be sequenced to high coverage. We demonstrate the effectiveness of this approach for reconstructing lineage trees during development and in hematologic malignancies. We discuss and propose an optimal experimental design for reconstructing lineage trees.<br> <b>More: </b><a href='https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkad254/7110756'>https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkad254/7110756</a><br> posted April 11, 2023 by Alexej Abyzov <br><br><li><b>Publication from the lab in Science</b><br><i>Analysis of somatic mutations in 131 human brains reveals aging-associated hypermutability</i><br> We analyzed 131 human brains (44 neurotypical, 19 with Tourette syndrome, 9 with schizophrenia, and 59 with autism) for somatic mutations after whole genome sequencing to a depth of more than 200×. Typically, brains had 20 to 60 detectable single-nucleotide mutations, but ~6% of brains harbored hundreds of somatic mutations. Hypermutability was associated with age and damaging mutations in genes implicated in cancers and, in some brains, reflected in vivo clonal expansions. Somatic duplications, likely arising during development, were found in ~5% of normal and diseased brains, reflecting background mutagenesis. Brains with autism were associated with mutations creating putative transcription factor binding motifs in enhancer-like regions in the developing brain. The top-ranked affected motifs corresponded to MEIS (myeloid ectopic viral integration site) transcription factors, suggesting a potential link between their involvement in gene regulation and autism.<br> <b>More: </b><a href='https://www.science.org/doi/10.1126/science.abm6222'>https://www.science.org/doi/10.1126/science.abm6222</a><br> posted August 1, 2022 by TJ and Suki <br><br><li><b>Publication from the lab in PLoS Compututional Biology</b><br><i>All2: A tool for selecting mosaic mutations from comprehensive multi-cell comparisons</i><br> Accurate discovery of somatic mutations in a cell is a challenge that partially lays in immaturity of dedicated analytical approaches. Approaches comparing a cell's genome to a control bulk sample miss common mutations, while approaches to find such mutations from bulk suffer from low sensitivity. We developed a tool, All2, which enables accurate filtering of mutations in a cell without the need for data from bulk(s). It is based on pair-wise comparisons of all cells to each other where every call for base pair substitution and indel is classified as either a germline variant, mosaic mutation, or false positive. As All2 allows for considering dropped-out regions, it is applicable to whole genome and exome analysis of cloned and amplified cells. By applying the approach to a variety of available data, we showed that its application reduces false positives, enables sensitive discovery of high frequency mutations, and is indispensable for conducting high resolution cell lineage tracing.<br> <b>More: </b><a href='https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009487'>https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009487</a><br> posted April 23, 2022 by Alexej Abyzov <br><br><li><b>Publication from the lab in GigaScience</b><br><i>CNVpytor: a tool for copy number variation detection and analysis from read depth and allele imbalance in whole-genome sequencing</i><br> Detecting copy number variations (CNVs) and copy number alterations (CNAs) based on whole-genome sequencing data is important for personalized genomics and treatment. CNVnator is one of the most popular tools for CNV/CNA discovery and analysis based on read depth. Herein, we present an extension of CNVnator developed in Python -- CNVpytor. CNVpytor inherits the reimplemented core engine of its predecessor and extends visualization, modularization, performance, and functionality. Additionally, CNVpytor uses B-allele frequency likelihood information from single-nucleotide polymorphisms and small indels data as additional evidence for CNVs/CNAs and as primary information for copy number neutral losses of heterozygosity. CNVpytor is significantly faster than CNVnator -- particularly for parsing alignment files (2-20 times faster) -- and has (20-50 times) smaller intermediate files. CNV calls can be filtered using several criteria, annotated, and merged over multiple samples. Modular architecture allows it to be used in shared and cloud environments such as Google Colab and Jupyter notebook. Data can be exported into JBrowse, while a lightweight plugin version of CNVpytor for JBrowse enables nearly instant and GUI-assisted analysis of CNVs by any user. CNVpytor release and the source code are available on GitHub at https://github.com/abyzovlab/CNVpytor under the MIT license.<br> <b>More: </b><a href='https://academic.oup.com/gigascience/article/10/11/giab074/6431715'>https://academic.oup.com/gigascience/article/10/11/giab074/6431715</a><br> posted November 29, 2021 by Alexej Abyzov <br><br><li><b>We are highlighted in Nature Reviews Genetics</b><br><i>Human cell-lineage imbalances</i><br> Various barcoding and labelling strategies have been developed for cell-lineage tracing ...<br> <b>More: </b><a href='https://www.nature.com/articles/s41576-021-00358-4'>https://www.nature.com/articles/s41576-021-00358-4</a><br> posted April 1, 2021 by Alexej Abyzov <br><br><li><b>Publication from the lab in Genome Biology</b><br><i>Comprehensive identification of somatic nucleotide variants in human brain tissue</i><br> Post-zygotic mutations incurred during DNA replication, DNA repair, and other cellular processes lead to somatic mosaicism. Somatic mosaicism is an established cause of various diseases, including cancers. However, detecting mosaic variants in DNA from non-cancerous somatic tissues poses significant challenges, particularly if the variants only are present in a small fraction of cells. Here, the Brain Somatic Mosaicism Network conducts a coordinated, multi-institutional study to examine the ability of existing methods to detect simulated somatic single-nucleotide variants (SNVs) in DNA mixing experiments, generate multiple replicates of whole-genome sequencing data from the dorsolateral prefrontal cortex, other brain regions, dura mater, and dural fibroblasts of a single neurotypical individual, devise strategies to discover somatic SNVs, and apply various approaches to validate somatic SNVs. These efforts lead to the identification of 43 bona fide somatic SNVs that range in variant allele fractions from 0.005 to 0.28. Guided by these results, we devise best practices for calling mosaic SNVs from 250× whole-genome sequencing data in the accessible portion of the human genome that achieve 90% specificity and sensitivity. Finally, we demonstrate that analysis of multiple bulk DNA samples from a single individual allows the reconstruction of early developmental cell lineage trees. This study provides a unified set of best practices to detect somatic SNVs in non-cancerous tissues. The data and methods are freely available to the scientific community and should serve as a guide to assess the contributions of somatic SNVs to neuropsychiatric diseases.<br> <b>More: </b><a href='https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02285-3'>https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02285-3</a><br> posted March 29, 2021 by Alexej Abyzov <br><br><li><b>Publication from the lab in Science</b><br><i>Early developmental asymmetries in cell lineage trees in living individuals</i><br> Mosaic mutations can be used to track cell lineages in humans. We used cell cloning to analyze embryonic cell lineages in two living individuals and a postmortem human specimen. Of 10 reconstructed postzygotic divisions, none resulted in balanced contributions of daughter lineages to tissues. In both living individuals, one of two lineages from the first cleavage was dominant across tissues, with 90% frequency in blood. We propose that the efficiency of DNA repair contributes to lineage imbalance. Allocation of lineages in postmortem brain correlated with anterior-posterior axis, associating lineage history with cell fate choices in embryos. We establish a minimally invasive framework for defining cell lineages in any living individual, which paves the way for studying their relevance in health and disease.<br> <b>More: </b><a href='https://science.sciencemag.org/content/371/6535/1245'>https://science.sciencemag.org/content/371/6535/1245</a><br> posted March 18, 2021 by Alexej Abyzov <br><br><li><b>Publication from the lab is featured by Genome Research</b><br><i>Complex mosaic structural variations in human fetal brains</i><br> Somatic mosaicism, manifesting as single nucleotide variants (SNVs), mobile element insertions, and structural changes in the DNA, is a common phenomenon in human brain cells, with potential functional consequences. Using a clonal approach, we previously detected 200-400 mosaic SNVs per cell in three human fetal brains (15-21 wk postconception). However, structural variation in the human fetal brain has not yet been investigated. Here, we discover and validate four mosaic structural variants (SVs) in the same brains and resolve their precise breakpoints. The SVs were of kilobase scale and complex, consisting of deletion(s) and rearranged genomic fragments, which sometimes originated from different chromosomes. Sequences at the breakpoints of these rearrangements had microhomologies, suggesting their origin from replication errors. One SV was found in two clones, and we timed its origin to ~14 wk postconception. No large scale mosaic copy number variants (CNVs) were detectable in normal fetal human brains, suggesting that previously reported megabase-scale CNVs in neurons arise at later stages of development. By reanalysis of public single nuclei data from adult brain neurons, we detected an extrachromosomal circular DNA event. Our study reveals the existence of mosaic SVs in the developing human brain, likely arising from cell proliferation during mid-neurogenesis. Although relatively rare compared to SNVs and present in ~10% of neurons, SVs in developing human brain affect a comparable number of bases in the genome (~6200 vs. ~4000 bp), implying that they may have similar functional consequences.<br> <b>More: </b><a href='https://genome.cshlp.org/content/30/12/1695'>https://genome.cshlp.org/content/30/12/1695</a><br> posted Dec 3, 2020 by Alexej Abyzov <br><br><li><b>Publication from the lab in BMC Bioinformatics</b><br><i>SCELLECTOR: ranking amplification bias in single cells using shallow sequencing</i><br> The study of mosaic mutation is important since it has been linked to cancer and various disorders. Single cell sequencing has become a powerful tool to study the genome of individual cells for the detection of mosaic mutations. The amount of DNA in a single cell needs to be amplified before sequencing and multiple displacement amplification (MDA) is widely used owing to its low error rate and long fragment length of amplified DNA. However, the phi29 polymerase used in MDA is sensitive to template fragmentation and presence of sites with DNA damage that can lead to biases such as allelic imbalance, uneven coverage and over representation of C to T mutations. It is therefore important to select cells with uniform amplification to decrease false positives and increase sensitivity for mosaic mutation detection. We propose a method, Scellector (single cell selector), which uses haplotype information to detect amplification quality in shallow coverage sequencing data. We tested Scellector on single human neuronal cells, obtained in vitro and amplified by MDA. Qualities were estimated from shallow sequencing with coverage as low as 0.3× per cell and then confirmed using 30× deep coverage sequencing. The high concordance between shallow and high coverage data validated the method. Scellector can potentially be used to rank amplifications obtained from single cell platforms relying on a MDA-like amplification step, such as Chromium Single Cell profiling solution.<br> <b>More: </b><a href='https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03858-y'>https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03858-y</a><br> posted Nov 13, 2020 by Alexej Abyzov <br><br><li><b>We are featured in neuroDEVELOPMENTS</b><br><i>Mosaics in mind</i><br> For this issue of neuroDEVELOPMENTS we focus on the startling reality of the mosaic nature of genomes in the human brain. Since the meeting of Craig Venter, Francis Collins, and Bill Clinton at the White House on June 6, 2000 to announce the first draft of the human genome, the idea that we all carry our own version of the human genetic code is commonplace. It is now clear that this is a simplified view of reality because every cell in our body does not have precisely the same genome ...<br> <b>More: </b><a href='https://mailchi.mp/libd/neurodevelopments-1361622'>https://mailchi.mp/libd/neurodevelopments-1361622</a><br> posted Oct 27, 2020 by Alexej Abyzov <br><br><li><b>Publication from the lab in Annual Review of Genomics and Human Genetics</b><br><i>Cell Lineage Tracing and Cellular Diversity in Humans</i><br> Tracing cell lineages is fundamental for understanding the rules governing development in multicellular organisms and delineating complex biological processes involving the differentiation of multiple cell types with distinct lineage hierarchies. In humans, experimental lineage tracing is unethical, and one has to rely on natural-mutation markers that are created within cells as they proliferate and age. Recent studies have demonstrated that it is now possible to trace lineages in normal, noncancerous cells with a variety of data types using natural variations in the nuclear and mitochondrial DNA as well as variations in DNA methylation status. It is also apparent that the scientific community is on the verge of being able to make a comprehensive and detailed cell lineage map of human embryonic and fetal development. In this review, we discuss the advantages and disadvantages of different approaches and markers for lineage tracing. We also describe the general conceptual design for how to derive a lineage map for humans.<br> <b>More: </b><a href='https://doi.org/10.1146/annurev-genom-083118-015241'>https://doi.org/10.1146/annurev-genom-083118-015241</a><br> posted Aug 12, 2020 by Alexej Abyzov <br><br><li><b>Publication from the lab in Bioinformatics</b><br><i>LongAGE: defining breakpoints of genomic structural variants through optimal and memory efficient alignments of long reads</i><br> Defining the precise location of structural variations (SVs) at single-nucleotide breakpoint resolution is a challenging problem due to large gaps in alignment. Previously, Alignment with Gap Excision (AGE) enabled us to define breakpoints of SVs at single-nucleotide resolution, however, AGE requires a vast amount of memory when aligning a pair of long sequences. To address this, we developed a memory-efficient implementation - LongAGE - based on the classical Hirschberg algorithm. We demonstrate an application of LongAGE for resolving breakpoints of SVs embedded into segmental duplications on Pacific Biosciences (PacBio) reads that can be longer than 10Kbp. Furthermore, we observed different breakpoints for a deletion and a duplication in the same locus, providing direct evidence that such multi-allelic copy number variants (mCNVs) arise from two or more independent ancestral mutations.<br> <b>More: </b><a href='https://doi.org/10.1093/bioinformatics/btaa703'>https://doi.org/10.1093/bioinformatics/btaa703</a><br> posted Aug 12, 2020 by Alexej Abyzov <br><br><li><b>New lab member!!!</b><br><i></i><br> Postdoctoral fellow Yifan Wang, Ph.D. joined the lab on February 17, 2020. Welcome!<br> <br> posted Feb 17, 2020 by Alexej Abyzov <br><br><li><b>New lab member!!!</b><br><i></i><br> Postdoctoral fellow Yeongjun Jang, Ph.D. joined the lab on September 2, 2019. Welcome!<br> <br> posted Sep 2, 2019 by Alexej Abyzov <br><br><li><b>New lab member!!!</b><br><i></i><br> Postdoctoral fellow Arijit Panda, Ph.D. joined the lab on June 10, 2019. Welcome!<br> <br> posted June 10, 2019 by Alexej Abyzov <br><br><li><b>New lab member!!!</b><br><i></i><br> Postdoctoral fellow Shobana Sekar, Ph.D. joined the lab on October 8, 2018. Welcome!<br> <br> posted Oct 16, 2018 by Alexej Abyzov <br><br><li><b>New lab member!!!</b><br><i></i><br> Postdoctoral fellow Milovan Suvakov, Ph.D. joined the lab on September 4, 2018. Welcome!<br> <br> posted Oct 7, 2018 by Alexej Abyzov <br><br><li><b>Publication from the lab in Nucleic Acids Research</b><br><i>Chromatin organization modulates the origin of heritable structural variations in human genome</i><br> Structural variations (SVs) in the human genome originate from different mechanisms related to DNA repair, replication errors, and retrotransposition. Our analyses of 26 927 SVs from the 1000 Genomes Project revealed differential distributions and consequences of SVs of different origin, e.g. deletions from non-allelic homologous recombination (NAHR) are more prone to disrupt chromatin organization while processed pseudogenes can create accessible chromatin. Spontaneous double stranded breaks (DSBs) are the best predictor of enrichment of NAHR deletions in open chromatin. This evidence, along with strong physical interaction of NAHR breakpoints belonging to the same deletion suggests that majority of NAHR deletions are non-meiotic i.e. originate from errors during homology directed repair (HDR) of spontaneous DSBs. In turn, the origin of the spontaneous DSBs is associated with transcription factor binding in accessible chromatin revealing the vulnerability of functional, open chromatin. The chromatin itself is enriched with repeats, particularly fixed Alu elements that provide the homology required to maintain stability via HDR. Through co-localization of fixed Alus and NAHR deletions in open chromatin we hypothesize that old Alu expansion had a stabilizing role on the human genome.<br> <b>More: </b><a href='https://doi.org/10.1093/nar/gkz103'>https://doi.org/10.1093/nar/gkz103</a><br> posted Feb 18, 2019 by Alexej Abyzov <br><br><li><b>Publication from the lab in Science</b><br><i>Transcriptome and epigenome landscape of human cortical development modeled in organoids</i><br> Genes implicated in neuropsychiatric disorders are active in human fetal brain, yet difficult to study in a longitudinal fashion. We demonstrate that organoids from human pluripotent cells model cerebral cortical development on the molecular level before 16 weeks postconception. A multiomics analysis revealed differentially active genes and enhancers, with the greatest changes occurring at the transition from stem cells to progenitors. Networks of converging gene and enhancer modules were as sembled into six and four global patterns of expression and activity across time. A pattern with progressive down-regulation was enriched with human-gained enhancers, suggesting their importance in early human brain development. A few convergent gene and enhancer modules were enriched in autism-associated genes and genomic variants in autistic children. The organoid model helps identify functional elements that may drive disease onset.<br> <b>More: </b><a href='http://doi.org/10.1126/science.aat6720'>http://doi.org/10.1126/science.aat6720</a><br> posted Dec 13, 2018 by Alexej Abyzov <br><br><li><b>Publication from the lab in Science</b><br><i>Different mutational rates and mechanisms in human cells at pregastrulation and neurogenesis</i><br> Somatic mosaicism in the human brain may alter function of individual neurons. We analyzed genomes of single cells from the forebrains of three human fetuses (15 to 21 weeks post-conception) using clonal cell populations. We detected 200-400 single nucleotide variations (SNVs) per cell. SNV patterns resembled those found in cancer cell genomes, indicating a role of background mutagenesis in cancer. SNVs with a frequency of >2% in brain were shared with the spleen, revealing a pregastrulation origin. We reconstructed cell lineages for the first five post-zygotic cleavages and calculated a mutation rate of ~1.3 per division per cell. Later in development, during neurogenesis, the mutation spectrum shifted toward oxidative damage and the mutation rate increased. Both neurogenesis and early embryogenesis exhibit drastically more mutagenesis than adulthood.<br> <b>More: </b><a href='http://doi.org/10.1126/science.aan8690'>http://doi.org/10.1126/science.aan8690</a><br> posted Dec 7, 2017 by Alexej Abyzov <br><br><li><b>Publication from the lab in Science</b><br><i>Intersection of diverse neuronal genomes and neuropsychiatric disease: The Brain Somatic Mosaicism Network</i><br> Neuropsychiatric disorders have a complex genetic architecture. Human genetic population-based studies have identified numerous heritable sequence and structural genomic variants associated with susceptibility to neuropsychiatric disease. However, these germline variants do not fully account for disease risk. During brain development, progenitor cells undergo billions of cell divisions to generate the ~80 billion neurons in the brain. The failure to accurately repair DNA damage arising during replication, transcription, and cellular metabolism amid this dramatic cellular expansion can lead to somatic mutations. Somatic mutations that alter subsets of neuronal transcriptomes and proteomes can, in turn, affect cell proliferation and survival and lead to neurodevelopmental disorders. The long life span of individual neurons and the direct relationship between neural circuits and behavior suggest that somatic mutations in small populations of neurons can significantly affect individual neurodevelopment. The Brain Somatic Mosaicism Network has been founded to study somatic mosaicism both in neurotypical human brains and in the context of complex neuropsychiatric disorders.<br> <b>More: </b><a href='http://science.sciencemag.org/content/356/6336/eaal1641.full'>http://science.sciencemag.org/content/356/6336/eaal1641.full</a><br> posted Apr 27, 2017 by Alexej Abyzov <br><br><li><b>Publication from the lab in Genome Research</b><br><i>One thousand somatic SNVs per skin fibroblast cell set baseline of mosaic mutational load with patterns that suggest proliferative origin</i><br> Few studies have been conducted to understand post-zygotic accumulation of mutations in cells of the healthy human body. We reprogrammed 32 skin fibroblast cells from families of donors into human induced pluripotent stem cell (hiPSC) lines. The clonal nature of hiPSC lines allows a high-resolution analysis of the genomes of the founder fibroblast cells without being confounded by the artifacts of single cell whole genome amplification. We estimate that on average a fibroblast cell in children has 1,035 mostly benign mosaic SNVs. On average, 235 SNVs could be directly confirmed in the original fibroblast population by ultra-deep sequencing, down to an allele frequency (AF) of 0.1%. More sensitive droplet digital PCR experiments confirmed more SNVs as mosaic with AF as low as 0.01%, suggesting that 1,035 mosaic SNVs per fibroblast cell is the true average. Similar analyses in adults revealed no significant increase in the number of SNVs per cell, suggesting that a major fraction of mosaic SNVs in fibroblasts arises during development. Mosaic SNVs were distributed uniformly across the genome and were enriched in a mutational signature previously observed in cancers and in de novo variants and which, we hypothesize, is a hallmark of normal cell proliferation. Finally, AF distribution of mosaic SNVs had distinct narrow peaks, which could be a characteristic of clonal cell selection, clonal expansion, or both. These findings reveal a large degree of somatic mosaicism in healthy human tissues, link de novo and cancer mutations to somatic mosaicism and couple somatic mosaicism with cell proliferation.<br> <b>More: </b><a href='http://genome.cshlp.org/content/early/2017/02/24/gr.215517.116.abstract'>http://genome.cshlp.org/content/early/2017/02/24/gr.215517.116.abstract</a><br> posted Mar 7, 2017 by Alexej Abyzov <br><br><li><b>Publication from the lab in Genome Research</b><br><i>Elevated variant density around SVs breakpoints in germline lineage lends support to error prone replication hypothesis</i><br> Copy number variants (CNVs) are a class of structural variants that may involve complex genomic rearrangements (CGRs) and that are hypothesized to have additional mutations around their breakpoints. Understanding the mechanisms underlying CNV formation is fundamental for understanding the repair and mutation mechanisms in cells, thereby shedding light on evolution, genomic disorders, cancer, and complex human traits. In this study, we employ data from the 1000 Genomes Project, to analyze hundreds of loci harboring heterozygous germline deletions in the subjects NA12878 and NA19240. By utilizing synthetic long-read data (longer than 2 kbp) in combination with high coverage short-read data and, in parallel, by comparing with parental genomes, we interrogated the phasing of these deletions with the flanking tens of thousands of heterozygous SNPs and indels. We found, that the density of SNPs/indels flanking the breakpoints of deletions (in-phase variants) is approximately twice as high as the corresponding density for the variants on the haplotype without deletion (out-of-phase variants). This fold-change was even larger, for the subset of deletions with signatures of replication-based mechanism of formation. The allele frequency (AF) spectrum for deletions is enriched for rare events<br> <b>More: </b><a href='http://genome.cshlp.org/content/early/2016/05/23/gr.205484.116.abstract'>http://genome.cshlp.org/content/early/2016/05/23/gr.205484.116.abstract</a><br> posted May 29, 2016 by Taejeong Bae <br><br><li><b>New lab member!!!</b><br><i></i><br> Postdoctoral fellow Tanmoy Roychowdhury, Ph.D. joined the lab on February 8, 2016. Welcome!<br> <br> posted Feb 17, 2016 by Alexej Abyzov <br><br><li><b>Publication from the lab in Nature</b><br><i>An integrated map of structural variation in 2,504 human genomes</i><br> Our collaborative work within the framework of the 1000 Genomes Project on discovery and analysis of genome structural variants has been published in Nature.<br><br>Structural variants are implicated in numerous diseases and make up the majority of varying nucleotides among human genomes. Here we describe an integrated set of eight structural variant classes comprising both balanced and unbalanced variants, which we constructed using short-read DNA sequencing data and statistically phased onto haplotype blocks in 26 human populations. Analysing this set, we identify numerous gene-intersecting structural variants exhibiting population stratification and describe naturally occurring homozygous gene knockouts that suggest the dispensability of a variety of human genes. We demonstrate that structural variants are enriched on haplotypes identified by genome-wide association studies and exhibit enrichment for expression quantitative trait loci. Additionally, we uncover appreciable levels of structural variant complexity at different scales, including genic loci subject to clusters of repeated rearrangement and complex structural variants with multiple breakpoints likely to have formed through individual mutational events. Our catalogue will enhance future studies into structural variant demography, functional impact and disease association.<br> <b>More: </b><a href='http://www.nature.com/nature/journal/v526/n7571/full/nature15394.html'>http://www.nature.com/nature/journal/v526/n7571/full/nature15394.html</a><br> posted Oct 1, 2015 by Alexej Abyzov <br><br>
</div>


<!-- Research Section -->
<!-- News Section -->
<div class="w3-container" style="padding:128px 16px" id="research">
  <h3 class="w3-center">RESEARCH</h3>
        <div class='research-div'>
<div class="w3-card-4 w3-margin research">
<header class="w3-container w3-light-grey">
  <h3>Tracing cell lineages and studying human development</h3>
</header>
<div class="w3-container w3-justify">
  <img src='imgs/lineage2.png' class='w3-center' style='width:100%;' /> 
  <p><p>Tracing cell ancestries (i.e., lineages) is fundamental to understanding the principles of development in multicellular organisms and unraveling complex biological processes involving the differentiation of diverse cell types with distinct lineage hierarchies. In humans, direct experimental lineage tracing is not ethically feasible, so researchers must rely on naturally occurring mutations that arise as cells divide and age. Our group has shown that it is possible to reconstruct lineages in normal, noncancerous human cells using natural variations in both nuclear and mitochondrial DNA, drawing on diverse types of genomic data. We are particularly focused on studying cell ancestries ancestry during the earliest stages of human development, including germline lineage tracing across generations. Our work had shed light on how early pre-gastrulation lineages are transmitted and contribute to the ongoing evolution of the human genome. With advances in genomic technologies and analytical methods, the scientific community is approaching a major milestone: the construction of a comprehensive, high-resolution map of human embryonic and fetal cell lineages. We are committed to contributing to this transformative effort including contribution to the SMaHT project.</p></b><br>
</div>
</div>


<div class="w3-card-4 w3-margin research">
<header class="w3-container w3-light-grey">
  <h3>Single cell studies</h3>
</header>
<div class="w3-container w3-justify">
  <img src='imgs/singlecell4.png' class='w3-center' style='width:100%;' /> 
  <p><p>Single-cell sequencing is a powerful approach for studying somatic mosaicism in both healthy tissues and cancer. However, because a single cell contains only a limited amount of DNA, amplification is necessary prior to sequencing. This can be achieved through clonal expansion—culturing a single cell to form a colony—or through in vitro whole-genome amplification (WGA) using polymerases. We are currently developing strategies to ensure the quality control of WGA and to distinguish true biological signal from artifacts introduced during cell culture or DNA amplification. In parallel, we are designing methods to estimate the respective contributions of signal and noise in cases where they cannot be clearly separated.</p></b><br>
</div>
</div>


<div class="w3-card-4 w3-margin research">
<header class="w3-container w3-light-grey">
  <h3>Cancer genomics</h3>
</header>
<div class="w3-container w3-justify">
  <img src='imgs/polyp2.png' class='w3-center' style='width:100%;' /> 
  <p><p>During the past decade, high-throughput next-generation technologies coupled with computational algorithms have enabled us to better understand the biology of cancer as well as the molecular underpinnings of its development and progression. Numerous functionally significant point mutations as well as structural alterations have been identified in several types and subtypes of cancers that illustrate the diverse landscape of the cancer genome. In our laboratory, we focus on the discovery and analysis of somatic point mutations and structural alterations, including deletions, duplications, and copy number changes, in colon cancer and glioma. We are especially interested in understanding the relationship between patterns of genetic alterations and modes of evolution of cancer, as well as molecular differences between cancer-free and cancer-adjacent polyps.</p></b><br>
</div>
</div>


<div class="w3-card-4 w3-margin research">
<header class="w3-container w3-light-grey">
  <h3>CNV and CNA analysis</h3>
</header>
<div class="w3-container w3-justify">
  <img src='imgs/cnvpytor2.jpg' class='w3-center' style='width:100%;' /> 
  <p><p>Copy number variation (CNV) in the genome is a complex phenomenon that remains incompletely understood. Frequent in cancers, somatic copy number alterations (CNA) have been related to cancer susceptibility, cancer progression and invasiveness, individual response to the treatment, and patients’ quality of life after treatment. The detection of CNVs and CNAs is important to address a wide spectrum of clinical and scientific questions. Research in our laboratory is focused on the discovery and analysis of CNVs and CNAs along with their relevance to diseases. We have developed and continually improved a method, CNVnator/CNVpytor, for CNV discovery and genotyping from a read-depth analysis of personal genome or cancer sequencing that currently ranks among the best, most widely used methods for CNV analysis.</p></b><br>
</div>
</div>


<div class="w3-card-4 w3-margin research">
<header class="w3-container w3-light-grey">
  <h3>Variant function</h3>
</header>
<div class="w3-container w3-justify">
  <img src='imgs/function3.png' class='w3-center' style='width:100%;' /> 
  <p><p>Advances in transcriptomic and epigenomic profiling techniques, including single-cell RNA-seq and ATAC-seq, provide powerful opportunities to study the consequences of genomic variants—both inherited and somatic—particularly in non-coding regions of the genome where their effects remain poorly understood. Our laboratory is focused on elucidating such effects, with an emphasis on variants that contribute to neurodevelopmental disorders such as autism spectrum disorder and Tourette syndrome (TS). Recently, leveraging single-nucleus transcriptomic and open chromatin datasets from the postmortem striatum of adult TS patients and matched controls, we analyzed gene expression and chromatin accessibility across defined cell types. These analyses revealed a reduction in GABAergic and cholinergic interneurons in TS brains. Additional findings suggest that interneuron loss leads to chronic hyperactivity of medium spiny neurons, resulting in metabolic stress, microglial activation, and disruption of basal ganglia homeostasis. Our ongoing work seeks to further define the regulatory landscape of TS using an expanded sample set.</p></b><br>
</div>
</div>


<div class="w3-card-4 w3-margin research">
<header class="w3-container w3-light-grey">
  <h3>Mosaic somatic mutations in humans</h3>
</header>
<div class="w3-container w3-justify">
  <img src='imgs/mosaic3.jpg' class='w3-center' style='width:100%;' /> 
  <p><p>Mutations that accumulate in DNA throughout an individual’s life give rise to a mosaic body, where each cell carries a unique set of genetic alterations — a phenomenon known as somatic mosaicism. Although somatic mosaicism is widespread, its study has been limited by challenges in detecting such variants, both at the single-cell level and in bulk tissue samples. Our group has developed computational methods to accurately detect somatic mosaic mutations by leveraging deep whole-genome sequencing of human tissue samples. Using this approach, we discovered that in the human brain, certain cells undergo clonal expansion with age, outcompeting their peers and leading to an increased burden of detectable mosaic mutations — a phenomenon we refer to as brain hypermutability. We found that this overgrowth is associated with damaging mutations in cancer driver genes. Our ongoing work focuses on uncovering the origin of this phenomenon and identifying the specific cell type(s) responsible for the clonal expansion.</p></b><br>
</div>
</div>


      </div>
</div>

<!-- News Section -->
<div class="w3-container" style="padding:128px 16px" id="news">
  <h3 class="w3-center">NEWS</h3>
        <div class='w3-row'><div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>Publication from the lab in Biological Psychiatry</h3>
</header>
<div class="w3-container">
  <b>Interneuron loss and microglia activation by transcriptome analyses in the basal ganglia of Tourette disorder</b><br>
  <p>Tourette disorder is characterized by motor hyperactivity and tics that are believed to originate in basal ganglia. Postmortem immunocytochemical analyses previously revealed decreases in cholinergic, parvalbumin, and somatostatin interneurons (IN) within the caudate/putamen of individuals with TS. We obtained transcriptome and open chromatin datasets by snRNAseq and snATAC-seq, respectively, from caudate/putamen postmortem specimens of 6 adult TS and 6 matched normal control (NC). Differential gene expression and differential chromatin accessibility analyses were performed in identified cell types.The data reproduced the known cellular composition of the human striatum, including a majority of medium spiny neurons (MSN) and small populations of GABAergic and cholinergic IN. IN were decreased by ∼50% in TS brains, with no difference in other cell types. Differential gene expression analysis suggested that mitochondrial oxidative metabolism in MSN and synaptic adhesion and function in IN were both decreased in TS subjects, while there was activation of immune response in microglia. Gene expression changes correlated with changes in activity of cis-regulatory elements, suggesting a relationship of transcriptomic and regulatory abnormalities in MSN, OL and AST of TS brains. This initial analysis of the TS basal ganglia transcriptome at the single cell level confirms the loss and synaptic dysfunction of basal ganglia IN, consistent with in vivo basal ganglia hyperactivity. In parallel, oxidative metabolism was decreased in MSN and correlated with activation of microglia cells, attributable at least in part to dysregulated activity of putative enhancers, implicating altered epigenomic regulation in TS.</b><br>
  <b>More: </b><a href='https://www.biologicalpsychiatryjournal.com/article/S0006-3223(25)00064-2/abstract'>https://www.biologicalpsychiatryjournal.com/article/S0006-3223(25)00064-2/abstract</a><br>
  
</div>
<footer class="w3-container w3-light-grey">
  <h5>posted March 31, 2025 by Alexej Abyzov</h5>
</footer>
</div>
</div>
<div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>Publication from the lab in Nature Communications</h3>
</header>
<div class="w3-container">
  <b>Transgenerational transmission of post-zygotic mutations suggests symmetric contribution of first two blastomeres to human germline</b><br>
  <p>Little is known about the origin of germ cells in humans. We previously leveraged post-zygotic mutations to reconstruct zygote-rooted cell lineage ancestry trees in a phenotypically normal woman, termed NC0. Here, by sequencing the genome of her children and their father, we analyze the transmission of early pre-gastrulation lineages and corresponding mutations across human generations. We find that the germline in NC0 is polyclonal and is founded by at least two cells likely descending from the two blastomeres arising from the first zygotic cleavage. Analyzes of public data from several multi-children families and from 1934 familial quads confirm this finding in larger cohorts, revealing that known imbalances of up to 90:10 in early lineages allocation in somatic tissues are not reflected in mutation transmission to offspring, establishing a fundamental difference in lineage allocation between the soma and the germline. Analyzes of all the data consistently suggest that the germline has a balanced 50:50 lineage allocation from the first two blastomeres.</b><br>
  <b>More: </b><a href='https://www.nature.com/articles/s41467-024-53485-x'>https://www.nature.com/articles/s41467-024-53485-x</a><br>
  
</div>
<footer class="w3-container w3-light-grey">
  <h5>posted October 23, 2024 by Alexej Abyzov</h5>
</footer>
</div>
</div>
<div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>Publication from the lab in Bioinformatics</h3>
</header>
<div class="w3-container">
  <b>Genome-wide analysis and visualization of copy number with CNVpytor in igv.js</b><br>
  <p>Copy number variation (CNV) and alteration (CNA) analysis is a crucial component in many genomic studies and its applications span from basic research to clinic diagnostics and personalized medicine. CNVpytor is a tool featuring a read depth-based caller and combined read depth and B-allele frequency (BAF) based 2D caller to find CNVs and CNAs. The tool stores processed intermediate data and CNV/CNA calls in a compact HDF5 file—pytor file. Here, we describe a new track in igv.js that utilizes pytor and whole genome variant files as input for on-the-fly read depth and BAF visualization, CNV/CNA calling and analysis. Embedding into HTML pages and Jupiter Notebooks enables convenient remote data access and visualization simplifying interpretation and analysis of omics data.</b><br>
  <b>More: </b><a href='https://academic.oup.com/bioinformatics/article/40/8/btae453/7715874'>https://academic.oup.com/bioinformatics/article/40/8/btae453/7715874</a><br>
  
</div>
<footer class="w3-container w3-light-grey">
  <h5>posted August 17, 2024 by Alexej Abyzov</h5>
</footer>
</div>
</div>
<div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>Publication from the lab in Scientific Reports</h3>
</header>
<div class="w3-container">
  <b>Characterization of enhancer activity in early human neurodevelopment using Massively Parallel Reporter Assay (MPRA) and forebrain organoids</b><br>
  <p>Regulation of gene expression through enhancers is one of the major processes shaping the structure and function of the human brain during development. High-throughput assays have predicted thousands of enhancers involved in neurodevelopment, and confirming their activity through orthogonal functional assays is crucial. Here, we utilized Massively Parallel Reporter Assays (MPRAs) in stem cells and forebrain organoids to evaluate the activity of ~ 7000 gene-linked enhancers previously identified in human fetal tissues and brain organoids. We used a Gaussian mixture model to evaluate the contribution of background noise in the measured activity signal to confirm the activity of ~ 35% of the tested enhancers, with most showing temporal-specific activity, suggesting their evolving role in neurodevelopment. The temporal specificity was further supported by the correlation of activity with gene expression. Our findings provide a valuable gene regulatory resource to the scientific community.</b><br>
  <b>More: </b><a href='https://www.nature.com/articles/s41598-024-54302-7'>https://www.nature.com/articles/s41598-024-54302-7</a><br>
  
</div>
<footer class="w3-container w3-light-grey">
  <h5>posted February 23, 2024 by Alexej Abyzov</h5>
</footer>
</div>
</div>
</div>
  <center><b><u><span onclick="news_archive()">GO TO NEWS ARCHIVE</span></u><b></center>
</div>

<!-- Team Section -->
<div class="w3-container" style="padding:128px 16px" id="team">
  <h3 class="w3-center">THE TEAM</h3>
  <div class="w3-row-padding w3" style="margin-top:64px"><div class='w3-row'>
      <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/Alexej_2022.jpg" alt="Alexej" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Alexej Abyzov, Ph.D.</b><br>
          (Алексей Абызов)<br>
          <span class="w3-opacity">LAB HEAD</span><br>
          <span>Associate Professor of Biomedical Informatics</span>
          <p><button onclick="location.href='mailto:Abyzov.Alexej@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/she.jpg" alt="Rubie" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Rubie Benike</b><br>
          (Rubie Benike)<br>
          <span class="w3-opacity">Lab Support Personnel</span><br>
          <span>Administrative Assistant</span>
          <p><button onclick="location.href='mailto:Benike.Rubie@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/Suki.jpg" alt="Milovan" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Milovan Suvakov, Ph.D.</b><br>
          (Милован Шуваков)<br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Research Associate</span>
          <p><button onclick="location.href='mailto:Suvakov.Milovan@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/Yeongjun.jpg" alt="Yeongjun" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Yeongjun Jang, Ph.D.</b><br>
          (장영준)<br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Research Associate</span>
          <p><button onclick="location.href='mailto:Jang.Yeongjun@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/Yifan_2022.jpg" alt="Yifan" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Yifan Wang, Ph.D.</b><br>
          (汪一凡)<br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Postdoctoral fellow</span>
          <p><button onclick="location.href='mailto:Yifan.Wang@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/Milana.jpg" alt="Milana" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Milana Djonovic, M.S.</b><br>
          <br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>PhD Student (MGS)</span>
          <p><button onclick="location.href='mailto:Djonovic.Milana@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/she.jpg" alt="Mrunal" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Mrunal Dehankar, M.S.</b><br>
          <br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>PhD Student (UofM)</span>
          <p><button onclick="location.href='mailto:dehankar.mrunal@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/he.jpg" alt="Wen" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Wenxuan Hong, B.S.</b><br>
          <br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>MS Student (UofM)</span>
          <p><button onclick="location.href='mailto:Hong.Wenxuan@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/Vivek_2022.jpg" alt="Vivek" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Vivekananda Sarangi, Ph.D.</b><br>
          <br>
          <span class="w3-opacity">Associate Member</span><br>
          <span>_</span>
          <p><button onclick="location.href='mailto:Sarangi.Vivekananda@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/TJ_2022.jpg" alt="Taejeong" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Taejeong Bae, Ph.D.</b><br>
          (배태정 裵泰正)<br>
          <span class="w3-opacity">Associate Member</span><br>
          <span>Collaborator</span>
          <p><button onclick="location.href='mailto:Bae.Taejeong@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l4 m8 w3-padding-16 w3-left">
      <div class="w3-card-4 w3-left" style="width:100%">
        <div class="w3-container" style="height:500px">
          <p class="w3-light-grey w3-block">ALUMNI:</p>
          <ul><li>Arijit Panda, Ph.D.<li>Silvana De Lorenzo, Ph.D.<li>Merit Kayastha<li>Shobana Sekar, Ph.D.<li>Tanmoy Roychowdhury, Ph.D.<li>Quang Tran, B.S.<li>Logan J. Manlove, B.S.<li>Marcela Aguilera, B.S.<li>Dhananjay Dhokarh, Ph.D.<li>Nikolaos Vasmatzis<li>Minsoo Kim, B.S.<li>Justin Wood</ul>
        </div>
      </div>
    </div>

  </div></div>
</div>


<div class="w3-container" style="padding:128px 16px" id="papers">
  <h3 class="w3-center">PAPERS</h3>
        </table>
</div>

<div class="w3-container" style="padding:128px 16px" id="tools">
  <h3 class="w3-center">TOOLS</h3>
        <div class='w3-row'><div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/CNVpytor" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
<div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/CNVnator" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
<div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/All2" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
<div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/SCellector" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
<div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/AGE" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
<div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/Leucippus" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
<div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/vcf2diploid" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
<div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/Belka" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
</div><script src="//cdn.jsdelivr.net/github-cards/latest/widget.js"></script>

</div>

<div class="w3-container" style="padding:128px 16px" id="positions">
  <h3 class="w3-center">POSITIONS</h3>
  <div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>PostDoc in computational biology and bioinformatics</h3>
</header>
<div class="w3-container">
  <p>application accepted year-round</p>
  <hr>
  <img src="imgs/phd.png" alt="Avatar" class="w3-left w3-circle w3-padding" style="width:20%;max-width:200px">
  <p>Applicants are invited to apply for a post-doctoral (i.e., postdoc) position in Abyzov lab at Mayo Clinic. The choice of project will depend on the applicant's interests and skills, however, the research must be purely computational and focus on one of the following main fields of computational biology: population/personal human omics, cancer omics, single cell and somatic omics, and the analysis of next-generation sequencing data. Specific sub-areas of interest are discovery, annotation, and the functional annotation of human genomic variants, cancer genomics, cancer evolution, somatic mosaicism in normal human cells.</p>
  <p>The ideal applicant will have a Ph.D. in computational biology or bioinformatics, experience in one of the aforementioned research areas, demonstrate a record of peer-reviewed publications, and possess motivation for independent research. He or she should have a very strong understanding of biology and be skilled in programming and using computers to solve problems (e.g., experience with C/C++, Java, Python/Perl, R/ROOT, etc.). Oral and written proficiency in English is also a big plus.</p>
  <p>To apply, please email your CV, including a list of publications and details for three references, to abyzov dot alexej at mayo dot edu. Please include the phrase “PostDoc application” and your full name in the subject of the email.</p>
</div>
<footer class="w3-container w3-light-grey">
<br>
</footer>
</div>
</div>

<div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>Internship in computational biology and bioinformatics</h3>
</header>
<div class="w3-container">
  <p>application accepted year-round</p>
  <hr>
  <img src="imgs/img_avatar3.png" alt="Avatar" class="w3-left w3-circle w3-padding" style="width:20%;max-width:200px">
  <p>Applications are invited for an internship at the Mayo Clinic. Anticipated projects will be related to the analysis of whole genome sequencing data, with the aims of studying germline and somatic variants (SNPs, CNVs, etc.). The analysis will involve applications of commonly used, and in-house developed, software tools, and making biological hypothesis from statistical data analysis. Intern applicants with strong programming skills will have opportunities to participate in developing new tools and improving our existing software.</p>
  <p>To apply, please email your CV, including a list of publications to abyzov dot alexej at mayo dot edu. Please include the phrase “Internship application” and your full name in the subject of the email.</p>
</div>
<footer class="w3-container w3-light-grey">
<br>
</footer>
</div>
</div>

<div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>Graduate student in computational biology and bioinformatics</h3>
</header>
<div class="w3-container">
  <img src="imgs/img_avatar3.png" alt="Avatar" class="w3-left w3-circle w3-padding" style="width:20%;max-width:200px">
  <p>Graduate students (M.S or Ph.D.) wishing to conduct research in the Abyzov lab at Mayo Clinic are invited to contact Dr. Abyzov (abyzov dot alexej at mayo dot edu). The choice of the project will depend on the applicant's interests and skills. However, the research must be purely computational and focus on one of the following main fields of computational biology: population/personal human omics, cancer omics, single cell and somatic omics, and the analysis of next-generation sequencing data. Specific sub-areas of interest are discovery, annotation, and the functional annotation of human genomic variants such as SNPs, SNVs, indels, structural variations, retrotransposition, etc.</p>
  <p>We are looking for candidates that possess motivation for independent research, have experience in computational biology or bioinformatics, and are familiar with one of the aforementioned research areas. They should have a very strong understanding of biology and be skilled in programming and using computers to solve problems (e.g., experience with C/C++ Java, Python/Perl, R/ROOT, etc.). Record of peer-reviewed publications and oral and written proficiency in English is also a big plus.</p>
  <p>Dr. Abyzov is affiliated with <a href="https://college.mayo.edu/academics/biomedical-research-training/admissions-and-benefits">Mayo Clinic College of Medicine</a> and with the program in <a href="https://r.umn.edu/umr-admissions/graduate/bicb-admissions">Biomedical Informatics and Computational Biology</a> at University of Minnesota Rochester.</p>
  <p>Please express your interest by emailing your CV, including a list of publications to abyzov dot alexej at mayo dot edu. Please include the phrase “PhD/MS interest” and your full name in the subject of the email.</p>
</div>
<footer class="w3-container w3-light-grey">
<br>
</footer>
</div>
</div>

</div>


<!-- Contact Section -->
<div class="w3-container" style="padding:128px 16px" id="contact">
  <div class="w3-row-padding">
    <div class="w3-col m6">
      Click on image for map:
      <a href="https://goo.gl/maps/TFaFnWVykEAG2qBDA"><img width="100%" src="imgs/map.png"></img></a>

    </div>
    <div class="w3-col m6 w3-center">
      <h3 class="w3-center">CONTACT</h3>
        Lets get in touch. Send us a message:<br><br>
        <b>Alexej Abyzov</b>, Ph.D., Principal Investigator<br>
        <p><i class="fa fa-envelope fa-fw w3-xxlarge w3-margin-right"> </i> Email: abyzov.alexej@mayo.edu</p>
        <br><br>
        <b>Rubie Benike</b>, Administrative Assistant<br>
        <p><i class="fa fa-phone fa-fw w3-xxlarge w3-margin-right"></i> Phone: (507) 288-7890</p>
        <p><i class="fa fa-envelope fa-fw w3-xxlarge w3-margin-right"> </i> Email:
        Benike.Rubie@mayo.edu</p><br><br>
      <table style="margin:auto;">
        <tr><td>
        <img src="imgs/pin.png" width="48px" /><br>
        <td> 
          Harwick Building 7th floor, Mayo Clinic<br>
          200 1st street SW<br>
          Rochester, MN 55905, USA<br>
      </table>
    
    </div>
  </div>
</div>

<!-- Footer -->
<footer class="w3-center w3-light-grey w3-padding-32">
  <a href="#home" class="w3-button w3-dark-grey"><i class="fa fa-arrow-up w3-margin-right"></i>To the top</a><br>
  <p>Powered by <a href="https://www.w3schools.com/w3css/default.asp" title="W3.CSS" target="_blank" class="w3-hover-text-green">w3.css</a></p>
</footer>
<img id="image" width=150 src="imgs/he.jpg"></img> 

<script>

function news_archive() {
  document.getElementById('news-panel').classList.toggle('hidden');
}

// Modal Image Gallery
function onClick(element) {
  document.getElementById("img01").src = element.src;
  document.getElementById("modal01").style.display = "block";
  var captionText = document.getElementById("caption");
  captionText.innerHTML = element.alt;
}


// Toggle between showing and hiding the sidebar when clicking the menu icon
var mySidebar = document.getElementById("mySidebar");

function w3_open() {
  if (mySidebar.style.display === 'block') {
    mySidebar.style.display = 'none';
  } else {
    mySidebar.style.display = 'block';
  }
}

// Close the sidebar with the close button
function w3_close() {
    mySidebar.style.display = "none";
}

$(document).mousemove(function(e){
    $("#image").css({left:e.pageX, top:e.pageY-$("#image").height()});
});

function showimage() {
  $("#image").show();
  $("#image").attr("src",author_image)
}

function hideimage() {
  $("#image").hide();
}

$(".author").hover(showimage,hideimage);

$('a[href^="#"]').on('click', function(event) {

    var target = $(this.getAttribute('href'));

    if( target.length ) {
        event.preventDefault();
        $('html, body').stop().animate({
            scrollTop: target.offset().top
        }, 600);
    }

});

window.onscroll = scroll;

function scroll() {
  off=70;
  maxh=160;
  st=$(document).scrollTop();
  if(st>(maxh+off)) sp=maxh;
  else if(st<off) sp=0;
  else sp=st-off;
    $(".sidebarh").css({ height: sp });
}

var slideIndex = 1;
showDivs(slideIndex);

function plusDivs(n) {
  showDivs(slideIndex += n);
}

function showDivs(n) {
  var i;
  var x = document.getElementsByClassName("mySlides");
  if (n > x.length) {slideIndex = 1}
  if (n < 1) {slideIndex = x.length}
  for (i = 0; i < x.length; i++) {
    x[i].style.display = "none";  
  }
  x[slideIndex-1].style.display = "block";  
}


var imageIx
var images

function init() {
  imageIx = 0
  images = ['imgs/group.2024.jpg', 'imgs/group.2023.2.jpg', 'imgs/group.2023.1.jpg', 'imgs/group.2022.1.jpg', 'imgs/group.2022.2.jpg', 'imgs/group.2021.jpg', 'imgs/group.2020.jpg', 'imgs/group.2019.jpg', 'imgs/group.2018.jpg', 'imgs/group.2018.19.jpg', 'imgs/group.2018.20.jpg', 'imgs/group.2016.jpg', 'imgs/group.2016.03.jpg', 'imgs/group.2015.27.jpg', 'imgs/group.2015.28.jpg']
  setInterval(changeImage,5000)
}

function changeImage() {
  console.log(images)
  var x = document.getElementsByClassName("bgimg-1")[0];
  imageIx += 1
  imageIx %= images.length  
  x.style.backgroundImage = "url('"+images[imageIx]+"')";
}

</script>

</body>
</html>


