<!DOCTYPE html>
<html>
<title>Abyzov Lab</title>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<link rel="shortcut icon" href="imgs/favicon.ico" type="image/x-icon" />
<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
<link href="https://fonts.googleapis.com/css?family=Cookie|Charm|Gentium+Basic|Gabriela|Kanit|Germania+One|Inconsolata|Raleway|Pacifico|Merriweather|Share+Tech+Mono" rel="stylesheet">

<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
<!--<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">-->
<style>
body {font-family: "Merriweather", sans-serif}
h1,h2,h3,h4,h5,h6 {font-family: "Gentium Basic", sans-serif}

.w3-top {font-family: "Merriweather", sans-serif;
  font-size:14px;
}
.logo {font-family: "Charm", sans-serif;
  font-size:32px;
  font-weight:bold;
  padding:16px;
  margin-top:4px;
}

body, html {
  height: 100%;
  line-height: 1.8;
}

#image{
  position:absolute;
  display:none
}

/* Full height image header */
.bgimg-1 {
  margin-top:50px;
  border:20px solid white;
  background-position: center;
  background-size: cover;
  background: #ffffff;
  background-image: url("imgs/group.2020.jpg");
  min-height: 100%;
  background-size: contain;
  background-repeat:no-repeat;
  background-position:center;
}

@media (orientation: landscape) {
  .bgimg-1 {
    background-image: url("imgs/group.2020.jpg");
    background-size: contain;
    background-position:center;
  }
}
.w3-bar {
  padding: 0px;
}
.w3-button {
  padding: 0px 16px;
}
.mySlides {display:none;}

#news-panel {
      width:75%;
      max-height: 60%;
      position: fixed;
      overflow-y:auto;
      top: 50%;
      left: 50%;
      transform: translate(-50%, -50%);
      z-index: 10;
      background-color: rgba(255,255,255,0.8);
      padding: 5px;
      border: 1px solid #999;
      text-align: left;
      padding-left: 10px;
}

.hidden { display: none !important; }

.papertable {
  border-collapse: collapse;
  width: 100%;
}

.papertable td, .papertable th {
  border: 0px solid #ddd;
  padding: 8px;
  vertical-align:top;
}

.papertable tr:nth-child(even){background-color: #fff;}

.papertable1 tr:hover {background-color: #fff;}

.papertable th {
  padding-top: 12px;
  padding-bottom: 12px;
  text-align: left;
  background-color: #4CAF50;
  color: white;
}

.paperimg {
  height:25px;
  margin-left:5px;
}

.research-div {
    -moz-column-count:2;
    -moz-column-gap: 0%;
    -webkit-column-count:2;
    -webkit-column-gap: 0%;
    column-count: 2;
    column-gap: 0%;
}

@media only screen and (max-width: 800px) {
.research-div {
    -moz-column-count:1;
    -moz-column-gap: 0%;
    -webkit-column-count:1;
    -webkit-column-gap: 0%;
    column-count: 1;
    column-gap: 0%;
}
}

.research {
   display: inline-block;
}

</style>

<body onload="init();">
<!-- Navbar (sit on top) -->
<div class="w3-top">
  <div class="w3-bar w3-white w3-card" id="topbar">
    <a href="#home" class="w3-bar-item w3-button logo">Abyzov lab</a>
    <!-- Right-sided navbar links -->
    <div class="w3-right w3-hide-medium w3-hide-small" style="margin-top:50px">
      <a href="#research" class="w3-bar-item w3-button"><i class="fas fa-flask"></i> RESEARCH</a>
      <a href="#news" class="w3-bar-item w3-button"><i class="fas fa-bullhorn"></i>NEWS</a>
      <a href="#team" class="w3-bar-item w3-button"><i class="fa fa-user"></i> TEAM</a>
      <a href="#papers" class="w3-bar-item w3-button"><i class="fas fa-file"></i>PAPERS</a>
      <a href="#tools" class="w3-bar-item w3-button"><i class="fas fa-tools"></i>TOOLS</a>
      <a href="#positions" class="w3-bar-item w3-button"><i class="fas fa-user-plus"></i>POSITIONS</a>
      <a href="#contact" class="w3-bar-item w3-button"><i class="fa fa-envelope"></i> CONTACT</a>
    </div>
    <!-- Hide right-floated links on small screens and replace them with a menu icon -->

    <a href="javascript:void(0)" class="w3-bar-item w3-button w3-right w3-hide-large" onclick="w3_open()">
      <i class="fa fa-bars"></i>
    </a>
  </div>
</div>

<!-- Sidebar on small screens when clicking the menu icon -->
<nav class="w3-sidebar w3-bar-block w3-black w3-card w3-animate-left w3-hide-large" style="display:none" id="mySidebar">
  <a href="javascript:void(0)" onclick="w3_close()" class="w3-bar-item w3-button w3-large w3-padding-16">Close ×</a>
  <a href="#research" onclick="w3_close()" class="w3-bar-item w3-button">RESEARCH</a>
  <a href="#news" onclick="w3_close()" class="w3-bar-item w3-button">NEWS</a>
  <a href="#team" onclick="w3_close()" class="w3-bar-item w3-button">TEAM</a>
  <a href="#papers" onclick="w3_close()" class="w3-bar-item w3-button">PAPERS</a>
  <a href="#tools" onclick="w3_close()" class="w3-bar-item w3-button">TOOLS</a>
  <a href="#positions" onclick="w3_close()" class="w3-bar-item w3-button">POSITIONS</a>
  <a href="#contact" onclick="w3_close()" class="w3-bar-item w3-button">CONTACT</a>
</nav>

<!-- Header with full-height image -->
<header class="bgimg-1 w3-display-container w3" id="home">
  <div class="w3-display-left w3-text-white" style="padding:48px">
  </div> 
</header>

<!-- News archive -->
<div id="news-panel" class="hidden" onclick="news_archive()">
    <button style="float:right">X</button>
      <h2>News archive</h2>
      <ul><li><b>Publication from the lab in BMC Bioinformatics</b> SCELLECTOR: ranking amplification bias in single cells using shallow sequencing <br> The study of mosaic mutation is important since it has been linked to cancer and various disorders. Single cell sequencing has become a powerful tool to study the genome of individual cells for the detection of mosaic mutations. The amount of DNA in a single cell needs to be amplified before sequencing and multiple displacement amplification (MDA) is widely used owing to its low error rate and long fragment length of amplified DNA. However, the phi29 polymerase used in MDA is sensitive to template fragmentation and presence of sites with DNA damage that can lead to biases such as allelic imbalance, uneven coverage and over representation of C to T mutations. It is therefore important to select cells with uniform amplification to decrease false positives and increase sensitivity for mosaic mutation detection. We propose a method, Scellector (single cell selector), which uses haplotype information to detect amplification quality in shallow coverage sequencing data. We tested Scellector on single human neuronal cells, obtained in vitro and amplified by MDA. Qualities were estimated from shallow sequencing with coverage as low as 0.3× per cell and then confirmed using 30× deep coverage sequencing. The high concordance between shallow and high coverage data validated the method. Scellector can potentially be used to rank amplifications obtained from single cell platforms relying on a MDA-like amplification step, such as Chromium Single Cell profiling solution.<br> <b>More: </b><a href='https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03858-y'>https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03858-y</a><br> posted Nov 13, 2020 by Alexej Abyzov <br><br><li><b>We are featured in neuroDEVELOPMENTS</b> Mosaics in mind <br> For this issue of neuroDEVELOPMENTS we focus on the startling reality of the mosaic nature of genomes in the human brain. Since the meeting of Craig Venter, Francis Collins, and Bill Clinton at the White House on June 6, 2000 to announce the first draft of the human genome, the idea that we all carry our own version of the human genetic code is commonplace. It is now clear that this is a simplified view of reality because every cell in our body does not have precisely the same genome ...<br> <b>More: </b><a href='https://mailchi.mp/libd/neurodevelopments-1361622'>https://mailchi.mp/libd/neurodevelopments-1361622</a><br> posted Oct 27, 2020 by Alexej Abyzov <br><br><li><b>Publication from the lab in Annual Review of Genomics and Human Genetics</b> Cell Lineage Tracing and Cellular Diversity in Humans <br> Tracing cell lineages is fundamental for understanding the rules governing development in multicellular organisms and delineating complex biological processes involving the differentiation of multiple cell types with distinct lineage hierarchies. In humans, experimental lineage tracing is unethical, and one has to rely on natural-mutation markers that are created within cells as they proliferate and age. Recent studies have demonstrated that it is now possible to trace lineages in normal, noncancerous cells with a variety of data types using natural variations in the nuclear and mitochondrial DNA as well as variations in DNA methylation status. It is also apparent that the scientific community is on the verge of being able to make a comprehensive and detailed cell lineage map of human embryonic and fetal development. In this review, we discuss the advantages and disadvantages of different approaches and markers for lineage tracing. We also describe the general conceptual design for how to derive a lineage map for humans.<br> <b>More: </b><a href='https://doi.org/10.1146/annurev-genom-083118-015241'>https://doi.org/10.1146/annurev-genom-083118-015241</a><br> posted Aug 12, 2020 by Alexej Abyzov <br><br><li><b>Publication from the lab in Bioinformatics</b> LongAGE: defining breakpoints of genomic structural variants through optimal and memory efficient alignments of long reads <br> Defining the precise location of structural variations (SVs) at single-nucleotide breakpoint resolution is a challenging problem due to large gaps in alignment. Previously, Alignment with Gap Excision (AGE) enabled us to define breakpoints of SVs at single-nucleotide resolution, however, AGE requires a vast amount of memory when aligning a pair of long sequences. To address this, we developed a memory-efficient implementation - LongAGE - based on the classical Hirschberg algorithm. We demonstrate an application of LongAGE for resolving breakpoints of SVs embedded into segmental duplications on Pacific Biosciences (PacBio) reads that can be longer than 10Kbp. Furthermore, we observed different breakpoints for a deletion and a duplication in the same locus, providing direct evidence that such multi-allelic copy number variants (mCNVs) arise from two or more independent ancestral mutations.<br> <b>More: </b><a href='https://doi.org/10.1093/bioinformatics/btaa703'>https://doi.org/10.1093/bioinformatics/btaa703</a><br> posted Aug 12, 2020 by Alexej Abyzov <br><br><li><b>New lab member!!!</b>  <br> Postdoctoral fellow Yifan Wang, Ph.D. joined the lab on February 17, 2020. Welcome!<br> <br> posted Feb 17, 2020 by Alexej Abyzov <br><br><li><b>New lab member!!!</b>  <br> Postdoctoral fellow Yeongjun Jang, Ph.D. joined the lab on September 2, 2019. Welcome!<br> <br> posted Sep 2, 2019 by Alexej Abyzov <br><br><li><b>New lab member!!!</b>  <br> Postdoctoral fellow Arijit Panda, Ph.D. joined the lab on June 10, 2019. Welcome!<br> <br> posted June 10, 2019 by Alexej Abyzov <br><br><li><b>New lab member!!!</b>  <br> Postdoctoral fellow Shobana Sekar, Ph.D. joined the lab on October 8, 2018. Welcome!<br> <br> posted Oct 16, 2018 by Alexej Abyzov <br><br><li><b>New lab member!!!</b>  <br> Postdoctoral fellow Milovan Suvakov, Ph.D. joined the lab on September 4, 2018. Welcome!<br> <br> posted Oct 7, 2018 by Alexej Abyzov <br><br><li><b>Publication from the lab in Nucleic Acids Research</b> Chromatin organization modulates the origin of heritable structural variations in human genome <br> Structural variations (SVs) in the human genome originate from different mechanisms related to DNA repair, replication errors, and retrotransposition. Our analyses of 26 927 SVs from the 1000 Genomes Project revealed differential distributions and consequences of SVs of different origin, e.g. deletions from non-allelic homologous recombination (NAHR) are more prone to disrupt chromatin organization while processed pseudogenes can create accessible chromatin. Spontaneous double stranded breaks (DSBs) are the best predictor of enrichment of NAHR deletions in open chromatin. This evidence, along with strong physical interaction of NAHR breakpoints belonging to the same deletion suggests that majority of NAHR deletions are non-meiotic i.e. originate from errors during homology directed repair (HDR) of spontaneous DSBs. In turn, the origin of the spontaneous DSBs is associated with transcription factor binding in accessible chromatin revealing the vulnerability of functional, open chromatin. The chromatin itself is enriched with repeats, particularly fixed Alu elements that provide the homology required to maintain stability via HDR. Through co-localization of fixed Alus and NAHR deletions in open chromatin we hypothesize that old Alu expansion had a stabilizing role on the human genome.<br> <b>More: </b><a href='https://doi.org/10.1093/nar/gkz103'>https://doi.org/10.1093/nar/gkz103</a><br> posted Feb 18, 2019 by Alexej Abyzov <br><br><li><b>Publication from the lab in Science</b> Transcriptome and epigenome landscape of human cortical development modeled in organoids <br> Genes implicated in neuropsychiatric disorders are active in human fetal brain, yet difficult to study in a longitudinal fashion. We demonstrate that organoids from human pluripotent cells model cerebral cortical development on the molecular level before 16 weeks postconception. A multiomics analysis revealed differentially active genes and enhancers, with the greatest changes occurring at the transition from stem cells to progenitors. Networks of converging gene and enhancer modules were as sembled into six and four global patterns of expression and activity across time. A pattern with progressive down-regulation was enriched with human-gained enhancers, suggesting their importance in early human brain development. A few convergent gene and enhancer modules were enriched in autism-associated genes and genomic variants in autistic children. The organoid model helps identify functional elements that may drive disease onset.<br> <b>More: </b><a href='http://doi.org/10.1126/science.aat6720'>http://doi.org/10.1126/science.aat6720</a><br> posted Dec 13, 2018 by Alexej Abyzov <br><br><li><b>Publication from the lab in Science</b> Different mutational rates and mechanisms in human cells at pregastrulation and neurogenesis <br> Somatic mosaicism in the human brain may alter function of individual neurons. We analyzed genomes of single cells from the forebrains of three human fetuses (15 to 21 weeks post-conception) using clonal cell populations. We detected 200-400 single nucleotide variations (SNVs) per cell. SNV patterns resembled those found in cancer cell genomes, indicating a role of background mutagenesis in cancer. SNVs with a frequency of >2% in brain were shared with the spleen, revealing a pregastrulation origin. We reconstructed cell lineages for the first five post-zygotic cleavages and calculated a mutation rate of ~1.3 per division per cell. Later in development, during neurogenesis, the mutation spectrum shifted toward oxidative damage and the mutation rate increased. Both neurogenesis and early embryogenesis exhibit drastically more mutagenesis than adulthood.<br> <b>More: </b><a href='http://doi.org/10.1126/science.aan8690'>http://doi.org/10.1126/science.aan8690</a><br> posted Dec 7, 2017 by Alexej Abyzov <br><br><li><b>Publication from the lab in Science</b> Intersection of diverse neuronal genomes and neuropsychiatric disease: The Brain Somatic Mosaicism Network <br> Neuropsychiatric disorders have a complex genetic architecture. Human genetic population-based studies have identified numerous heritable sequence and structural genomic variants associated with susceptibility to neuropsychiatric disease. However, these germline variants do not fully account for disease risk. During brain development, progenitor cells undergo billions of cell divisions to generate the ~80 billion neurons in the brain. The failure to accurately repair DNA damage arising during replication, transcription, and cellular metabolism amid this dramatic cellular expansion can lead to somatic mutations. Somatic mutations that alter subsets of neuronal transcriptomes and proteomes can, in turn, affect cell proliferation and survival and lead to neurodevelopmental disorders. The long life span of individual neurons and the direct relationship between neural circuits and behavior suggest that somatic mutations in small populations of neurons can significantly affect individual neurodevelopment. The Brain Somatic Mosaicism Network has been founded to study somatic mosaicism both in neurotypical human brains and in the context of complex neuropsychiatric disorders.<br> <b>More: </b><a href='http://science.sciencemag.org/content/356/6336/eaal1641.full'>http://science.sciencemag.org/content/356/6336/eaal1641.full</a><br> posted Apr 27, 2017 by Alexej Abyzov <br><br><li><b>Publication from the lab in Genome Research</b> One thousand somatic SNVs per skin fibroblast cell set baseline of mosaic mutational load with patterns that suggest proliferative origin <br> Few studies have been conducted to understand post-zygotic accumulation of mutations in cells of the healthy human body. We reprogrammed 32 skin fibroblast cells from families of donors into human induced pluripotent stem cell (hiPSC) lines. The clonal nature of hiPSC lines allows a high-resolution analysis of the genomes of the founder fibroblast cells without being confounded by the artifacts of single cell whole genome amplification. We estimate that on average a fibroblast cell in children has 1,035 mostly benign mosaic SNVs. On average, 235 SNVs could be directly confirmed in the original fibroblast population by ultra-deep sequencing, down to an allele frequency (AF) of 0.1%. More sensitive droplet digital PCR experiments confirmed more SNVs as mosaic with AF as low as 0.01%, suggesting that 1,035 mosaic SNVs per fibroblast cell is the true average. Similar analyses in adults revealed no significant increase in the number of SNVs per cell, suggesting that a major fraction of mosaic SNVs in fibroblasts arises during development. Mosaic SNVs were distributed uniformly across the genome and were enriched in a mutational signature previously observed in cancers and in de novo variants and which, we hypothesize, is a hallmark of normal cell proliferation. Finally, AF distribution of mosaic SNVs had distinct narrow peaks, which could be a characteristic of clonal cell selection, clonal expansion, or both. These findings reveal a large degree of somatic mosaicism in healthy human tissues, link de novo and cancer mutations to somatic mosaicism and couple somatic mosaicism with cell proliferation.<br> <b>More: </b><a href='http://genome.cshlp.org/content/early/2017/02/24/gr.215517.116.abstract'>http://genome.cshlp.org/content/early/2017/02/24/gr.215517.116.abstract</a><br> posted Mar 7, 2017 by Alexej Abyzov <br><br><li><b>Publication from the lab in Genome Research</b> Elevated variant density around SVs breakpoints in germline lineage lends support to error prone replication hypothesis <br> Copy number variants (CNVs) are a class of structural variants that may involve complex genomic rearrangements (CGRs) and that are hypothesized to have additional mutations around their breakpoints. Understanding the mechanisms underlying CNV formation is fundamental for understanding the repair and mutation mechanisms in cells, thereby shedding light on evolution, genomic disorders, cancer, and complex human traits. In this study, we employ data from the 1000 Genomes Project, to analyze hundreds of loci harboring heterozygous germline deletions in the subjects NA12878 and NA19240. By utilizing synthetic long-read data (longer than 2 kbp) in combination with high coverage short-read data and, in parallel, by comparing with parental genomes, we interrogated the phasing of these deletions with the flanking tens of thousands of heterozygous SNPs and indels. We found, that the density of SNPs/indels flanking the breakpoints of deletions (in-phase variants) is approximately twice as high as the corresponding density for the variants on the haplotype without deletion (out-of-phase variants). This fold-change was even larger, for the subset of deletions with signatures of replication-based mechanism of formation. The allele frequency (AF) spectrum for deletions is enriched for rare events<br> <b>More: </b><a href='http://genome.cshlp.org/content/early/2016/05/23/gr.205484.116.abstract'>http://genome.cshlp.org/content/early/2016/05/23/gr.205484.116.abstract</a><br> posted May 29, 2016 by Taejeong Bae <br><br><li><b>New lab member!!!</b>  <br> Postdoctoral fellow Tanmoy Roychowdhury, Ph.D. joined the lab on February 8, 2016. Welcome!<br> <br> posted Feb 17, 2016 by Alexej Abyzov <br><br><li><b>Publication from the lab in Nature</b> An integrated map of structural variation in 2,504 human genomes <br> Our collaborative work within the framework of the 1000 Genomes Project on discovery and analysis of genome structural variants has been published in Nature.<br><br>Structural variants are implicated in numerous diseases and make up the majority of varying nucleotides among human genomes. Here we describe an integrated set of eight structural variant classes comprising both balanced and unbalanced variants, which we constructed using short-read DNA sequencing data and statistically phased onto haplotype blocks in 26 human populations. Analysing this set, we identify numerous gene-intersecting structural variants exhibiting population stratification and describe naturally occurring homozygous gene knockouts that suggest the dispensability of a variety of human genes. We demonstrate that structural variants are enriched on haplotypes identified by genome-wide association studies and exhibit enrichment for expression quantitative trait loci. Additionally, we uncover appreciable levels of structural variant complexity at different scales, including genic loci subject to clusters of repeated rearrangement and complex structural variants with multiple breakpoints likely to have formed through individual mutational events. Our catalogue will enhance future studies into structural variant demography, functional impact and disease association.<br> <b>More: </b><a href='http://www.nature.com/nature/journal/v526/n7571/full/nature15394.html'>http://www.nature.com/nature/journal/v526/n7571/full/nature15394.html</a><br> posted Oct 1, 2015 by Alexej Abyzov <br><br>
</div>


<!-- Research Section -->
<!-- News Section -->
<div class="w3-container" style="padding:128px 16px" id="research">
  <h3 class="w3-center">RESEARCH</h3>
        <div class='research-div'>
<div class="w3-card-4 w3-margin research">
<header class="w3-container w3-light-grey">
  <h3>Analysis of mosaic variants and cell lineage reconstruction</h3>
</header>
<div class="w3-container w3-justify">
  <img src='imgs/mosaic.png' class='w3-center' style='width:100%;' /> 
  <p><p>Mutations in DNA that accumulate during the lifespan of each individual result in mosaic bodies, in which each cell has unique variants in the genome. That phenomenon is called somatic mosaicism. Despite the prevalence of somatic mosaicism, studying it has been limited by the lack of means to detect such variants at the level of single cells. Recent advances in single-cell genomics, however, make such research possible. Our group develops computational methods for precisely detecting somatic mosaic variants by harnessing new experimental approaches, including clonal expansion and whole genome amplification. By applying those methods to human samples, we aim to answer questions about the origin, spread, and consequence of mosaic mutations, which involves determining mutation rates, differences in the number and pattern of mutations between tissues and ages, relevance of the mutation to diseases and aging. Additionally, we are developing scalable approaches for tracing cell lineages using mutations as lineage markers.</p></b><br>
</div>
</div>


<div class="w3-card-4 w3-margin research">
<header class="w3-container w3-light-grey">
  <h3>Single cell studies</h3>
</header>
<div class="w3-container w3-justify">
  <img src='imgs/singlecell2.jpg' class='w3-center' style='width:100%;' /> 
  <p><p>Single-cell sequencing is the ultimate way to study somatic mosaicism in healthy tissues and in cancer. However, due to the scarcity of DNA in a single cell, an amplification process is required. Such amplifications can be achieved via clonal expansion, in which a single cell is cultured to produce a colony, and via in vitro whole genome amplification (WGA), in which DNA is amplified by using polymerases. We are currently formulating strategies for the quality control of WGA and to distinguish signal from noise that may be introduced during cell culture or DNA amplification, as well as developing approaches to estimate the contributions of signal and noise when they cannot be distinguished unambiguously.</p></b><br>
</div>
</div>


<div class="w3-card-4 w3-margin research">
<header class="w3-container w3-light-grey">
  <h3>Cancer genomics</h3>
</header>
<div class="w3-container w3-justify">
  <img src='imgs/polyp2.png' class='w3-center' style='width:100%;' /> 
  <p><p>During the past decade, high-throughput next-generation technologies coupled with computational algorithms have enabled us to better understand the biology of cancer as well as the molecular underpinnings of its development and progression. Numerous functionally significant point mutations as well as structural alterations have been identified in several types and subtypes of cancers that illustrate the diverse landscape of the cancer genome. In our laboratory, we focus on the discovery and analysis of somatic point mutations and structural alterations, including deletions, duplications, and copy number changes, in colon cancer and glioma. We are especially interested in understanding the relationship between patterns of genetic alterations and modes of evolution of cancer, as well as molecular differences between cancer-free and cancer-adjacent polyps.</p></b><br>
</div>
</div>


<div class="w3-card-4 w3-margin research">
<header class="w3-container w3-light-grey">
  <h3>CNV and CNA analysis</h3>
</header>
<div class="w3-container w3-justify">
  <img src='imgs/cnvpytor.jpg' class='w3-center' style='width:100%;' /> 
  <p><p>Copy number variation (CNV) in the genome is a complex phenomenon that remains incompletely understood. Frequent in cancers, somatic copy number alterations (CNA) have been related to cancer susceptibility, cancer progression and invasiveness, individual response to the treatment, and patients’ quality of life after treatment. The detection of CNVs and CNAs is important to address a wide spectrum of clinical and scientific questions. Research in our laboratory is focused on the discovery and analysis of CNVs and CNAs along with their relevance to diseases. We have developed and continually improved a method, CNVnator/CNVpytor, for CNV discovery and genotyping from a read-depth analysis of personal genome or cancer sequencing that currently ranks among the best, most widely used methods for CNV analysis.</p></b><br>
</div>
</div>


<div class="w3-card-4 w3-margin research">
<header class="w3-container w3-light-grey">
  <h3>Variant function</h3>
</header>
<div class="w3-container w3-justify">
  <img src='imgs/function.png' class='w3-center' style='width:100%;' /> 
  <p><p>Simultaneous advances in genomics (i.e., in variant discovery), epigenomics, and functional genomics (i.e., emergence of ChiP-seq, ATAC-seq, Hi-C, and RNA-seq techniques) provide opportunities to study both the origins and consequences of genomic variants. We are interested in understanding various epigenomic properties that predispose mutational processes generating single nucleotide variation (SNV) and structural variation (SV). Inversely, germline and somatic variants affect genome function. However, because many of those variants occur in non-coding regions of the genome, their effects remain poorly understood. In response, our laboratory is actively working to elucidate such effects with a particular focus on variants contributing to neuro-developmental disorders such as autism spectrum disorders and Tourette syndrome.</p></b><br>
</div>
</div>


      </div>
</div>

<!-- News Section -->
<div class="w3-container" style="padding:128px 16px" id="news">
  <h3 class="w3-center">NEWS</h3>
        <div class='w3-row'><div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>Publication from the lab in BMC Bioinformatics</h3>
</header>
<div class="w3-container">
  <b>SCELLECTOR: ranking amplification bias in single cells using shallow sequencing</b><br>
  <p>The study of mosaic mutation is important since it has been linked to cancer and various disorders. Single cell sequencing has become a powerful tool to study the genome of individual cells for the detection of mosaic mutations. The amount of DNA in a single cell needs to be amplified before sequencing and multiple displacement amplification (MDA) is widely used owing to its low error rate and long fragment length of amplified DNA. However, the phi29 polymerase used in MDA is sensitive to template fragmentation and presence of sites with DNA damage that can lead to biases such as allelic imbalance, uneven coverage and over representation of C to T mutations. It is therefore important to select cells with uniform amplification to decrease false positives and increase sensitivity for mosaic mutation detection. We propose a method, Scellector (single cell selector), which uses haplotype information to detect amplification quality in shallow coverage sequencing data. We tested Scellector on single human neuronal cells, obtained in vitro and amplified by MDA. Qualities were estimated from shallow sequencing with coverage as low as 0.3× per cell and then confirmed using 30× deep coverage sequencing. The high concordance between shallow and high coverage data validated the method. Scellector can potentially be used to rank amplifications obtained from single cell platforms relying on a MDA-like amplification step, such as Chromium Single Cell profiling solution.</b><br>
  <b>More: </b><a href='https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03858-y'>https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03858-y</a><br>
  
</div>
<footer class="w3-container w3-light-grey">
  <h5>posted Nov 13, 2020 by Alexej Abyzov</h5>
</footer>
</div>
</div>
<div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>We are featured in neuroDEVELOPMENTS</h3>
</header>
<div class="w3-container">
  <b>Mosaics in mind</b><br>
  <p>For this issue of neuroDEVELOPMENTS we focus on the startling reality of the mosaic nature of genomes in the human brain. Since the meeting of Craig Venter, Francis Collins, and Bill Clinton at the White House on June 6, 2000 to announce the first draft of the human genome, the idea that we all carry our own version of the human genetic code is commonplace. It is now clear that this is a simplified view of reality because every cell in our body does not have precisely the same genome ...</b><br>
  <b>More: </b><a href='https://mailchi.mp/libd/neurodevelopments-1361622'>https://mailchi.mp/libd/neurodevelopments-1361622</a><br>
  
</div>
<footer class="w3-container w3-light-grey">
  <h5>posted Oct 27, 2020 by Alexej Abyzov</h5>
</footer>
</div>
</div>
<div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>Publication from the lab in Annual Review of Genomics and Human Genetics</h3>
</header>
<div class="w3-container">
  <b>Cell Lineage Tracing and Cellular Diversity in Humans</b><br>
  <p>Tracing cell lineages is fundamental for understanding the rules governing development in multicellular organisms and delineating complex biological processes involving the differentiation of multiple cell types with distinct lineage hierarchies. In humans, experimental lineage tracing is unethical, and one has to rely on natural-mutation markers that are created within cells as they proliferate and age. Recent studies have demonstrated that it is now possible to trace lineages in normal, noncancerous cells with a variety of data types using natural variations in the nuclear and mitochondrial DNA as well as variations in DNA methylation status. It is also apparent that the scientific community is on the verge of being able to make a comprehensive and detailed cell lineage map of human embryonic and fetal development. In this review, we discuss the advantages and disadvantages of different approaches and markers for lineage tracing. We also describe the general conceptual design for how to derive a lineage map for humans.</b><br>
  <b>More: </b><a href='https://doi.org/10.1146/annurev-genom-083118-015241'>https://doi.org/10.1146/annurev-genom-083118-015241</a><br>
  
</div>
<footer class="w3-container w3-light-grey">
  <h5>posted Aug 12, 2020 by Alexej Abyzov</h5>
</footer>
</div>
</div>
<div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>Publication from the lab in Bioinformatics</h3>
</header>
<div class="w3-container">
  <b>LongAGE: defining breakpoints of genomic structural variants through optimal and memory efficient alignments of long reads</b><br>
  <p>Defining the precise location of structural variations (SVs) at single-nucleotide breakpoint resolution is a challenging problem due to large gaps in alignment. Previously, Alignment with Gap Excision (AGE) enabled us to define breakpoints of SVs at single-nucleotide resolution, however, AGE requires a vast amount of memory when aligning a pair of long sequences. To address this, we developed a memory-efficient implementation - LongAGE - based on the classical Hirschberg algorithm. We demonstrate an application of LongAGE for resolving breakpoints of SVs embedded into segmental duplications on Pacific Biosciences (PacBio) reads that can be longer than 10Kbp. Furthermore, we observed different breakpoints for a deletion and a duplication in the same locus, providing direct evidence that such multi-allelic copy number variants (mCNVs) arise from two or more independent ancestral mutations.</b><br>
  <b>More: </b><a href='https://doi.org/10.1093/bioinformatics/btaa703'>https://doi.org/10.1093/bioinformatics/btaa703</a><br>
  
</div>
<footer class="w3-container w3-light-grey">
  <h5>posted Aug 12, 2020 by Alexej Abyzov</h5>
</footer>
</div>
</div>
</div>
  <center><b><u><span onclick="news_archive()">GO TO NEWS ARCHIVE</span></u><b></center>
</div>

<!-- Team Section -->
<div class="w3-container" style="padding:128px 16px" id="team">
  <h3 class="w3-center">THE TEAM</h3>
  <div class="w3-row-padding w3" style="margin-top:64px"><div class='w3-row'>
      <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/Alexej.jpg" alt="Alexej" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Alexej Abyzov, Ph.D.</b><br>
          <span class="w3-opacity">LAB HEAD</span><br>
          <span>Associate Professor of Biomedical Informatics</span>
          <p><button onclick="location.href='mailto:Abyzov.Alexej@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/Leighann.jpg" alt="Leighann" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Leighann Wagner</b><br>
          <span class="w3-opacity">Lab Support Personnel</span><br>
          <span>Administrative Assistant</span>
          <p><button onclick="location.href='mailto:Wagner,Wagner.Leighann@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/TJ.jpg" alt="Taejeong" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Taejeong Bae, Ph.D.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Research Associate</span>
          <p><button onclick="location.href='mailto:Bae.Taejeong@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/Suki.jpg" alt="Milovan" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Milovan Suvakov, Ph.D.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Research Associate</span>
          <p><button onclick="location.href='mailto:Suvakov.Milovan@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/Arijit.jpg" alt="Arijit" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Arijit Panda, Ph.D.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Postdoctoral fellow</span>
          <p><button onclick="location.href='mailto:Panda.Arijit@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/Yeongjun.jpg" alt="Yeongjun" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Yeongjun Jang, Ph.D.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Postdoctoral fellow</span>
          <p><button onclick="location.href='mailto:Jang.Yeongjun@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/Yifan.jpg" alt="Yifan" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Yifan Wang, Ph.D.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Postdoctoral fellow</span>
          <p><button onclick="location.href='mailto:Yifan.Wang@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/Vivek.jpg" alt="Vivek" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Vivekananda Sarangi, M.S.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>PhD Student</span>
          <p><button onclick="location.href='mailto:Sarangi.Vivekananda@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l2 m4 w3-padding-16 w3-center">
      <div class="w3-card-4 w3-center" style="height:500px">
        <img src="imgs/Chen.jpg" alt="Chen" style="height:300px;max-width:90%; !important">
        <div class="w3-container">
          <b>Chen Wang, Ph.D.</b><br>
          <span class="w3-opacity">Affiliated member</span><br>
          <span>Senior Associate Consultant</span>
          <p><button onclick="location.href='mailto:Wang.Chen@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l4 m8 w3-padding-16 w3-left">
      <div class="w3-card-4 w3-left" style="width:100%">
        <div class="w3-container" style="height:500px">
          <p class="w3-light-grey w3-block">ALUMNI:</p>
          <ul><li>Shobana Sekar, Ph.D.<li>Tanmoy Roychowdhury, Ph.D.<li>Quang Tran, B.S.<li>Logan J. Manlove, B.S.<li>Marcela Aguilera, B.S.<li>Dhananjay Dhokarh, Ph.D.<li>Nikolaos Vasmatzis<li>Minsoo Kim<li>Justin Wood</ul>
        </div>
      </div>
    </div>

  </div></div>
</div>


<div class="w3-container" style="padding:128px 16px" id="papers">
  <h3 class="w3-center">PAPERS</h3>
        </table>
</div>

<div class="w3-container" style="padding:128px 16px" id="tools">
  <h3 class="w3-center">TOOLS</h3>
        <div class='w3-row'><div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/CNVpytor" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
<div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/CNVnator" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
<div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/All2" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
<div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/SCellector" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
<div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/AGE" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
<div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/Leucippus" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
<div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/vcf2diploid" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
<div class="w3-col l4 m6 w3-padding-16">
  <div class="w3-margin" style="height:200px; !important">
    <div class="github-card" data-github="abyzovlab/Belka" data-width="100%" data-height="200" data-theme="default"></div>
  </div>
</div>
</div><script src="//cdn.jsdelivr.net/github-cards/latest/widget.js"></script>

</div>

<div class="w3-container" style="padding:128px 16px" id="positions">
  <h3 class="w3-center">POSITIONS</h3>
  <div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>PostDoc in computational biology and bioinformatics</h3>
</header>
<div class="w3-container">
  <p>application accepted year-round</p>
  <hr>
  <img src="imgs/phd.png" alt="Avatar" class="w3-left w3-circle w3-padding" style="width:20%;max-width:200px">
  <p>Applicants are invited to apply for a post-doctoral (i.e., postdoc) position in Abyzov lab at Mayo Clinic. The choice of project will depend on the applicant's interests and skills, however, the research must be purely computational and focus on one of the following main fields of computational biology: population/personal human omics, cancer omics, single cell and somatic omics, and the analysis of next-generation sequencing data. Specific sub-areas of interest are discovery, annotation, and the functional annotation of human genomic variants, cancer genomics, cancer evolution, somatic mosaicism in normal human cells.</p>
  <p>The ideal applicant will have a Ph.D. in computational biology or bioinformatics, experience in one of the aforementioned research areas, demonstrate a record of peer-reviewed publications, and possess motivation for independent research. He or she should have a very strong understanding of biology and be skilled in programming and using computers to solve problems (e.g., experience with C/C++, Java, Python/Perl, R/ROOT, etc.). Oral and written proficiency in English is also a big plus.</p>
  <p>To apply, please email your CV, including a list of publications and details for three references, to abyzov dot alexej at mayo dot edu. Please include the phrase “PostDoc application” and your full name in the subject of the email.</p>
</div>
<footer class="w3-container w3-light-grey">
<br>
</footer>
</div>
</div>

<div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>Internship in computational biology and bioinformatics</h3>
</header>
<div class="w3-container">
  <p>application accepted year-round</p>
  <hr>
  <img src="imgs/img_avatar3.png" alt="Avatar" class="w3-left w3-circle w3-padding" style="width:20%;max-width:200px">
  <p>Applications are invited for an internship at the Mayo Clinic. Anticipated projects will be related to the analysis of whole genome sequencing data, with the aims of studying germline and somatic variants (SNPs, CNVs, etc.). The analysis will involve applications of commonly used, and in-house developed, software tools, and making biological hypothesis from statistical data analysis. Intern applicants with strong programming skills will have opportunities to participate in developing new tools and improving our existing software.</p>
  <p>To apply, please email your CV, including a list of publications to abyzov dot alexej at mayo dot edu. Please include the phrase “Internship application” and your full name in the subject of the email.</p>
</div>
<footer class="w3-container w3-light-grey">
<br>
</footer>
</div>
</div>

<div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>Graduate student in computational biology and bioinformatics</h3>
</header>
<div class="w3-container">
  <img src="imgs/img_avatar3.png" alt="Avatar" class="w3-left w3-circle w3-padding" style="width:20%;max-width:200px">
  <p>Graduate students (M.S or Ph.D.) wishing to conduct research in the Abyzov lab at Mayo Clinic are invited to contact Dr. Abyzov (abyzov dot alexej at mayo dot edu). The choice of the project will depend on the applicant's interests and skills. However, the research must be purely computational and focus on one of the following main fields of computational biology: population/personal human omics, cancer omics, single cell and somatic omics, and the analysis of next-generation sequencing data. Specific sub-areas of interest are discovery, annotation, and the functional annotation of human genomic variants such as SNPs, SNVs, indels, structural variations, retrotransposition, etc.</p>
  <p>We are looking for candidates that possess motivation for independent research, have experience in computational biology or bioinformatics, and are familiar with one of the aforementioned research areas. They should have a very strong understanding of biology and be skilled in programming and using computers to solve problems (e.g., experience with C/C++ Java, Python/Perl, R/ROOT, etc.). Record of peer-reviewed publications and oral and written proficiency in English is also a big plus.</p>
  <p>Dr. Abyzov is affiliated with <a href="https://college.mayo.edu">Mayo Clinic College of Medicine</a> and with the program in <a href="https://r.umn.edu/academics-research/graduate-programs/bicb">Biomedical Informatics and Computational Biology</a> at University of Minnesota Rochester.</p>
  <p>Please express your interest by emailing your CV, including a list of publications to abyzov dot alexej at mayo dot edu. Please include the phrase “PhD/MS interest” and your full name in the subject of the email.</p>
</div>
<footer class="w3-container w3-light-grey">
<br>
</footer>
</div>
</div>

</div>


<!-- Contact Section -->
<div class="w3-container" style="padding:128px 16px" id="contact">
  <div class="w3-row-padding">
    <div class="w3-col m6">
      Click on image for map:
      <a href="https://goo.gl/maps/TFaFnWVykEAG2qBDA"><img width="100%" src="imgs/map.png"></img></a>

    </div>
    <div class="w3-col m6 w3-center">
      <h3 class="w3-center">CONTACT</h3>
        Lets get in touch. Send us a message:<br><br>
        <b>Alexej Abyzov</b>, Ph.D., Principal Investigator<br>
        <p><i class="fa fa-envelope fa-fw w3-xxlarge w3-margin-right"> </i> Email: abyzov.alexej@mayo.edu</p>
        <br><br>
        <b>Leighann Wagner</b>, Administrative Assistant<br>
        <p><i class="fa fa-phone fa-fw w3-xxlarge w3-margin-right"></i> Phone: (507) 284-5569</p>
        <p><i class="fa fa-envelope fa-fw w3-xxlarge w3-margin-right"> </i> Email:
        wagner.leighann@mayo.edu</p><br><br>
      <table style="margin:auto;">
        <tr><td>
        <img src="imgs/pin.png" width="48px" /><br>
        <td> 
          Harwick Building 3rd floor, Mayo Clinic<br>
          200 1st street SW<br>
          Rochester, MN 55905, USA<br>
      </table>
    
    </div>
  </div>
</div>

<!-- Footer -->
<footer class="w3-center w3-light-grey w3-padding-32">
  <a href="#home" class="w3-button w3-dark-grey"><i class="fa fa-arrow-up w3-margin-right"></i>To the top</a><br>
  <p>Powered by <a href="https://www.w3schools.com/w3css/default.asp" title="W3.CSS" target="_blank" class="w3-hover-text-green">w3.css</a></p>
</footer>
<img id="image" width=150 src="imgs/he.jpg"></img> 

<script>

function news_archive() {
  document.getElementById('news-panel').classList.toggle('hidden');
}

// Modal Image Gallery
function onClick(element) {
  document.getElementById("img01").src = element.src;
  document.getElementById("modal01").style.display = "block";
  var captionText = document.getElementById("caption");
  captionText.innerHTML = element.alt;
}


// Toggle between showing and hiding the sidebar when clicking the menu icon
var mySidebar = document.getElementById("mySidebar");

function w3_open() {
  if (mySidebar.style.display === 'block') {
    mySidebar.style.display = 'none';
  } else {
    mySidebar.style.display = 'block';
  }
}

// Close the sidebar with the close button
function w3_close() {
    mySidebar.style.display = "none";
}

$(document).mousemove(function(e){
    $("#image").css({left:e.pageX, top:e.pageY-$("#image").height()});
});

function showimage() {
  $("#image").show();
  $("#image").attr("src",author_image)
}

function hideimage() {
  $("#image").hide();
}

$(".author").hover(showimage,hideimage);

$('a[href^="#"]').on('click', function(event) {

    var target = $(this.getAttribute('href'));

    if( target.length ) {
        event.preventDefault();
        $('html, body').stop().animate({
            scrollTop: target.offset().top
        }, 600);
    }

});

window.onscroll = scroll;

function scroll() {
  off=70;
  maxh=160;
  st=$(document).scrollTop();
  if(st>(maxh+off)) sp=maxh;
  else if(st<off) sp=0;
  else sp=st-off;
    $(".sidebarh").css({ height: sp });
}

var slideIndex = 1;
showDivs(slideIndex);

function plusDivs(n) {
  showDivs(slideIndex += n);
}

function showDivs(n) {
  var i;
  var x = document.getElementsByClassName("mySlides");
  if (n > x.length) {slideIndex = 1}
  if (n < 1) {slideIndex = x.length}
  for (i = 0; i < x.length; i++) {
    x[i].style.display = "none";  
  }
  x[slideIndex-1].style.display = "block";  
}


var imageIx
var images

function init() {
  imageIx = 0
  images = ['imgs/group.2020.jpg', 'imgs/group.2019.jpg', 'imgs/group.2018.jpg', 'imgs/group.2018.19.jpg', 'imgs/group.2018.20.jpg', 'imgs/group.2016.jpg', 'imgs/group.2016.03.jpg', 'imgs/group.2015.27.jpg', 'imgs/group.2015.28.jpg']
  setInterval(changeImage,5000)
}

function changeImage() {
  console.log(images)
  var x = document.getElementsByClassName("bgimg-1")[0];
  imageIx += 1
  imageIx %= images.length  
  x.style.backgroundImage = "url('"+images[imageIx]+"')";
}

</script>

</body>
</html>


