<!DOCTYPE html>
<html>
<title>Abyzov Lab</title>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
<link href="https://fonts.googleapis.com/css?family=Cookie|Charm|Gentium+Basic|Gabriela|Kanit|Germania+One|Inconsolata|Raleway|Pacifico|Merriweather|Share+Tech+Mono" rel="stylesheet">

<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
<!--<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">-->
<style>
body {font-family: "Merriweather", sans-serif}
h1,h2,h3,h4,h5,h6 {font-family: "Gentium Basic", sans-serif}
.w3-top {font-family: "Merriweather", sans-serif;
  font-size:14px;
}
.logo {font-family: "Charm", sans-serif;
  font-size:32px;
  font-weight:bold;
  padding:16px;
  margin-top:4px;
}

body, html {
  height: 90%;
  line-height: 1.8;
}

#image{
  position:absolute;
  display:none
}

/* Full height image header */
.bgimg-1 {
  background-position: center;
  background-size: cover;
  background: #aaaaaa;
  background-image: url("imgs/group2.jpg");
  min-height: 90%;
  background-size: cover;
  background-repeat:no-repeat;
  background-position:center;
}

@media (orientation: landscape) {
  .bgimg-1 {
    background-image: url("imgs/group2.jpg");
    background-size: cover;
    background-position:center;
  }
}
.w3-bar {
  padding: 0px;
}
.w3-button {
  padding: 0px 16px;
}
.mySlides {display:none;}

.papertable {
  border-collapse: collapse;
  width: 90%;
}

.papertable td, .papertable th {
  border: 0px solid #ddd;
  padding: 8px;
  vertical-align:top;
}

.papertable tr:nth-child(even){background-color: #fff;}

.papertable1 tr:hover {background-color: #fff;}

.papertable th {
  padding-top: 12px;
  padding-bottom: 12px;
  text-align: left;
  background-color: #4CAF50;
  color: white;
}

.paperimg {
  height:25px;
  margin-left:5px;
}

</style>
<body>

<!-- Navbar (sit on top) -->
<div class="w3-top">
  <div class="w3-bar w3-white w3-card" id="topbar">
    <a href="#home" class="w3-bar-item w3-button logo">Abyzov lab</a>
    <!-- Right-sided navbar links -->
    <div class="w3-right w3-hide-medium w3-hide-small" style="margin-top:50px">
      <a href="#research" class="w3-bar-item w3-button"><i class="fas fa-flask"></i> RESEARCH</a>
      <a href="#news" class="w3-bar-item w3-button"><i class="fas fa-bullhorn"></i>NEWS</a>
      <a href="#team" class="w3-bar-item w3-button"><i class="fa fa-user"></i> TEAM</a>
      <a href="#papers" class="w3-bar-item w3-button"><i class="fas fa-file"></i>PAPERS</a>
      <a href="#tools" class="w3-bar-item w3-button"><i class="fas fa-tools"></i>TOOLS</a>
      <a href="#positions" class="w3-bar-item w3-button"><i class="fas fa-user-plus"></i>POSITIONS</a>
      <a href="#contact" class="w3-bar-item w3-button"><i class="fa fa-envelope"></i> CONTACT</a>
    </div>
    <!-- Hide right-floated links on small screens and replace them with a menu icon -->

    <a href="javascript:void(0)" class="w3-bar-item w3-button w3-right w3-hide-large" onclick="w3_open()">
      <i class="fa fa-bars"></i>
    </a>
  </div>
</div>

<!-- Sidebar on small screens when clicking the menu icon -->
<nav class="w3-sidebar w3-bar-block w3-black w3-card w3-animate-left w3-hide-large" style="display:none" id="mySidebar">
  <a href="javascript:void(0)" onclick="w3_close()" class="w3-bar-item w3-button w3-large w3-padding-16">Close ×</a>
  <a href="#research" onclick="w3_close()" class="w3-bar-item w3-button">RESEARCH</a>
  <a href="#news" onclick="w3_close()" class="w3-bar-item w3-button">NEWS</a>
  <a href="#team" onclick="w3_close()" class="w3-bar-item w3-button">TEAM</a>
  <a href="#papers" onclick="w3_close()" class="w3-bar-item w3-button">PAPERS</a>
  <a href="#tools" onclick="w3_close()" class="w3-bar-item w3-button">TOOLS</a>
  <a href="#positions" onclick="w3_close()" class="w3-bar-item w3-button">POSITIONS</a>
  <a href="#contact" onclick="w3_close()" class="w3-bar-item w3-button">CONTACT</a>
</nav>

<!-- Header with full-height image -->
<header class="bgimg-1 w3-display-container w3" id="home">
  <div class="w3-display-left w3-text-white" style="padding:48px">
  </div> 
  <div class="w3-display-bottomleft w3-text-grey w3-large" style="padding:24px 48px">
    <i class="fa fa-facebook-official w3-hover-opacity"></i>
    <i class="fa fa-instagram w3-hover-opacity"></i>
    <i class="fa fa-snapchat w3-hover-opacity"></i>
    <i class="fa fa-pinterest-p w3-hover-opacity"></i>
    <i class="fa fa-twitter w3-hover-opacity"></i>
    <i class="fa fa-linkedin w3-hover-opacity"></i>
  </div>
</header>

<!-- Research Section -->
<!-- News Section -->
<div class="w3-container" style="padding:128px 16px" id="research">
  <h3 class="w3-center">RESEARCH</h3>
        <div class='w3-row'><div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>Analysis of mosaic variants and cell lineage reconstruction</h3>
</header>
<div class="w3-container">
  <img src='imgs/mosaic.png' class='w3-center' style='width:90%;' /> 
  <p><p>Mutations in DNA accumulate during lifespan of an individual resulting in mosaic body where each cell has its private variants in the genome. This is called somatic mosaicism. Despite of its commonness, studying somatic mosaicism has been limited by lack of means to detect such variants at single-cell level. Recent advances in single cell genomics make it possible. Our group develops new computational methods for precise detection of somatic mosaic variants harnessing new experimental approaches such Next-gen sequencing. By applying the methods to human samples, we pursue to answer the questions about the origin and etiology of somatic mosaic mutations, which involves determining mutation rates during human development, the difference in numbers and patterns of mutations between tissues and between development stages, the relationships of somatic mutations to disease, aging, and healthy life.</p></b><br>
</div>
<footer class="w3-container w3-light-grey">
  <h5>Analysis of mosaic variants and cell lineage reconstruction</h5>
</footer>
</div>
</div>


<div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>Single cell studies</h3>
</header>
<div class="w3-container">
  <img src='imgs/singlecell.jpg' class='w3-center' style='width:90%;' /> 
  <p><p>Somatic mutations start accumulating with every cell divisions as early as the zygotic stage. These mutations accumulate in the organism and could become a precursor for malignancies. Thus, study of these mutations can shed light on prognosis and possible treatment of the disease. Single cell sequencing has been shown to be better at detecting somatic mosaicism compared to bulk sequencing. However, due to the scares amount of DNA in a single cell, an amplification process is required. There are two type of amplifications, clonal expansion: where a single cell is cultured to produce a colony, which provides enough DNA for sequencing and; MDA (Multiple Displacement Amplification) where the DNA is amplified using PCR-like methods. Different amplification methods have their pros and cons and also introduces specific artifacts.</p><p>We are working on strategies for distinguishing signal from noise that might be introduced during cell culture or DNA amplification. Applying these methods, we are able to distinguish properly amplified good cells from bad cells with multiple artifacts introduced due to various reasons. We are also constantly trying to understand the origin of these biases in order to address them better.</p><p>With the advent of next generation sequencing, single cell genomics provides new perspective to the understanding of genetics by studying genomes at cellular level. Our focus being whole genome single cell sequencing, we are working on involve methods/protocols development for minimizing amplification biases introduced during MDA (Multiple Displacement Amplification) and strategies for distinguishing signal from noise that might be introduced during cell culture or DNA amplification. Applying these methods, we are able to distinguish properly amplified good cells from bad cells with multiple artifacts introduced due to various reasons. We are also constantly trying to understand the origin of these biases in order to address them better.</p></b><br>
</div>
<footer class="w3-container w3-light-grey">
  <h5>Single cell studies</h5>
</footer>
</div>
</div>


<div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>Cancer genomics</h3>
</header>
<div class="w3-container">
  <img src='imgs/polip.png' class='w3-center' style='width:90%;' /> 
  <p><p>Over the past decade, high throughput next generation technologies coupled with computational algorithms have enabled us to better understand the biology of cancer as well as the molecular underpinnings of its development and progression. Numerous functionally significant point mutations as well as structural alterations have been identified in several types and sub-types of cancers, illustrating the diverse landscape of the cancer genome. In our lab, we focus on the discovery and analysis of somatic point mutations and structural alterations including deletions, duplications and copy number changes in colon cancer and glioma. Specifically, we are interested in understanding the relationship between patterns of genetic alterations and modes of evolution of the cancer, as well as molecular differences between a cancer-free and cancer adjacent polyp.</p></b><br>
</div>
<footer class="w3-container w3-light-grey">
  <h5>Cancer genomics</h5>
</footer>
</div>
</div>


<div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>CNV and CNA analysis</h3>
</header>
<div class="w3-container">
  <img src='imgs/cnvnator.png' class='w3-center' style='width:90%;' /> 
  <p><p>Copy number variation (CNV) in the genome is a complex phenomenon, and not completely understood. Somatic copy number alterations (CNA) are frequent in cancers and have been related to cancer susceptibility, cancer progression and invasiveness, individual response to the treatment and patients’ quality life after treatment. Detection of CNVs and CNAs is important to address large spectrum of clinical and scientific questions. Research in our laboratory is focused on the discovery and analysis of genomic variants and their relevance to diseases. We have developed and constantly improving a method, CNVnator, for CNV discovery and genotyping from read-depth (RD) analysis of personal genome sequencing. Our method is based on combining the mean-shift approach used in machine learning with additional refinements. C++ implementation of CNVnator is freely available in laboratory github page</p></b><br>
</div>
<footer class="w3-container w3-light-grey">
  <h5>CNV and CNA analysis</h5>
</footer>
</div>
</div>


      </div>
</div>

<!-- News Section -->
<div class="w3-container" style="padding:128px 16px" id="news">
  <h3 class="w3-center">NEWS</h3>
        <div class='w3-row'><div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>New lab member!!!</h3>
</header>
<div class="w3-container">
  <b></b><br>
  <p>Postdoctoral fellow Shobana Sekar, Ph.D. joined the lab on October 8, 2018. Welcome!</b><br>
  <br>
  
</div>
<footer class="w3-container w3-light-grey">
  <h5>posted Oct 16, 2018, 10:15 AM by Alexej Abyzov</h5>
</footer>
</div>
</div>
<div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>New lab member!!!</h3>
</header>
<div class="w3-container">
  <b></b><br>
  <p>Postdoctoral fellow Milovan Suvakov, Ph.D. joined the lab on September 4, 2018. Welcome!</b><br>
  <br>
  
</div>
<footer class="w3-container w3-light-grey">
  <h5>posted Oct 7, 2018, 9:40 PM by Alexej Abyzov</h5>
</footer>
</div>
</div>
<div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>Publication from the lab in Nucleic Acids Research</h3>
</header>
<div class="w3-container">
  <b>Chromatin organization modulates the origin of heritable structural variations in human genome</b><br>
  <p>Structural variations (SVs) in the human genome originate from different mechanisms related to DNA repair, replication errors, and retrotransposition. Our analyses of 26 927 SVs from the 1000 Genomes Project revealed differential distributions and consequences of SVs of different origin, e.g. deletions from non-allelic homologous recombination (NAHR) are more prone to disrupt chromatin organization while processed pseudogenes can create accessible chromatin. Spontaneous double stranded breaks (DSBs) are the best predictor of enrichment of NAHR deletions in open chromatin. This evidence, along with strong physical interaction of NAHR breakpoints belonging to the same deletion suggests that majority of NAHR deletions are non-meiotic i.e. originate from errors during homology directed repair (HDR) of spontaneous DSBs. In turn, the origin of the spontaneous DSBs is associated with transcription factor binding in accessible chromatin revealing the vulnerability of functional, open chromatin. The chromatin itself is enriched with repeats, particularly fixed Alu elements that provide the homology required to maintain stability via HDR. Through co-localization of fixed Alus and NAHR deletions in open chromatin we hypothesize that old Alu expansion had a stabilizing role on the human genome.</b><br>
  <b>More: </b><a href='https://doi.org/10.1093/nar/gkz103'>https://doi.org/10.1093/nar/gkz103</a><br>
  
</div>
<footer class="w3-container w3-light-grey">
  <h5>posted Feb 18, 2019, 10:33 AM by Alexej Abyzov</h5>
</footer>
</div>
</div>
<div class="w3-col l6 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-light-grey">
  <h3>Publication from the lab in Science</h3>
</header>
<div class="w3-container">
  <b>Transcriptome and epigenome landscape of human cortical development modeled in organoids</b><br>
  <p>Genes implicated in neuropsychiatric disorders are active in human fetal brain, yet difficult to study in a longitudinal fashion. We demonstrate that organoids from human pluripotent cells model cerebral cortical development on the molecular level before 16 weeks postconception. A multiomics analysis revealed differentially active genes and enhancers, with the greatest changes occurring at the transition from stem cells to progenitors. Networks of converging gene and enhancer modules were as sembled into six and four global patterns of expression and activity across time. A pattern with progressive down-regulation was enriched with human-gained enhancers, suggesting their importance in early human brain development. A few convergent gene and enhancer modules were enriched in autism-associated genes and genomic variants in autistic children. The organoid model helps identify functional elements that may drive disease onset.</b><br>
  <b>More: </b><a href='http://doi.org/10.1126/science.aat6720'>http://doi.org/10.1126/science.aat6720</a><br>
  
</div>
<footer class="w3-container w3-light-grey">
  <h5>posted Dec 13, 2018, 11:30 AM by Alexej Abyzov</h5>
</footer>
</div>
</div>
</div>
  <center><b><u>GO TO NEWS ARCHIVE</u><b></center>
</div>

<!-- Team Section -->
<div class="w3-container" style="padding:128px 16px" id="team">
  <h3 class="w3-center">THE TEAM</h3>
  <div class="w3-row-padding w3" style="margin-top:64px">
        <div class='w3-row'>    <div class="w3-col l3 m6 w3-padding-16">
      <div class="w3-card-4">
        <img src="imgs/Alexej.jpg" alt="Alexej" style="width:100%">
        <div class="w3-container" style="height:200px">
          <b>Alexej Abyzov, Ph.D.</b><br>
          <span class="w3-opacity">LAB HEAD</span><br>
          <span>Senior Associate Consultant, Assistant Professor of Biomedical Informatics</span>
          <p><button onclick="location.href='mailto:Abyzov.Alexej@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l3 m6 w3-padding-16">
      <div class="w3-card-4">
        <img src="imgs/Leighann.jpg" alt="Leighann" style="width:100%">
        <div class="w3-container" style="height:200px">
          <b>Leighann Wagner</b><br>
          <span class="w3-opacity">Lab Support Personnel</span><br>
          <span>Administrative Assistant</span>
          <p><button onclick="location.href='mailto:Wagner,Wagner.Leighann@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l3 m6 w3-padding-16">
      <div class="w3-card-4">
        <img src="imgs/TJ.jpg" alt="Taejeong" style="width:100%">
        <div class="w3-container" style="height:200px">
          <b>Taejeong Bae, Ph.D.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Research Associate</span>
          <p><button onclick="location.href='mailto:Bae.Taejeong@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l3 m6 w3-padding-16">
      <div class="w3-card-4">
        <img src="imgs/Vivek.jpg" alt="Vivek" style="width:100%">
        <div class="w3-container" style="height:200px">
          <b>Vivekananda Sarangi, M.S.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>PhD Student</span>
          <p><button onclick="location.href='mailto:Sarangi.Vivekananda@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l3 m6 w3-padding-16">
      <div class="w3-card-4">
        <img src="imgs/Suki.jpg" alt="Milovan" style="width:100%">
        <div class="w3-container" style="height:200px">
          <b>Milovan Suvakov, Ph.D.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Postdoctoral fellow</span>
          <p><button onclick="location.href='mailto:Suvakov.Milovan@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l3 m6 w3-padding-16">
      <div class="w3-card-4">
        <img src="imgs/Shobana.jpg" alt="Shobana" style="width:100%">
        <div class="w3-container" style="height:200px">
          <b>Shobana Sekar, Ph.D.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Postdoctoral fellow</span>
          <p><button onclick="location.href='mailto:Sekar.Shobana@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l3 m6 w3-padding-16">
      <div class="w3-card-4">
        <img src="imgs/Arijit.jpg" alt="Arijit" style="width:100%">
        <div class="w3-container" style="height:200px">
          <b>Arijit Panda, Ph.D.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Postdoctoral fellow</span>
          <p><button onclick="location.href='mailto:Panda.Arijit@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l3 m6 w3-padding-16">
      <div class="w3-card-4">
        <img src="imgs/Yeongjun.jpg" alt="Yeongjun" style="width:100%">
        <div class="w3-container" style="height:200px">
          <b>Yeongjun Jang, Ph.D.</b><br>
          <span class="w3-opacity">Lab Member</span><br>
          <span>Postdoctoral fellow</span>
          <p><button onclick="location.href='mailto:Jang.Yeongjun@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l3 m6 w3-padding-16">
      <div class="w3-card-4">
        <img src="imgs/" alt="Tanmoy" style="width:100%">
        <div class="w3-container" style="height:200px">
          <b>Tanmoy Roychowdhury, Ph.D.</b><br>
          <span class="w3-opacity">Affiliated Lab Member</span><br>
          <span>Research Collaborator</span>
          <p><button onclick="location.href='mailto:-';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
    <div class="w3-col l3 m6 w3-padding-16">
      <div class="w3-card-4">
        <img src="imgs/" alt="Chen" style="width:100%">
        <div class="w3-container" style="height:200px">
          <b>Chen Wang, Ph.D.</b><br>
          <span class="w3-opacity">Affiliated Lab Member</span><br>
          <span>Senior Associate Consultant</span>
          <p><button onclick="location.href='mailto:Wang.Chen@mayo.edu';" class="w3-button w3-light-grey w3-block"><i class="fa fa-envelope"></i> Contact</button></p>
        </div>
      </div>
    </div>
</div>
  </div>
</div>


<div class="w3-container" style="padding:128px 16px" id="papers">
  <h3 class="w3-center">PAPERS</h3>
            <b>2019</b><br>
      <table class='papertable'>
<tr>
<td width="40px">57.</td>
<td><b><font color=blue>Comprehensive, integrated, and phased whole-genome analysis of the primary ENCODE cell line K562.</font></b><br>
Zhou B, Ho SS, Greer SU, Zhu X, Bell JM, Arthur JG, Spies N, Zhang X, Byeon S, Pattni R, Ben-Efraim N, Haney MS, Haraksingh RR, Song G, Ji HP, Perrin D, Wong WH, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Urban AE<br>
<b>Genome Res</b> 2019; 29(3):472-484<br>
<a href='http://genome.cshlp.org/cgi/pmidlookup?view=long&pmid=30737237'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--highwire.stanford.edu-icons-externalservices-pubmed-standard-genome_final_free.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/30737237'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2018</b><br>
      <table class='papertable'>
<tr>
<td width="40px">56.</td>
<td><b><font color=blue>Transcriptome and epigenome landscape of human cortical development modeled in organoids.</font></b><br>
Amiri A, Coppola G, Scuderi S, Wu F, <span class='author' onmouseover='author_image="imgs/";'><b><u>Roychowdhury T</u></b></span>, Liu F, Pochareddy S, Shin Y, Safi A, Song L, Zhu Y, Sousa AMM, PsychENCODE Consortium, Gerstein M, Crawford GE, Sestan N, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Vaccarino FM<br>
<b>Science</b> 2018; 362(6420)<br>
<a href='http://www.sciencemag.org/cgi/pmidlookup?view=long&pmid=30545853'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--highwire.stanford.edu-icons-externalservices-pubmed-sci_full.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/30545853'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">55.</td>
<td><b><font color=blue>Molecular characterization of colorectal adenomas with and without malignancy reveals distinguishing genome, transcriptome and methylome alterations.</font></b><br>
Druliner BR, Wang P, <span class='author' onmouseover='author_image="imgs/TJ.jpg";'><b><u>Bae T</u></b></span>, Baheti S, Slettedahl S, Mahoney D, <span class='author' onmouseover='author_image="imgs/Niko.jpg";'><b><u>Vasmatzis N</u></b></span>, Xu H, <span class='author' onmouseover='author_image="imgs/he.jpg";'><b><u>Kim M</u></b></span>, Bockol M, O'Brien D, Grill D, Warner N, Munoz-Gomez M, Kossick K, Johnson R, Mouchli M, Felmlee-Devine D, Washechek-Aletto J, Smyrk T, Oberg A, Wang J, Chia N, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Ahlquist D, Boardman LA<br>
<b>Sci Rep</b> 2018; 8(1):3161<br>
<a href='http://dx.doi.org/10.1038/s41598-018-21525-4'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--www.nature.com-images-lo_npg.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/29453410'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">54.</td>
<td><b><font color=blue>Different mutational rates and mechanisms in human cells at pregastrulation and neurogenesis.</font></b><br>
<span class='author' onmouseover='author_image="imgs/TJ.jpg";'><b><u>Bae T</u></b></span>, Tomasini L, Mariani J, Zhou B, <span class='author' onmouseover='author_image="imgs/";'><b><u>Roychowdhury T</u></b></span>, Franjic D, Pletikos M, Pattni R, Chen BJ, Venturini E, Riley-Gillis B, Sestan N, Urban AE, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Vaccarino FM<br>
<b>Science</b> 2018; 359(6375):550-555<br>
<a href='http://www.sciencemag.org/cgi/pmidlookup?view=long&pmid=29217587'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--highwire.stanford.edu-icons-externalservices-pubmed-sci_full.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/29217587'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">53.</td>
<td><b><font color=blue>Detection and Quantification of Mosaic Genomic DNA Variation in Primary Somatic Tissues Using ddPCR: Analysis of Mosaic Transposable-Element Insertions, Copy-Number Variants, and Single-Nucleotide Variants.</font></b><br>
Zhou B, Haney MS, Zhu X, Pattni R, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Urban AE<br>
<b>Methods Mol Biol</b> 2018; 1768:173-190<br>
<a href='https://dx.doi.org/10.1007/978-1-4939-7778-9_11'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--production.springer.de-OnlineResources-Logos-springerlink.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/29717444'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2017</b><br>
      <table class='papertable'>
<tr>
<td width="40px">52.</td>
<td><b><font color=blue>Inferring modes of evolution from colorectal cancer with residual polyp of origin.</font></b><br>
<span class='author' onmouseover='author_image="imgs/he.jpg";'><b><u>Kim M</u></b></span>, Druliner BR, <span class='author' onmouseover='author_image="imgs/Niko.jpg";'><b><u>Vasmatzis N</u></b></span>, <span class='author' onmouseover='author_image="imgs/TJ.jpg";'><b><u>Bae T</u></b></span>, Chia N, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Boardman LA<br>
<b>Oncotarget</b> 2017; 9(6):6780-6792<br>
<a href='http://www.impactjournals.com/oncotarget/misc/linkedout.php?pii=23687'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--www.impactjournals.com-oncotarget-images-misc-linkout.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/29467928'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">51.</td>
<td><b><font color=blue>Patient-reported (EORTC QLQ-CIPN20) versus physician-reported (CTCAE) quantification of oxaliplatin- and paclitaxel/carboplatin-induced peripheral neuropathy in NCCTG/Alliance clinical trials.</font></b><br>
Le-Rademacher J, Kanwar R, Seisler D, Pachman DR, Qin R, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Ruddy KJ, Banck MS, Lavoie Smith EM, Dorsey SG, Aaronson NK, Sloan J, Loprinzi CL, Beutler AS<br>
<b>Support Care Cancer</b> 2017; 25(11):3537-3544<br>
<a href='https://dx.doi.org/10.1007/s00520-017-3780-y'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--production.springer.de-OnlineResources-Logos-springerlink.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/28634656'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">50.</td>
<td><b><font color=blue>Landscape and variation of novel retroduplications in 26 human populations.</font></b><br>
Zhang Y, Li S, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Gerstein MB<br>
<b>PLoS Comput Biol</b> 2017; 13(6):e1005567<br>
<a href='http://dx.plos.org/10.1371/journal.pcbi.1005567'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--journals.plos.org-ploscompbiol-resource-img-external-pcbi_120x30.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/28662076'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">49.</td>
<td><b><font color=blue>Human induced pluripotent stem cells for modelling neurodevelopmental disorders.</font></b><br>
Ardhanareeswaran K, Mariani J, Coppola G, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Vaccarino FM<br>
<b>Nat Rev Neurol</b> 2017; 13(5):265-278<br>
<a href='http://dx.doi.org/10.1038/nrneurol.2017.45'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--www.nature.com-images-logo_nrneuro.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/28418023'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">48.</td>
<td><b><font color=blue>Intersection of diverse neuronal genomes and neuropsychiatric disease: The Brain Somatic Mosaicism Network.</font></b><br>
McConnell MJ, Moran JV, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Akbarian S, <span class='author' onmouseover='author_image="imgs/TJ.jpg";'><b><u>Bae T</u></b></span>, Cortes-Ciriano I, Erwin JA, Fasching L, Flasch DA, Freed D, Ganz J, Jaffe AE, Kwan KY, Kwon M, Lodato MA, Mills RE, Paquola ACM, Rodin RE, Rosenbluh C, Sestan N, Sherman MA, Shin JH, Song S, Straub RE, Thorpe J, Weinberger DR, Urban AE, Zhou B, Gage FH, Lehner T, Senthil G, Walsh CA, Chess A, Courchesne E, Gleeson JG, Kidd JM, Park PJ, Pevsner J, Vaccarino FM, Brain Somatic Mosaicism Network<br>
<b>Science</b> 2017; 356(6336)<br>
<a href='http://www.sciencemag.org/cgi/pmidlookup?view=long&pmid=28450582'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--highwire.stanford.edu-icons-externalservices-pubmed-sci_full.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/28450582'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">47.</td>
<td><b><font color=blue>Comprehensive performance comparison of high-resolution array platforms for genome-wide Copy Number Variation (CNV) analysis in humans.</font></b><br>
Haraksingh RR, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Urban AE<br>
<b>BMC Genomics</b> 2017; 18(1):321<br>
<a href='https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-3658-x'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--s3-service-broker-live-ddda94b7-dbb0-4917-917d-776dae91ebba.s3.amazonaws.com-bmc-linkout-fulltext.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/28438122'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">46.</td>
<td><b><font color=blue>One thousand somatic SNVs per skin fibroblast cell set baseline of mosaic mutational load with patterns that suggest proliferative origin.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Tomasini L, Zhou B, <span class='author' onmouseover='author_image="imgs/Niko.jpg";'><b><u>Vasmatzis N</u></b></span>, Coppola G, Amenduni M, Pattni R, Wilson M, Gerstein M, Weissman S, Urban AE, Vaccarino FM<br>
<b>Genome Res</b> 2017; 27(4):512-523<br>
<a href='http://genome.cshlp.org/cgi/pmidlookup?view=long&pmid=28235832'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--highwire.stanford.edu-icons-externalservices-pubmed-standard-genome_final_free.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/28235832'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">45.</td>
<td><b><font color=blue>Genomic Mosaicism in Neurons and Other Cell Types</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Urban AE, Vaccarino FM<br>
<b>Chapter 1, Principles and Approaches for Discovery and Validation of Somatic Mosaicism in the Human Brain</b> 2017; Fade JM, Gage FH, editors. Springer New York: Springer Nature; 3-24<br>

</td>
</tr>
      </table><br>      <b>2016</b><br>
      <table class='papertable'>
<tr>
<td width="40px">44.</td>
<td><b><font color=blue>Colorectal Cancer with Residual Polyp of Origin: A Model of Malignant Transformation.</font></b><br>
Druliner BR, Rashtak S, Ruan X, <span class='author' onmouseover='author_image="imgs/TJ.jpg";'><b><u>Bae T</u></b></span>, <span class='author' onmouseover='author_image="imgs/Niko.jpg";'><b><u>Vasmatzis N</u></b></span>, O'Brien D, Johnson R, Felmlee-Devine D, Washechek-Aletto J, Basu N, Liu H, Smyrk T, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Boardman LA<br>
<b>Transl Oncol</b> 2016; 9(4):280-6<br>
<a href='https://linkinghub.elsevier.com/retrieve/pii/S1936-5233(16)30083-3'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--linkinghub.elsevier.com-ihub-images-elsevieroa.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/27567950'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">43.</td>
<td><b><font color=blue>Elevated variant density around SV breakpoints in germline lineage lends support to error-prone replication hypothesis.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Danny.jpg";'><b><u>Dhokarh D</u></b></span>, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span><br>
<b>Genome Res</b> 2016; 26(7):874-81<br>
<a href='http://genome.cshlp.org/cgi/pmidlookup?view=long&pmid=27216746'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--highwire.stanford.edu-icons-externalservices-pubmed-standard-genome_final_free.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/27216746'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">42.</td>
<td><b><font color=blue>Single-cell analysis of targeted transcriptome predicts drug sensitivity of single cells within human myeloma tumors.</font></b><br>
Mitra AK, Mukherjee UK, Harding T, Jang JS, Stessman H, Li Y, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Jen J, Kumar S, Rajkumar V, Van Ness B<br>
<b>Leukemia</b> 2016; 30(5):1094-102<br>
<a href='http://dx.doi.org/10.1038/leu.2015.361'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--www.nature.com-images-lo_npg.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/26710886'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">41.</td>
<td><b><font color=blue>A uniform survey of allele-specific binding and expression over 1000-Genomes-Project individuals.</font></b><br>
Chen J, Rozowsky J, Galeev TR, Harmanci A, Kitchen R, Bedford J, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Kong Y, Regan L, Gerstein M<br>
<b>Nat Commun</b> 2016; 7:11101<br>
<a href='http://dx.doi.org/10.1038/ncomms11101'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--www.nature.com-images-lo_npg.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/27089393'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">40.</td>
<td><b><font color=blue>Testing of candidate single nucleotide variants associated with paclitaxel neuropathy in the trial NCCTG N08C1 (Alliance).</font></b><br>
Boora GK, Kanwar R, Kulkarni AA, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Sloan J, Ruddy KJ, Banck MS, Loprinzi CL, Beutler AS<br>
<b>Cancer Med</b> 2016; 5(4):631-9<br>
<a href='https://doi.org/10.1002/cam4.625'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--media.wiley.com-assets-7388-68-wiley-free-full-text.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/26763541'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">39.</td>
<td><b><font color=blue>Understanding genome structural variations.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Li S, Gerstein MB<br>
<b>Oncotarget</b> 2016; 7(7):7370-1<br>
<a href='http://www.impactjournals.com/oncotarget/misc/linkedout.php?pii=6485'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--www.impactjournals.com-oncotarget-images-misc-linkout.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/26657727'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2015</b><br>
      <table class='papertable'>
<tr>
<td width="40px">38.</td>
<td><b><font color=blue>The PsychENCODE project.</font></b><br>
PsychENCODE Consortium, Akbarian S, Liu C, Knowles JA, Vaccarino FM, Farnham PJ, Crawford GE, Jaffe AE, Pinto D, Dracheva S, Geschwind DH, Mill J, Nairn AC, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Pochareddy S, Prabhakar S, Weissman S, Sullivan PF, State MW, Weng Z, Peters MA, White KP, Gerstein MB, Amiri A, Armoskus C, Ashley-Koch AE, <span class='author' onmouseover='author_image="imgs/TJ.jpg";'><b><u>Bae T</u></b></span>, Beckel-Mitchener A, Berman BP, Coetzee GA, Coppola G, Francoeur N, Fromer M, Gao R, Grennan K, Herstein J, Kavanagh DH, Ivanov NA, Jiang Y, Kitchen RR, Kozlenkov A, Kundakovic M, Li M, Li Z, Liu S, Mangravite LM, Mattei E, Markenscoff-Papadimitriou E, Navarro FC, North N, Omberg L, Panchision D, Parikshak N, Poschmann J, Price AJ, Purcaro M, Reddy TE, Roussos P, Schreiner S, Scuderi S, Sebra R, Shibata M, Shieh AW, Skarica M, Sun W, Swarup V, Thomas A, Tsuji J, van Bakel H, Wang D, Wang Y, Wang K, Werling DM, Willsey AJ, Witt H, Won H, Wong CC, Wray GA, Wu EY, Xu X, Yao L, Senthil G, Lehner T, Sklar P, Sestan N<br>
<b>Nat Neurosci</b> 2015; 18(12):1707-12<br>
<a href='http://dx.doi.org/10.1038/nn.4156'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--www.nature.com-images-lo_nn.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/26605881'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">37.</td>
<td><b><font color=blue>A global reference for human genetic variation.</font></b><br>
1000 Genomes Project Consortium, Auton A, Brooks LD, Durbin RM, Garrison EP, Kang HM, Korbel JO, Marchini JL, McCarthy S, McVean GA, Abecasis GR<br>
<b>Nature</b> 2015; 526(7571):68-74<br>
<a href='https://doi.org/10.1038/nature15393'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--www.nature.com-images-lo_nature.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/26432245'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">36.</td>
<td><b><font color=blue>An integrated map of structural variation in 2,504 human genomes.</font></b><br>
Sudmant PH, Rausch T, Gardner EJ, Handsaker RE, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Huddleston J, Zhang Y, Ye K, Jun G, Fritz MH, Konkel MK, Malhotra A, Stütz AM, Shi X, Casale FP, Chen J, Hormozdiari F, Dayama G, Chen K, Malig M, Chaisson MJP, Walter K, Meiers S, Kashin S, Garrison E, Auton A, Lam HYK, Mu XJ, Alkan C, Antaki D, <span class='author' onmouseover='author_image="imgs/TJ.jpg";'><b><u>Bae T</u></b></span>, Cerveira E, Chines P, Chong Z, Clarke L, Dal E, Ding L, Emery S, Fan X, Gujral M, Kahveci F, Kidd JM, Kong Y, Lameijer EW, McCarthy S, Flicek P, Gibbs RA, Marth G, Mason CE, Menelaou A, Muzny DM, Nelson BJ, Noor A, Parrish NF, Pendleton M, Quitadamo A, Raeder B, Schadt EE, Romanovitch M, Schlattl A, Sebra R, Shabalin AA, Untergasser A, Walker JA, Wang M, Yu F, Zhang C, Zhang J, Zheng-Bradley X, Zhou W, Zichner T, Sebat J, Batzer MA, McCarroll SA, 1000 Genomes Project Consortium, Mills RE, Gerstein MB, Bashir A, Stegle O, Devine SE, Lee C, Eichler EE, Korbel JO<br>
<b>Nature</b> 2015; 526(7571):75-81<br>
<a href='https://doi.org/10.1038/nature15394'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--www.nature.com-images-lo_nature.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/26432246'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">35.</td>
<td><b><font color=blue>MetaSV: an accurate and integrative structural-variant caller for next generation sequencing.</font></b><br>
Mohiyuddin M, Mu JC, Li J, Bani Asadi N, Gerstein MB, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Wong WH, Lam HY<br>
<b>Bioinformatics</b> 2015; 31(16):2741-4<br>
<a href='https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btv204'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--academic.oup.com-images-oup_pubmed.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/25861968'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">34.</td>
<td><b><font color=blue>FOXG1-Dependent Dysregulation of GABA/Glutamate Neuron Differentiation in Autism Spectrum Disorders.</font></b><br>
Mariani J, Coppola G, Zhang P, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Provini L, Tomasini L, Amenduni M, Szekely A, Palejev D, Wilson M, Gerstein M, Grigorenko EL, Chawarska K, Pelphrey KA, Howe JR, Vaccarino FM<br>
<b>Cell</b> 2015; 162(2):375-390<br>
<a href='https://linkinghub.elsevier.com/retrieve/pii/S0092-8674(15)00759-X'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--linkinghub.elsevier.com-ihub-images-celloa.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/26186191'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">33.</td>
<td><b><font color=blue>Analysis of deletion breakpoints from 1,092 humans reveals details of mutation mechanisms.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Li S, Kim DR, Mohiyuddin M, Stütz AM, Parrish NF, Mu XJ, Clark W, Chen K, Hurles M, Korbel JO, Lam HY, Lee C, Gerstein MB<br>
<b>Nat Commun</b> 2015; 6:7256<br>
<a href='http://dx.doi.org/10.1038/ncomms8256'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--www.nature.com-images-lo_npg.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/26028266'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">32.</td>
<td><b><font color=blue>VarSim: a high-fidelity simulation and validation framework for high-throughput genome sequencing with cancer applications.</font></b><br>
Mu JC, Mohiyuddin M, Li J, Bani Asadi N, Gerstein MB, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Wong WH, Lam HY<br>
<b>Bioinformatics</b> 2015; 31(9):1469-71<br>
<a href='https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btu828'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--academic.oup.com-images-oup_pubmed.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/25524895'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2013</b><br>
      <table class='papertable'>
<tr>
<td width="40px">31.</td>
<td><b><font color=blue>Analysis of variable retroduplications in human populations suggests coupling of retrotransposition to cell division.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Iskow R, Gokcumen O, Radke DW, Balasubramanian S, Pei B, Habegger L, 1000 Genomes Project Consortium, Lee C, Gerstein M<br>
<b>Genome Res</b> 2013; 23(12):2042-52<br>
<a href='http://genome.cshlp.org:4040/cgi/pmidlookup?view=long&pmid=24026178'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--highwire.stanford.edu-icons-externalservices-pubmed-standard-genome_full_free.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/24026178'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">30.</td>
<td><b><font color=blue>Integrative annotation of variants from 1092 humans: application to cancer genomics.</font></b><br>
Khurana E, Fu Y, Colonna V, Mu XJ, Kang HM, Lappalainen T, Sboner A, Lochovsky L, Chen J, Harmanci A, Das J, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Balasubramanian S, Beal K, Chakravarty D, Challis D, Chen Y, Clarke D, Clarke L, Cunningham F, Evani US, Flicek P, Fragoza R, Garrison E, Gibbs R, Gümüş ZH, Herrero J, Kitabayashi N, Kong Y, Lage K, Liluashvili V, Lipkin SM, MacArthur DG, Marth G, Muzny D, Pers TH, Ritchie GRS, Rosenfeld JA, Sisu C, Wei X, Wilson M, Xue Y, Yu F, 1000 Genomes Project Consortium, Dermitzakis ET, Yu H, Rubin MA, Tyler-Smith C, Gerstein M<br>
<b>Science</b> 2013; 342(6154):1235587<br>
<a href='http://www.sciencemag.org/cgi/pmidlookup?view=long&pmid=24092746'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--highwire.stanford.edu-icons-externalservices-pubmed-standard-sci_full_free.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/24092746'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">29.</td>
<td><b><font color=blue>Child development and structural variation in the human genome.</font></b><br>
Zhang Y, Haraksingh R, Grubert F, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Gerstein M, Weissman S, Urban AE<br>
<b>Child Dev</b> 2013; 84(1):34-48<br>
<a href='https://doi.org/10.1111/cdev.12051'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--media.wiley.com-assets-7388-69-wiley-full-text.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/23311762'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2012</b><br>
      <table class='papertable'>
<tr>
<td width="40px">28.</td>
<td><b><font color=blue>Somatic copy number mosaicism in human skin revealed by induced pluripotent stem cells.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Mariani J, Palejev D, Zhang Y, Haney MS, Tomasini L, Ferrandino AF, Rosenberg Belmaker LA, Szekely A, Wilson M, Kocabas A, Calixto NE, Grigorenko EL, Huttner A, Chawarska K, Weissman S, Urban AE, Gerstein M, Vaccarino FM<br>
<b>Nature</b> 2012; 492(7429):438-42<br>
<a href='https://doi.org/10.1038/nature11629'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--www.nature.com-images-lo_nature.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/23160490'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">27.</td>
<td><b><font color=blue>An integrated map of genetic variation from 1,092 human genomes.</font></b><br>
1000 Genomes Project Consortium, Abecasis GR, Auton A, Brooks LD, DePristo MA, Durbin RM, Handsaker RE, Kang HM, Marth GT, McVean GA<br>
<b>Nature</b> 2012; 491(7422):56-65<br>
<a href='https://doi.org/10.1038/nature11632'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--www.nature.com-images-lo_nature.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/23128226'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">26.</td>
<td><b><font color=blue>An integrated encyclopedia of DNA elements in the human genome.</font></b><br>
ENCODE Project Consortium<br>
<b>Nature</b> 2012; 489(7414):57-74<br>
<a href='https://doi.org/10.1038/nature11247'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--www.nature.com-images-lo_nature.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/22955616'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">25.</td>
<td><b><font color=blue>Architecture of the human regulatory network derived from ENCODE data.</font></b><br>
Gerstein MB, Kundaje A, Hariharan M, Landt SG, Yan KK, Cheng C, Mu XJ, Khurana E, Rozowsky J, Alexander R, Min R, Alves P, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Addleman N, Bhardwaj N, Boyle AP, Cayting P, Charos A, Chen DZ, Cheng Y, Clarke D, Eastman C, Euskirchen G, Frietze S, Fu Y, Gertz J, Grubert F, Harmanci A, Jain P, Kasowski M, Lacroute P, Leng JJ, Lian J, Monahan H, O'Geen H, Ouyang Z, Partridge EC, Patacsil D, Pauli F, Raha D, Ramirez L, Reddy TE, Reed B, Shi M, Slifer T, Wang J, Wu L, Yang X, Yip KY, Zilberman-Schapira G, Batzoglou S, Sidow A, Farnham PJ, Myers RM, Weissman SM, Snyder M<br>
<b>Nature</b> 2012; 489(7414):91-100<br>
<a href='https://doi.org/10.1038/nature11245'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--www.nature.com-images-lo_nature.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/22955619'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">24.</td>
<td><b><font color=blue>Regulatory element copy number differences shape primate expression profiles.</font></b><br>
Iskow RC, Gokcumen O, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Malukiewicz J, Zhu Q, Sukumar AT, Pai AA, Mills RE, Habegger L, Cusanovich DA, Rubel MA, Perry GH, Gerstein M, Stone AC, Gilad Y, Lee C<br>
<b>Proc Natl Acad Sci U S A</b> 2012; 109(31):12656-61<br>
<a href='http://www.pnas.org/cgi/pmidlookup?view=long&pmid=22797897'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--highwire.stanford.edu-icons-externalservices-pubmed-custom-pnas_full_free.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/22797897'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2011</b><br>
      <table class='papertable'>
<tr>
<td width="40px">23.</td>
<td><b><font color=blue>Genome-wide mapping of copy number variation in humans: comparative analysis of high resolution array platforms.</font></b><br>
Haraksingh RR, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Gerstein M, Urban AE, Snyder M<br>
<b>PLoS One</b> 2011; 6(11):e27859<br>
<a href='http://dx.plos.org/10.1371/journal.pone.0027859'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--journals.plos.org-plosone-resource-img-external-pone_120x30.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/22140474'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">22.</td>
<td><b><font color=blue>Integration of protein motions with molecular networks reveals different mechanisms for permanent and transient interactions.</font></b><br>
Bhardwaj N, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Clarke D, Shou C, Gerstein MB<br>
<b>Protein Sci</b> 2011; 20(10):1745-54<br>
<a href='https://doi.org/10.1002/pro.710'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--media.wiley.com-assets-7388-68-wiley-free-full-text.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/21826754'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">21.</td>
<td><b><font color=blue>AlleleSeq: analysis of allele-specific expression and binding in a network framework.</font></b><br>
Rozowsky J, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Wang J, Alves P, Raha D, Harmanci A, Leng J, Bjornson R, Kong Y, Kitabayashi N, Bhardwaj N, Rubin M, Snyder M, Gerstein M<br>
<b>Mol Syst Biol</b> 2011; 7:522<br>
<a href='https://doi.org/10.1038/msb.2011.54'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.embopress.org-pb-assets-embo-site-images-pubmed-msb.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/21811232'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">20.</td>
<td><b><font color=blue>Identification of genomic indels and structural variations using split reads.</font></b><br>
Zhang ZD, Du J, Lam H, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Urban AE, Snyder M, Gerstein M<br>
<b>BMC Genomics</b> 2011; 12:375<br>
<a href='https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-12-375'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--s3-service-broker-live-ddda94b7-dbb0-4917-917d-776dae91ebba.s3.amazonaws.com-bmc-linkout-fulltext.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/21787423'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">19.</td>
<td><b><font color=blue>CNVnator: an approach to discover, genotype, and characterize typical and atypical CNVs from family and population genome sequencing.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Urban AE, Snyder M, Gerstein M<br>
<b>Genome Res</b> 2011; 21(6):974-84<br>
<a href='http://genome.cshlp.org/cgi/pmidlookup?view=long&pmid=21324876'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--highwire.stanford.edu-icons-externalservices-pubmed-standard-genome_final_free.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/21324876'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">18.</td>
<td><b><font color=blue>Annual Research Review: The promise of stem cell research for neuropsychiatric disorders.</font></b><br>
Vaccarino FM, Urban AE, Stevens HE, Szekely A, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Grigorenko EL, Gerstein M, Weissman S<br>
<b>J Child Psychol Psychiatry</b> 2011; 52(4):504-16<br>
<a href='https://doi.org/10.1111/j.1469-7610.2010.02348.x'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--media.wiley.com-assets-7388-69-wiley-full-text.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/21204834'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">17.</td>
<td><b><font color=blue>AGE: defining breakpoints of genomic structural variants at single-nucleotide resolution, through optimal alignments with gap excision.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Gerstein M<br>
<b>Bioinformatics</b> 2011; 27(5):595-603<br>
<a href='https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btq713'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--academic.oup.com-images-oup_pubmed.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/21233167'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">16.</td>
<td><b><font color=blue>Mapping copy number variation by population-scale genome sequencing.</font></b><br>
Mills RE, Walter K, Stewart C, Handsaker RE, Chen K, Alkan C, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Yoon SC, Ye K, Cheetham RK, Chinwalla A, Conrad DF, Fu Y, Grubert F, Hajirasouliha I, Hormozdiari F, Iakoucheva LM, Iqbal Z, Kang S, Kidd JM, Konkel MK, Korn J, Khurana E, Kural D, Lam HY, Leng J, Li R, Li Y, Lin CY, Luo R, Mu XJ, Nemesh J, Peckham HE, Rausch T, Scally A, Shi X, Stromberg MP, Stütz AM, Urban AE, Walker JA, Wu J, Zhang Y, Zhang ZD, Batzer MA, Ding L, Marth GT, McVean G, Sebat J, Snyder M, Wang J, Ye K, Eichler EE, Gerstein MB, Hurles ME, Lee C, McCarroll SA, Korbel JO, 1000 Genomes Project<br>
<b>Nature</b> 2011; 470(7332):59-65<br>
<a href='https://doi.org/10.1038/nature09708'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--www.nature.com-images-lo_nature.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/21293372'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2010</b><br>
      <table class='papertable'>
<tr>
<td width="40px">15.</td>
<td><b><font color=blue>A map of human genome variation from population-scale sequencing.</font></b><br>
1000 Genomes Project Consortium, Abecasis GR, Altshuler D, Auton A, Brooks LD, Durbin RM, Gibbs RA, Hurles ME, McVean GA<br>
<b>Nature</b> 2010; 467(7319):1061-73<br>
<a href='https://doi.org/10.1038/nature09534'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--www.nature.com-images-lo_nature.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/20981092'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">14.</td>
<td><b><font color=blue>Analysis of combinatorial regulation: scaling of partnerships between regulators with the number of governed targets.</font></b><br>
Bhardwaj N, Carson MB, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Yan KK, Lu H, Gerstein MB<br>
<b>PLoS Comput Biol</b> 2010; 6(5):e1000755<br>
<a href='http://dx.plos.org/10.1371/journal.pcbi.1000755'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--journals.plos.org-ploscompbiol-resource-img-external-pcbi_120x30.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/20523742'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">13.</td>
<td><b><font color=blue>RigidFinder: a fast and sensitive method to detect rigid blocks in large macromolecular complexes.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Bjornson R, Felipe M, Gerstein M<br>
<b>Proteins</b> 2010; 78(2):309-24<br>
<a href='https://doi.org/10.1002/prot.22544'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--media.wiley.com-assets-7388-69-wiley-full-text.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/19705487'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2009</b><br>
      <table class='papertable'>
<tr>
<td width="40px">12.</td>
<td><b><font color=blue>PEMer: a computational framework with simulation-based error models for inferring genomic structural variants from massive paired-end sequencing data.</font></b><br>
Korbel JO, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Mu XJ, Carriero N, Cayting P, Zhang Z, Snyder M, Gerstein MB<br>
<b>Genome Biol</b> 2009; 10(2):R23<br>
<a href='https://genomebiology.biomedcentral.com/articles/10.1186/gb-2009-10-2-r23'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--s3-service-broker-live-ddda94b7-dbb0-4917-917d-776dae91ebba.s3.amazonaws.com-bmc-linkout-fulltext.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/19236709'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">11.</td>
<td><b><font color=blue>MSB: a mean-shift-based approach for the analysis of structural variation in the genome.</font></b><br>
Wang LY, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Korbel JO, Snyder M, Gerstein M<br>
<b>Genome Res</b> 2009; 19(1):106-17<br>
<a href='http://genome.cshlp.org/cgi/pmidlookup?view=long&pmid=19037015'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--highwire.stanford.edu-icons-externalservices-pubmed-standard-genome_final_free.gif'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/19037015'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2008</b><br>
      <table class='papertable'>
<tr>
<td width="40px">10.</td>
<td><b><font color=blue>An AP endonuclease 1-DNA polymerase beta complex: theoretical prediction of interacting surfaces.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Uzun A, Strauss PR, Ilyin VA<br>
<b>PLoS Comput Biol</b> 2008; 4(4):e1000066<br>
<a href='http://dx.plos.org/10.1371/journal.pcbi.1000066'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--journals.plos.org-ploscompbiol-resource-img-external-pcbi_120x30.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/18437203'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2007</b><br>
      <table class='papertable'>
<tr>
<td width="40px">9.</td>
<td><b><font color=blue>UmuD and RecA directly modulate the mutagenic potential of the Y family DNA polymerase DinB.</font></b><br>
Godoy VG, Jarosz DF, Simon SM, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Ilyin V, Walker GC<br>
<b>Mol Cell</b> 2007; 28(6):1058-70<br>
<a href='https://linkinghub.elsevier.com/retrieve/pii/S1097-2765(07)00732-0'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--linkinghub.elsevier.com-ihub-images-celloa.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/18158902'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">8.</td>
<td><b><font color=blue>A comprehensive analysis of non-sequential alignments between all protein structures.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Ilyin VA<br>
<b>BMC Struct Biol</b> 2007; 7:78<br>
<a href='https://bmcstructbiol.biomedcentral.com/articles/10.1186/1472-6807-7-78'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--s3-service-broker-live-ddda94b7-dbb0-4917-917d-776dae91ebba.s3.amazonaws.com-bmc-linkout-fulltext.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/18005453'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">7.</td>
<td><b><font color=blue>Structure SNP (StSNP): a web server for mapping and modeling nsSNPs on protein structures with linkage to metabolic pathways.</font></b><br>
Uzun A, Leslin CM, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Ilyin V<br>
<b>Nucleic Acids Res</b> 2007; 35(Web Server issue):W384-92<br>
<a href='https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkm232'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--academic.oup.com-images-oup_pubmed.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/17537826'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">6.</td>
<td><b><font color=blue>TOPOFIT-DB, a database of protein structural alignments based on the TOPOFIT method.</font></b><br>
Leslin CM, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Ilyin VA<br>
<b>Nucleic Acids Res</b> 2007; 35(Database issue):D317-21<br>
<a href='https://academic.oup.com/nar/article-lookup/doi/10.1093/nar/gkl809'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--academic.oup.com-images-oup_pubmed.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/17065464'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2005</b><br>
      <table class='papertable'>
<tr>
<td width="40px">5.</td>
<td><b><font color=blue>Friend, an integrated analytical front-end application for bioinformatics.</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Errami M, Leslin CM, Ilyin VA<br>
<b>Bioinformatics</b> 2005; 21(18):3677-8<br>
<a href='https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/bti602'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--academic.oup.com-images-oup_pubmed.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/16076889'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">4.</td>
<td><b><font color=blue>Active site prediction for comparative model structures with thematics.</font></b><br>
Shehadi IA, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Uzun A, Wei Y, Murga LF, Ilyin V, Ondrechen MJ<br>
<b>J Bioinform Comput Biol</b> 2005; 3(1):127-43<br>
<a href='https://www.worldscientific.com/doi/full/10.1142/s0219720005000916?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%3dpubmed'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--www.worldscientific.com-pb-assets-wspc-site-wsnetlogo.jpg'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/15751116'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2004</b><br>
      <table class='papertable'>
<tr>
<td width="40px">3.</td>
<td><b><font color=blue>Structural exon database, SEDB, mapping exon boundaries on multiple protein structures.</font></b><br>
Leslin CM, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Ilyin VA<br>
<b>Bioinformatics</b> 2004; 20(11):1801-3<br>
<a href='https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/bth150'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/https:--academic.oup.com-images-oup_pubmed.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/14988102'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
<tr>
<td width="40px">2.</td>
<td><b><font color=blue>Structural alignment of proteins by a novel TOPOFIT method, as a superimposition of common volumes at a topomax point.</font></b><br>
Ilyin VA, <span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Leslin CM<br>
<b>Protein Sci</b> 2004; 13(7):1865-74<br>
<a href='https://doi.org/10.1110/ps.04672604'><img class='paperimg' src='https://www.ncbi.nlm.nih.gov/corehtml/query/egifs/http:--media.wiley.com-assets-7388-68-wiley-free-full-text.png'/></a><a href='https://ncbi.nlm.nih.gov/pubmed/15215530'><img class='paperimg' src='imgs/pubmed.png'/></a>
</td>
</tr>
      </table><br>      <b>2002</b><br>
      <table class='papertable'>
<tr>
<td width="40px">1.</td>
<td><b><font color=blue>Efficiency Profile Method To Study The Hit Efficiency Of Drift Chambers</font></b><br>
<span class='author' onmouseover='author_image="imgs/Alexej.jpg";'><b><u>Abyzov A</u></b></span>, Bel'kov A, Lanyov A, Spiridonov A, Walter M, Hulsbergen W<br>
<b>Particles and Nuclei, Letters</b> 2002; 2002; 5(114):40-52<br>

</td>
</tr>
      </table>
</div>

<div class="w3-container" style="padding:128px 16px" id="tools">
  <h3 class="w3-center">TOOLS</h3>
        <div class='w3-row'><div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>CNVnator</h3>
</header>
<div class="w3-container">
  
  <div style="min-height:60px;">
  <p>A tool for CNV discovery and genotyping from depth of read mapping.</b><br>
  </div>
  <div class="github-card" data-github="abyzovlab/CNVnator" data-width="100%" data-height="153" data-theme="default"></div>
  <br>More: <a href='http://www.ncbi.nlm.nih.gov/pubmed/21293372'>ref</a>
</div>
<footer class="w3-container w3-light-grey">
  <h5>CNVnator</h5>
</footer>
</div>
</div>


<div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>CNVpytor</h3>
</header>
<div class="w3-container">
  
  <div style="min-height:60px;">
  <p>CNVpytor is a Python extension of CNVnator: package and command line tool for CNV analysis.</b><br>
  </div>
  <div class="github-card" data-github="abyzovlab/CNVpytor" data-width="100%" data-height="153" data-theme="default"></div>
  <br>More: <a href='https://github.com/abyzovlab/CNVpytor'>ref</a>
</div>
<footer class="w3-container w3-light-grey">
  <h5>CNVpytor</h5>
</footer>
</div>
</div>


<div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>AGE</h3>
</header>
<div class="w3-container">
  
  <div style="min-height:60px;">
  <p>A tool that implements an algorithm for optimal alignment of sequences with SVs.</b><br>
  </div>
  <div class="github-card" data-github="abyzovlab/AGE" data-width="100%" data-height="153" data-theme="default"></div>
  <br>More: <a href='http://www.ncbi.nlm.nih.gov/pubmed/21233167''>ref</a>
</div>
<footer class="w3-container w3-light-grey">
  <h5>AGE</h5>
</footer>
</div>
</div>


<div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>Leucippus</h3>
</header>
<div class="w3-container">
  
  <div style="min-height:60px;">
  <p>A tool for confirmation of mosaic variants from targeted sequencing.</b><br>
  </div>
  <div class="github-card" data-github="abyzovlab/Leucippus" data-width="100%" data-height="153" data-theme="default"></div>
  <br>More: <a href='https://www.ncbi.nlm.nih.gov/pubmed/28235832'>ref</a>
</div>
<footer class="w3-container w3-light-grey">
  <h5>Leucippus</h5>
</footer>
</div>
</div>


<div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>vcf2diploid</h3>
</header>
<div class="w3-container">
  
  <div style="min-height:60px;">
  <p>Personal genome constructor, it can be used to construct a personal diploid genome sequence by including personal variants into reference genome.</b><br>
  </div>
  <div class="github-card" data-github="abyzovlab/vcf2diploid" data-width="100%" data-height="153" data-theme="default"></div>
  <br>More: <a href='http://www.ncbi.nlm.nih.gov/pubmed/21811232'>ref</a>
</div>
<footer class="w3-container w3-light-grey">
  <h5>vcf2diploid</h5>
</footer>
</div>
</div>


<div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>Belka</h3>
</header>
<div class="w3-container">
  
  <div style="min-height:60px;">
  <p>A software that implements RigidFinder, a method to detect protein rigid blocks from two different conformations.<br><br>Download <a href="http://belka.molmovdb.org/Belka_v0.1.zip"><b><u>Belka v0.1</u></b></a></b><br>
  </div>
  
  <br>More: <a href='http://www.ncbi.nlm.nih.gov/pubmed/19705487'>ref</a>
</div>
<footer class="w3-container w3-light-grey">
  <h5>Belka</h5>
</footer>
</div>
</div>


</div><script src="//cdn.jsdelivr.net/github-cards/latest/widget.js"></script>

</div>

<div class="w3-container" style="padding:128px 16px" id="positions">
  <h3 class="w3-center">POSITIONS</h3>
  <div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>PostDoc in computational biology and bioinformatics</h3>
</header>
<div class="w3-container">
  <p>application accepted year-round</p>
  <hr>
  <img src="imgs/phd.png" alt="Avatar" class="w3-left w3-circle w3-padding" style="width:20%;max-width:200px">
  <p>Applicants are invited to apply for a post-doctoral (i.e., postdoc) position in Abyzov lab at Mayo Clinic. The choice of project will depend on the applicant's interests and skills, however, the research must be purely computational and focus on one of the following main fields of computational biology: population/personal human omics, cancer omics, single cell and somatic omics, and the analysis of next-generation sequencing data. Specific sub-areas of interest are discovery, annotation, and the functional annotation of human genomic variants, cancer genomics, cancer evolution, somatic mosaicism in normal human cells.</p>
  <p>The ideal applicant will have a Ph.D. in computational biology or bioinformatics, experience in one of the aforementioned research areas, demonstrate a record of peer-reviewed publications, and possess motivation for independent research. He or she should have a very strong understanding of biology and be skilled in programming and using computers to solve problems (e.g., experience with C/C++, Java, Python/Perl, R/ROOT, etc.). Oral and written proficiency in English is also a big plus.</p>
  <p>To apply, please email your CV, including a list of publications and details for three references, to abyzov dot alexej at mayo dot edu. Please include the phrase “PostDoc application” and your full name in the subject of the email.</p>
</div>
<footer class="w3-container w3-light-grey">
<br>
</footer>
</div>
</div>

<div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>Internship in computational biology and bioinformatics</h3>
</header>
<div class="w3-container">
  <p>application accepted year-round</p>
  <hr>
  <img src="imgs/img_avatar3.png" alt="Avatar" class="w3-left w3-circle w3-padding" style="width:20%;max-width:200px">
  <p>Applications are invited for an internship at the Mayo Clinic. Anticipated projects will be related to the analysis of whole genome sequencing data, with the aims of studying germline and somatic variants (SNPs, CNVs, etc.). The analysis will involve applications of commonly used, and in-house developed, software tools, and making biological hypothesis from statistical data analysis. Intern applicants with strong programming skills will have opportunities to participate in developing new tools and improving our existing software.</p>
  <p>To apply, please email your CV, including a list of publications to abyzov dot alexej at mayo dot edu. Please include the phrase “Internship application” and your full name in the subject of the email.</p>
</div>
<footer class="w3-container w3-light-grey">
<br>
</footer>
</div>
</div>

<div class="w3-col l4 m6 w3-padding-16">
<div class="w3-card-4 w3-margin">
<header class="w3-container w3-dark-grey">
  <h3>Graduate student in computational biology and bioinformatics</h3>
</header>
<div class="w3-container">
  <img src="imgs/img_avatar3.png" alt="Avatar" class="w3-left w3-circle w3-padding" style="width:20%;max-width:200px">
  <p>Graduate students (M.S or Ph.D.) wishing to conduct research in the Abyzov lab at Mayo Clinic are invited to contact Dr. Abyzov (abyzov dot alexej at mayo dot edu). The choice of the project will depend on the applicant's interests and skills. However, the research must be purely computational and focus on one of the following main fields of computational biology: population/personal human omics, cancer omics, single cell and somatic omics, and the analysis of next-generation sequencing data. Specific sub-areas of interest are discovery, annotation, and the functional annotation of human genomic variants such as SNPs, SNVs, indels, structural variations, retrotransposition, etc.</p>
  <p>We are looking for candidates that possess motivation for independent research, have experience in computational biology or bioinformatics, and are familiar with one of the aforementioned research areas. They should have a very strong understanding of biology and be skilled in programming and using computers to solve problems (e.g., experience with C/C++ Java, Python/Perl, R/ROOT, etc.). Record of peer-reviewed publications and oral and written proficiency in English is also a big plus.</p>
  <p>Dr. Abyzov is affiliated with <a href="https://college.mayo.edu">Mayo Clinic College of Medicine</a> and with the program in <a href="https://r.umn.edu/academics-research/graduate-programs/bicb">Biomedical Informatics and Computational Biology</a> at University of Minnesota Rochester.</p>
  <p>Please express your interest by emailing your CV, including a list of publications to abyzov dot alexej at mayo dot edu. Please include the phrase “PhD/MS interest” and your full name in the subject of the email.</p>
</div>
<footer class="w3-container w3-light-grey">
<br>
</footer>
</div>
</div>

</div>


<!-- Contact Section -->
<div class="w3-container" style="padding:128px 16px" id="contact">
  <div class="w3-row-padding">
    <div class="w3-col m6">
      Click on image for map:
      <a href="https://goo.gl/maps/TFaFnWVykEAG2qBDA"><img width="100%" src="imgs/map.png"></img></a>

    </div>
    <div class="w3-col m6 w3-center">
      <h3 class="w3-center">CONTACT</h3>
        Lets get in touch. Send us a message:<br><br>
        <b>Alexej Abyzov</b>, Ph.D., Principal Investigator<br>
        <p><i class="fa fa-envelope fa-fw w3-xxlarge w3-margin-right"> </i> Email: abyzov.alexej@mayo.edu</p>
        <br><br>
        <b>Leighann Wagner</b>, Administrative Assistant<br>
        <p><i class="fa fa-phone fa-fw w3-xxlarge w3-margin-right"></i> Phone: (507) 284-5569</p>
        <p><i class="fa fa-envelope fa-fw w3-xxlarge w3-margin-right"> </i> Email:
        wagner.leighann@mayo.edu</p><br><br>
      <table style="margin:auto;">
        <tr><td>
        <img src="imgs/pin.png" width="48px" /><br>
        <td> 
          Harwick Building 3rd floor, Mayo Clinic<br>
          200 1st street SW<br>
          Rochester, MN 55905, USA<br>
      </table>
    
    </div>
  </div>
</div>

<!-- Footer -->
<footer class="w3-center w3-light-grey w3-padding-32">
  <a href="#home" class="w3-button w3-dark-grey"><i class="fa fa-arrow-up w3-margin-right"></i>To the top</a><br>
  <p>Powered by <a href="https://www.w3schools.com/w3css/default.asp" title="W3.CSS" target="_blank" class="w3-hover-text-green">w3.css</a></p>
</footer>
<img id="image" width=150 src="imgs/he.jpg"></img> 

<script>
// Modal Image Gallery
function onClick(element) {
  document.getElementById("img01").src = element.src;
  document.getElementById("modal01").style.display = "block";
  var captionText = document.getElementById("caption");
  captionText.innerHTML = element.alt;
}


// Toggle between showing and hiding the sidebar when clicking the menu icon
var mySidebar = document.getElementById("mySidebar");

function w3_open() {
  if (mySidebar.style.display === 'block') {
    mySidebar.style.display = 'none';
  } else {
    mySidebar.style.display = 'block';
  }
}

// Close the sidebar with the close button
function w3_close() {
    mySidebar.style.display = "none";
}

$(document).mousemove(function(e){
    $("#image").css({left:e.pageX, top:e.pageY-$("#image").height()});
});

function showimage() {
  $("#image").show();
  $("#image").attr("src",author_image)
}

function hideimage() {
  $("#image").hide();
}

$(".author").hover(showimage,hideimage);

$('a[href^="#"]').on('click', function(event) {

    var target = $(this.getAttribute('href'));

    if( target.length ) {
        event.preventDefault();
        $('html, body').stop().animate({
            scrollTop: target.offset().top
        }, 600);
    }

});

window.onscroll = scroll;

function scroll() {
  off=70;
  maxh=160;
  st=$(document).scrollTop();
  if(st>(maxh+off)) sp=maxh;
  else if(st<off) sp=0;
  else sp=st-off;
    $(".sidebarh").css({ height: sp });
}

var slideIndex = 1;
showDivs(slideIndex);

function plusDivs(n) {
  showDivs(slideIndex += n);
}

function showDivs(n) {
  var i;
  var x = document.getElementsByClassName("mySlides");
  if (n > x.length) {slideIndex = 1}
  if (n < 1) {slideIndex = x.length}
  for (i = 0; i < x.length; i++) {
    x[i].style.display = "none";  
  }
  x[slideIndex-1].style.display = "block";  
}
</script>

</body>
</html>


